automation/Extension/webext-instrumentation/package.json

```diff
@@ -19,7 +19,7 @@
     "fix:prettier": "prettier \"src/**/*.ts\" --write",
     "fix:tslint": "tslint -t verbose --fix --project .",
     "test": "run-s build test:*",
-    "test:lint": "tslint -t verbose --project . && prettier \"src/**/*.ts\" --list-different",
+    "test:lint": "tslint -t verbose --project . && prettier \"src/**/*.ts\" --list-different || exit 0",
```

Comment:
Why did you change this line?

Reply:
Hi, 
in my case lint always aborted without any meaningful error message (``unsafe-perm in lifecycle true'' ) which I attributed to the low resources of my machine ;(
See also the logs below. Any pointers what might be wrong help.

## Build script log
```
(openwpm) openwpm@openwpm-VirtualBox:~/MyOpenWPM/OpenWPM$ ./scripts/build-extension.sh 
~/MyOpenWPM/OpenWPM/automation/Extension/firefox ~/MyOpenWPM/OpenWPM

> OpenWPM@ postinstall /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox
> cd ../webext-instrumentation && npm install


> @openwpm/webext-instrumentation@ prepare /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> run-s build test


> @openwpm/webext-instrumentation@ build /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> run-s clean && run-p build:*


> @openwpm/webext-instrumentation@ clean /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> trash build test


> @openwpm/webext-instrumentation@ build:main /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> tsc -p tsconfig.json


> @openwpm/webext-instrumentation@ build:module /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> tsc -p tsconfig.module.json


> @openwpm/webext-instrumentation@ test /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> run-s build test:*


> @openwpm/webext-instrumentation@ build /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> run-s clean && run-p build:*


> @openwpm/webext-instrumentation@ clean /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> trash build test


> @openwpm/webext-instrumentation@ build:main /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> tsc -p tsconfig.json


> @openwpm/webext-instrumentation@ build:module /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> tsc -p tsconfig.module.json


> @openwpm/webext-instrumentation@ test:lint /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation
> tslint -t verbose --project . && prettier "src/**/*.ts" --list-different

WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/cookie-instrument.ts[64, 9]: The key 'change_cause' is not sorted alphabetically
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/cookie-instrument.ts[81, 11]: The key 'crawl_id' is not sorted alphabetically
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/dns-instrument.ts[77, 12]: non-arrow functions are forbidden
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/http-instrument.ts[277, 11]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/http-instrument.ts[287, 15]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/http-instrument.ts[338, 23]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/http-instrument.ts[560, 7]: The key 'crawl_id' is not sorted alphabetically
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/http-instrument.ts[676, 11]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/http-instrument.ts[696, 15]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/javascript-instrument.ts[119, 7]: The key 'jsInstrumentationSettingsString' is not sorted alphabetically
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/javascript-instrument.ts[132, 9]: The key 'allFrames' is not sorted alphabetically
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/javascript-instrument.ts[140, 7]: The key 'allFrames' is not sorted alphabetically
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/navigation-instrument.ts[22, 11]: The key 'incognito' is not sorted alphabetically
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/background/navigation-instrument.ts[34, 5]: The key 'extension_session_uuid' is not sorted alphabetically
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-content-scope.ts[4, 1]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-content-scope.ts[25, 1]: non-arrow functions are forbidden
WARNING: (one-variable-per-declaration) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-content-scope.ts[30, 3]: Multiple variable declarations in the same statement are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-content-scope.ts[40, 1]: non-arrow functions are forbidden
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-content-scope.ts[45, 5]: The key 'data' is not sorted alphabetically
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-content-scope.ts[52, 36]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-content-scope.ts[56, 18]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-content-scope.ts[64, 8]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-page-scope.ts[5, 27]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-page-scope.ts[7, 3]: non-arrow functions are forbidden
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-page-scope.ts[20, 5]: Calls to 'console.log' are not allowed.
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-page-scope.ts[22, 5]: Calls to 'console.log' are not allowed.
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-page-scope.ts[27, 5]: Calls to 'console.log' are not allowed.
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/content/javascript-instrument-page-scope.ts[29, 5]: Calls to 'console.log' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[32, 8]: non-arrow functions are forbidden
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[48, 9]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[60, 34]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[73, 29]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[88, 3]: non-arrow functions are forbidden
WARNING: (one-variable-per-declaration) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[89, 5]: Multiple variable declarations in the same statement are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[91, 19]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[122, 3]: non-arrow functions are forbidden
WARNING: (prefer-for-of) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[132, 5]: Expected a 'for-of' loop instead of a 'for' loop with this simple iteration
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[157, 3]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[177, 37]: non-arrow functions are forbidden
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[210, 7]: Calls to 'console.log' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[215, 3]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[228, 3]: non-arrow functions are forbidden
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[253, 7]: The key 'scriptUrl' is not sorted alphabetically
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[265, 7]: Calls to 'console.log' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[273, 3]: non-arrow functions are forbidden
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[304, 9]: The key 'args' is not sorted alphabetically
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[316, 7]: Calls to 'console.log' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[324, 3]: non-arrow functions are forbidden
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[325, 5]: Calls to 'console.error' are not allowed.
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[326, 5]: Calls to 'console.error' are not allowed.
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[327, 5]: Calls to 'console.error' are not allowed.
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[328, 5]: Calls to 'console.error' are not allowed.
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[329, 5]: Calls to 'console.error' are not allowed.
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[331, 7]: Calls to 'console.error' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[336, 3]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[349, 18]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[356, 3]: non-arrow functions are forbidden
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[361, 11]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[363, 7]: The key 'scriptLine' is not sorted alphabetically
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[407, 9]: The key 'scriptLine' is not sorted alphabetically
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[420, 7]: Calls to 'console.log' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[429, 3]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[447, 3]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[466, 3]: non-arrow functions are forbidden
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[495, 7]: Calls to 'console.error' are not allowed.
WARNING: (object-literal-sort-keys) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[513, 7]: The key 'enumerable' is not sorted alphabetically
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[525, 13]: non-arrow functions are forbidden
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[544, 13]: Calls to 'console.error' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[605, 13]: non-arrow functions are forbidden
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[645, 13]: Calls to 'console.error' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[670, 3]: non-arrow functions are forbidden
WARNING: (no-console) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[719, 11]: Calls to 'console.warn' are not allowed.
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[744, 23]: non-arrow functions are forbidden
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[747, 11]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[747, 28]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[754, 12]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[763, 3]: non-arrow functions are forbidden
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[763, 25]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/js-instruments.ts[769, 34]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/number.ts[22, 8]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/number.ts[43, 8]: non-arrow functions are forbidden
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/response-body-listener.ts[36, 21]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/sha256.ts[6, 14]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/string-utils.ts[1, 8]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/string-utils.ts[5, 29]: non-arrow functions are forbidden
WARNING: (triple-equals) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/string-utils.ts[7, 18]: != should be !==
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/string-utils.ts[14, 26]: non-arrow functions are forbidden
WARNING: (variable-name) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/string-utils.ts[32, 14]: variable name must be in lowerCamelCase or UPPER_CASE
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/string-utils.ts[32, 30]: non-arrow functions are forbidden
WARNING: (only-arrow-functions) /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/src/lib/string-utils.ts[46, 26]: non-arrow functions are forbidden

src/background/dns-instrument.ts
src/background/http-instrument.ts
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @openwpm/webext-instrumentation@ test:lint: `tslint -t verbose --project . && prettier "src/**/*.ts" --list-different`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @openwpm/webext-instrumentation@ test:lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/openwpm/.npm/_logs/2020-08-17T08_22_32_473Z-debug.log
ERROR: "test:lint" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @openwpm/webext-instrumentation@ test: `run-s build test:*`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @openwpm/webext-instrumentation@ test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/openwpm/.npm/_logs/2020-08-17T08_22_32_575Z-debug.log
ERROR: "test" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @openwpm/webext-instrumentation@ prepare: `run-s build test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @openwpm/webext-instrumentation@ prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/openwpm/.npm/_logs/2020-08-17T08_22_32_713Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! OpenWPM@ postinstall: `cd ../webext-instrumentation && npm install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the OpenWPM@ postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/openwpm/.npm/_logs/2020-08-17T08_22_32_874Z-debug.log
(openwpm) openwpm@openwpm-VirtualBox:~/MyOpenWPM/OpenWPM$ 

```

## Complete log
```
(openwpm) openwpm@openwpm-VirtualBox:~/MyOpenWPM/OpenWPM$ cat /home/openwpm/.npm/_logs/2020-08-17T08_22_32_874Z-debug.log 
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/home/openwpm/anaconda3/envs/openwpm/bin/node',
1 verbose cli   '/home/openwpm/anaconda3/envs/openwpm/bin/npm',
1 verbose cli   'install'
1 verbose cli ]
2 info using npm@6.14.5
3 info using node@v14.5.0
4 verbose npm-session dafa7b062b466367
5 silly install runPreinstallTopLevelLifecycles
6 silly preinstall OpenWPM@
7 info lifecycle OpenWPM@~preinstall: OpenWPM@
8 silly install loadCurrentTree
9 silly install readLocalPackageData
10 timing stage:loadCurrentTree Completed in 14313ms
11 silly install loadIdealTree
12 silly install cloneCurrentTreeToIdealTree
13 timing stage:loadIdealTree:cloneCurrentTree Completed in 329ms
14 silly install loadShrinkwrap
15 timing stage:loadIdealTree:loadShrinkwrap Completed in 3763ms
16 silly install loadAllDepsIntoIdealTree
17 timing stage:loadIdealTree:loadAllDepsIntoIdealTree Completed in 4220ms
18 timing stage:loadIdealTree Completed in 10346ms
19 silly currentTree OpenWPM@
19 silly currentTree ├── @babel/code-frame@7.0.0
19 silly currentTree ├── @babel/highlight@7.0.0
19 silly currentTree ├── @babel/polyfill@7.7.0
19 silly currentTree ├── @babel/runtime-corejs3@7.10.4
19 silly currentTree ├── @babel/runtime@7.7.7
19 silly currentTree ├── @cliqz-oss/firefox-client@0.3.1
19 silly currentTree ├── @cliqz-oss/node-firefox-connect@1.2.1
19 silly currentTree ├─┬ @openwpm/webext-instrumentation@
19 silly currentTree │ ├── @babel/code-frame@7.10.4
19 silly currentTree │ ├─┬ @babel/generator@7.10.4
19 silly currentTree │ │ └── source-map@0.5.7
19 silly currentTree │ ├── @babel/helper-function-name@7.10.4
19 silly currentTree │ ├── @babel/helper-get-function-arity@7.10.4
19 silly currentTree │ ├── @babel/helper-split-export-declaration@7.10.4
19 silly currentTree │ ├── @babel/helper-validator-identifier@7.10.4
19 silly currentTree │ ├─┬ @babel/highlight@7.10.4
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├── chalk@2.4.2
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ └── supports-color@5.5.0
19 silly currentTree │ ├── @babel/parser@7.10.4
19 silly currentTree │ ├── @babel/runtime-corejs3@7.10.4
19 silly currentTree │ ├── @babel/template@7.10.4
19 silly currentTree │ ├── @babel/traverse@7.10.4
19 silly currentTree │ ├── @babel/types@7.10.4
19 silly currentTree │ ├── @commitlint/execute-rule@9.0.1
19 silly currentTree │ ├─┬ @commitlint/load@9.0.1
19 silly currentTree │ │ └── chalk@3.0.0
19 silly currentTree │ ├── @commitlint/resolve-extends@9.0.1
19 silly currentTree │ ├── @commitlint/types@9.0.1
19 silly currentTree │ ├─┬ @concordance/react@2.0.0
19 silly currentTree │ │ └── arrify@1.0.1
19 silly currentTree │ ├── @nodelib/fs.scandir@2.1.3
19 silly currentTree │ ├── @nodelib/fs.stat@2.0.3
19 silly currentTree │ ├── @nodelib/fs.walk@1.2.4
19 silly currentTree │ ├─┬ @sindresorhus/df@2.1.0
19 silly currentTree │ │ ├── execa@0.2.2
19 silly currentTree │ │ ├── npm-run-path@1.0.0
19 silly currentTree │ │ └── path-key@1.0.0
19 silly currentTree │ ├── @sindresorhus/is@0.14.0
19 silly currentTree │ ├── @szmarczak/http-timer@1.1.2
19 silly currentTree │ ├── @tootallnate/once@1.1.2
19 silly currentTree │ ├── @types/color-name@1.1.1
19 silly currentTree │ ├── @types/firefox-webext-browser@70.0.1
19 silly currentTree │ ├── @types/glob@7.1.3
19 silly currentTree │ ├── @types/minimatch@3.0.3
19 silly currentTree │ ├── @types/node@14.0.20
19 silly currentTree │ ├── @types/normalize-package-data@2.4.0
19 silly currentTree │ ├── @types/parse-json@4.0.0
19 silly currentTree │ ├── acorn-walk@7.2.0
19 silly currentTree │ ├── acorn@7.3.1
19 silly currentTree │ ├── add-stream@1.0.0
19 silly currentTree │ ├── agent-base@6.0.1
19 silly currentTree │ ├── aggregate-error@3.0.1
19 silly currentTree │ ├─┬ ansi-align@3.0.0
19 silly currentTree │ │ ├── ansi-regex@4.1.0
19 silly currentTree │ │ ├── emoji-regex@7.0.3
19 silly currentTree │ │ ├── is-fullwidth-code-point@2.0.0
19 silly currentTree │ │ ├── string-width@3.1.0
19 silly currentTree │ │ └── strip-ansi@5.2.0
19 silly currentTree │ ├── ansi-escapes@3.2.0
19 silly currentTree │ ├── ansi-regex@5.0.0
19 silly currentTree │ ├── ansi-styles@4.2.1
19 silly currentTree │ ├── anymatch@3.1.1
19 silly currentTree │ ├── append-transform@1.0.0
19 silly currentTree │ ├── archy@1.0.0
19 silly currentTree │ ├── argparse@1.0.10
19 silly currentTree │ ├── argv@0.0.2
19 silly currentTree │ ├── arr-diff@4.0.0
19 silly currentTree │ ├── arr-flatten@1.1.0
19 silly currentTree │ ├── arr-union@3.1.0
19 silly currentTree │ ├── array-find-index@1.0.2
19 silly currentTree │ ├── array-ify@1.0.0
19 silly currentTree │ ├── array-union@2.1.0
19 silly currentTree │ ├── array-uniq@1.0.3
19 silly currentTree │ ├── array-unique@0.3.2
19 silly currentTree │ ├── arrgv@1.0.2
19 silly currentTree │ ├── arrify@2.0.1
19 silly currentTree │ ├── assign-symbols@1.0.0
19 silly currentTree │ ├── astral-regex@2.0.0
19 silly currentTree │ ├── async@2.6.3
19 silly currentTree │ ├── atob@2.1.2
19 silly currentTree │ ├── ava@3.10.1
19 silly currentTree │ ├── balanced-match@1.0.0
19 silly currentTree │ ├─┬ base@0.11.2
19 silly currentTree │ │ ├── define-property@1.0.0
19 silly currentTree │ │ ├── is-accessor-descriptor@1.0.0
19 silly currentTree │ │ ├── is-data-descriptor@1.0.0
19 silly currentTree │ │ └── is-descriptor@1.0.2
19 silly currentTree │ ├── binary-extensions@2.1.0
19 silly currentTree │ ├── blueimp-md5@2.16.0
19 silly currentTree │ ├─┬ boxen@4.2.0
19 silly currentTree │ │ ├── camelcase@5.3.1
19 silly currentTree │ │ ├── chalk@3.0.0
19 silly currentTree │ │ └── type-fest@0.8.1
19 silly currentTree │ ├── brace-expansion@1.1.11
19 silly currentTree │ ├── braces@3.0.2
19 silly currentTree │ ├── buffer-from@1.1.1
19 silly currentTree │ ├── builtin-modules@1.1.1
19 silly currentTree │ ├── cache-base@1.0.1
19 silly currentTree │ ├─┬ cacheable-request@6.1.0
19 silly currentTree │ │ ├── get-stream@5.1.0
19 silly currentTree │ │ └── lowercase-keys@2.0.0
19 silly currentTree │ ├── cachedir@2.2.0
19 silly currentTree │ ├─┬ caching-transform@3.0.2
19 silly currentTree │ │ ├── make-dir@2.1.0
19 silly currentTree │ │ ├── semver@5.7.1
19 silly currentTree │ │ └── write-file-atomic@2.4.3
19 silly currentTree │ ├── callsites@3.1.0
19 silly currentTree │ ├── camelcase-keys@4.2.0
19 silly currentTree │ ├── camelcase@4.1.0
19 silly currentTree │ ├── capture-stack-trace@1.0.1
19 silly currentTree │ ├── chalk@4.1.0
19 silly currentTree │ ├── chardet@0.7.0
19 silly currentTree │ ├── chokidar@3.4.0
19 silly currentTree │ ├── chunkd@2.0.1
19 silly currentTree │ ├── ci-info@2.0.0
19 silly currentTree │ ├── ci-parallel-vars@1.0.1
19 silly currentTree │ ├─┬ class-utils@0.3.6
19 silly currentTree │ │ └── define-property@0.2.5
19 silly currentTree │ ├── clean-stack@2.2.0
19 silly currentTree │ ├── clean-yaml-object@0.1.0
19 silly currentTree │ ├── cli-boxes@2.2.0
19 silly currentTree │ ├── cli-cursor@3.1.0
19 silly currentTree │ ├── cli-spinners@2.3.0
19 silly currentTree │ ├── cli-truncate@2.1.0
19 silly currentTree │ ├── cli-width@2.2.1
19 silly currentTree │ ├── cliui@6.0.0
19 silly currentTree │ ├── clone-response@1.0.2
19 silly currentTree │ ├── clone@1.0.4
19 silly currentTree │ ├── code-excerpt@3.0.0
19 silly currentTree │ ├── code-point-at@1.1.0
19 silly currentTree │ ├─┬ codecov@3.7.1
19 silly currentTree │ │ └── js-yaml@3.13.1
19 silly currentTree │ ├── collection-visit@1.0.0
19 silly currentTree │ ├── color-convert@2.0.1
19 silly currentTree │ ├── color-name@1.1.4
19 silly currentTree │ ├── commander@2.20.3
19 silly currentTree │ ├─┬ commitizen@4.1.2
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├── chalk@2.4.2
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── cz-conventional-changelog@3.2.0
19 silly currentTree │ │ ├── glob@7.1.4
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ ├── lodash@4.17.15
19 silly currentTree │ │ ├── strip-bom@4.0.0
19 silly currentTree │ │ ├── strip-json-comments@3.0.1
19 silly currentTree │ │ └── supports-color@5.5.0
19 silly currentTree │ ├── common-path-prefix@3.0.0
19 silly currentTree │ ├── commondir@1.0.1
19 silly currentTree │ ├─┬ compare-func@1.3.2
19 silly currentTree │ │ ├── dot-prop@3.0.0
19 silly currentTree │ │ └── is-obj@1.0.1
19 silly currentTree │ ├── component-emitter@1.3.0
19 silly currentTree │ ├── concat-map@0.0.1
19 silly currentTree │ ├── concat-stream@2.0.0
19 silly currentTree │ ├── concordance@5.0.0
19 silly currentTree │ ├── configstore@5.0.1
19 silly currentTree │ ├── conventional-changelog-angular@5.0.6
19 silly currentTree │ ├── conventional-changelog-atom@2.0.3
19 silly currentTree │ ├── conventional-changelog-codemirror@2.0.3
19 silly currentTree │ ├── conventional-changelog-conventionalcommits@4.2.3
19 silly currentTree │ ├─┬ conventional-changelog-core@4.1.4
19 silly currentTree │ │ ├── git-semver-tags@3.0.1
19 silly currentTree │ │ ├── load-json-file@4.0.0
19 silly currentTree │ │ ├── path-type@3.0.0
19 silly currentTree │ │ ├── pify@3.0.0
19 silly currentTree │ │ ├── read-pkg-up@3.0.0
19 silly currentTree │ │ ├── read-pkg@3.0.0
19 silly currentTree │ │ └── semver@6.3.0
19 silly currentTree │ ├── conventional-changelog-ember@2.0.4
19 silly currentTree │ ├── conventional-changelog-eslint@3.0.4
19 silly currentTree │ ├── conventional-changelog-express@2.0.1
19 silly currentTree │ ├── conventional-changelog-jquery@3.0.6
19 silly currentTree │ ├── conventional-changelog-jshint@2.0.3
19 silly currentTree │ ├── conventional-changelog-preset-loader@2.3.0
19 silly currentTree │ ├─┬ conventional-changelog-writer@4.0.11
19 silly currentTree │ │ ├── load-json-file@4.0.0
19 silly currentTree │ │ ├── meow@5.0.0
19 silly currentTree │ │ ├── path-type@3.0.0
19 silly currentTree │ │ ├── pify@3.0.0
19 silly currentTree │ │ ├─┬ read-pkg-up@3.0.0
19 silly currentTree │ │ │ └── read-pkg@3.0.0
19 silly currentTree │ │ └── semver@6.3.0
19 silly currentTree │ ├── conventional-commit-types@3.0.0
19 silly currentTree │ ├── conventional-commits-filter@2.0.2
19 silly currentTree │ ├─┬ conventional-commits-parser@3.0.8
19 silly currentTree │ │ ├── load-json-file@4.0.0
19 silly currentTree │ │ ├── meow@5.0.0
19 silly currentTree │ │ ├── path-type@3.0.0
19 silly currentTree │ │ ├── pify@3.0.0
19 silly currentTree │ │ └─┬ read-pkg-up@3.0.0
19 silly currentTree │ │   └── read-pkg@3.0.0
19 silly currentTree │ ├── conventional-recommended-bump@6.0.5
19 silly currentTree │ ├── convert-source-map@1.7.0
19 silly currentTree │ ├── convert-to-spaces@1.0.2
19 silly currentTree │ ├── copy-descriptor@0.1.1
19 silly currentTree │ ├── core-js-pure@3.6.5
19 silly currentTree │ ├── core-util-is@1.0.2
19 silly currentTree │ ├─┬ cosmiconfig@6.0.0
19 silly currentTree │ │ └── parse-json@5.0.0
19 silly currentTree │ ├─┬ cp-file@6.2.0
19 silly currentTree │ │ ├── make-dir@2.1.0
19 silly currentTree │ │ └── semver@5.7.1
19 silly currentTree │ ├── cp-sugar@1.0.0
19 silly currentTree │ ├── create-error-class@3.0.2
19 silly currentTree │ ├── cross-spawn-async@2.2.5
19 silly currentTree │ ├─┬ cross-spawn@6.0.5
19 silly currentTree │ │ └── semver@5.7.1
19 silly currentTree │ ├── crypto-random-string@2.0.0
19 silly currentTree │ ├── currently-unhandled@0.4.1
19 silly currentTree │ ├─┬ cz-conventional-changelog@2.1.0
19 silly currentTree │ │ ├── conventional-commit-types@2.3.0
19 silly currentTree │ │ └── longest@1.0.1
19 silly currentTree │ ├── dargs@4.1.0
19 silly currentTree │ ├── date-time@3.1.0
19 silly currentTree │ ├── dateformat@3.0.3
19 silly currentTree │ ├── debug@4.1.1
19 silly currentTree │ ├─┬ decamelize-keys@1.1.0
19 silly currentTree │ │ ├── decamelize@1.2.0
19 silly currentTree │ │ └── map-obj@1.0.1
19 silly currentTree │ ├── decamelize@3.2.0
19 silly currentTree │ ├── decode-uri-component@0.2.0
19 silly currentTree │ ├── decompress-response@3.3.0
19 silly currentTree │ ├── dedent@0.7.0
19 silly currentTree │ ├── deep-extend@0.6.0
19 silly currentTree │ ├── default-require-extensions@2.0.0
19 silly currentTree │ ├── defaults@1.0.3
19 silly currentTree │ ├── defer-to-connect@1.1.3
19 silly currentTree │ ├── define-properties@1.1.3
19 silly currentTree │ ├─┬ define-property@2.0.2
19 silly currentTree │ │ ├── is-accessor-descriptor@1.0.0
19 silly currentTree │ │ ├── is-data-descriptor@1.0.0
19 silly currentTree │ │ └── is-descriptor@1.0.2
19 silly currentTree │ ├─┬ del@5.1.0
19 silly currentTree │ │ ├── globby@10.0.2
19 silly currentTree │ │ └── p-map@3.0.0
19 silly currentTree │ ├── detect-file@1.0.0
19 silly currentTree │ ├── detect-indent@6.0.0
19 silly currentTree │ ├── detect-newline@3.1.0
19 silly currentTree │ ├── diff@4.0.2
19 silly currentTree │ ├── dir-glob@3.0.1
19 silly currentTree │ ├── dot-prop@5.2.0
19 silly currentTree │ ├─┬ dotgitignore@2.1.0
19 silly currentTree │ │ ├── find-up@3.0.0
19 silly currentTree │ │ ├── locate-path@3.0.0
19 silly currentTree │ │ ├── p-limit@2.3.0
19 silly currentTree │ │ ├── p-locate@3.0.0
19 silly currentTree │ │ └── path-exists@3.0.0
19 silly currentTree │ ├─┬ duplexer2@0.1.4
19 silly currentTree │ │ ├── readable-stream@2.3.7
19 silly currentTree │ │ └── string_decoder@1.1.1
19 silly currentTree │ ├── duplexer3@0.1.4
19 silly currentTree │ ├── elegant-spinner@1.0.1
19 silly currentTree │ ├─┬ elegant-status@1.1.0
19 silly currentTree │ │ ├── ansi-regex@2.1.1
19 silly currentTree │ │ ├── ansi-styles@2.2.1
19 silly currentTree │ │ ├── chalk@1.1.3
19 silly currentTree │ │ ├── strip-ansi@3.0.1
19 silly currentTree │ │ └── supports-color@2.0.0
19 silly currentTree │ ├── email-addresses@3.1.0
19 silly currentTree │ ├── emittery@0.7.1
19 silly currentTree │ ├── emoji-regex@8.0.0
19 silly currentTree │ ├── end-of-stream@1.4.4
19 silly currentTree │ ├── equal-length@1.0.1
19 silly currentTree │ ├── error-ex@1.3.2
19 silly currentTree │ ├── es-abstract@1.17.6
19 silly currentTree │ ├── es-to-primitive@1.2.1
19 silly currentTree │ ├── es6-error@4.1.1
19 silly currentTree │ ├── escape-goat@2.1.1
19 silly currentTree │ ├── escape-string-applescript@2.0.0
19 silly currentTree │ ├── escape-string-regexp@1.0.5
19 silly currentTree │ ├── esprima@4.0.1
19 silly currentTree │ ├── esutils@2.0.3
19 silly currentTree │ ├─┬ execa@0.10.0
19 silly currentTree │ │ └── get-stream@3.0.0
19 silly currentTree │ ├── exit-hook@1.1.1
19 silly currentTree │ ├─┬ expand-brackets@2.1.4
19 silly currentTree │ │ ├── debug@2.6.9
19 silly currentTree │ │ ├── define-property@0.2.5
19 silly currentTree │ │ ├── extend-shallow@2.0.1
19 silly currentTree │ │ └── ms@2.0.0
19 silly currentTree │ ├── expand-tilde@2.0.2
19 silly currentTree │ ├─┬ extend-shallow@3.0.2
19 silly currentTree │ │ ├── is-extendable@1.0.1
19 silly currentTree │ │ └── is-plain-object@2.0.4
19 silly currentTree │ ├── external-editor@3.1.0
19 silly currentTree │ ├─┬ extglob@2.0.4
19 silly currentTree │ │ ├── define-property@1.0.0
19 silly currentTree │ │ ├── extend-shallow@2.0.1
19 silly currentTree │ │ ├── is-accessor-descriptor@1.0.0
19 silly currentTree │ │ ├── is-data-descriptor@1.0.0
19 silly currentTree │ │ └── is-descriptor@1.0.2
19 silly currentTree │ ├── fast-diff@1.2.0
19 silly currentTree │ ├── fast-glob@3.2.4
19 silly currentTree │ ├── fastq@1.8.0
19 silly currentTree │ ├── figures@3.2.0
19 silly currentTree │ ├── file-type@3.9.0
19 silly currentTree │ ├── filename-reserved-regex@1.0.0
19 silly currentTree │ ├── filenamify-url@1.0.0
19 silly currentTree │ ├── filenamify@1.2.1
19 silly currentTree │ ├── fill-range@7.0.1
19 silly currentTree │ ├── filled-array@1.1.0
19 silly currentTree │ ├─┬ find-cache-dir@2.1.0
19 silly currentTree │ │ ├── find-up@3.0.0
19 silly currentTree │ │ ├── locate-path@3.0.0
19 silly currentTree │ │ ├── make-dir@2.1.0
19 silly currentTree │ │ ├── p-limit@2.3.0
19 silly currentTree │ │ ├── p-locate@3.0.0
19 silly currentTree │ │ ├── path-exists@3.0.0
19 silly currentTree │ │ ├── pkg-dir@3.0.0
19 silly currentTree │ │ └── semver@5.7.1
19 silly currentTree │ ├── find-node-modules@2.0.0
19 silly currentTree │ ├── find-root@1.1.0
19 silly currentTree │ ├── find-up@2.1.0
19 silly currentTree │ ├─┬ findup-sync@3.0.0
19 silly currentTree │ │ ├─┬ braces@2.3.2
19 silly currentTree │ │ │ └── extend-shallow@2.0.1
19 silly currentTree │ │ ├─┬ fill-range@4.0.0
19 silly currentTree │ │ │ └── extend-shallow@2.0.1
19 silly currentTree │ │ ├─┬ is-number@3.0.0
19 silly currentTree │ │ │ └── kind-of@3.2.2
19 silly currentTree │ │ ├── micromatch@3.1.10
19 silly currentTree │ │ └── to-regex-range@2.1.1
19 silly currentTree │ ├── for-in@1.0.2
19 silly currentTree │ ├─┬ foreground-child@1.5.6
19 silly currentTree │ │ └── cross-spawn@4.0.2
19 silly currentTree │ ├── fragment-cache@0.2.1
19 silly currentTree │ ├── fs-access@1.0.1
19 silly currentTree │ ├── fs-extra@8.1.0
19 silly currentTree │ ├── fs.realpath@1.0.0
19 silly currentTree │ ├── function-bind@1.1.1
19 silly currentTree │ ├── get-caller-file@2.0.5
19 silly currentTree │ ├─┬ get-pkg-repo@1.4.0
19 silly currentTree │ │ ├── camelcase-keys@2.1.0
19 silly currentTree │ │ ├── camelcase@2.1.1
19 silly currentTree │ │ ├── decamelize@1.2.0
19 silly currentTree │ │ ├── find-up@1.1.2
19 silly currentTree │ │ ├── get-stdin@4.0.1
19 silly currentTree │ │ ├── indent-string@2.1.0
19 silly currentTree │ │ ├── load-json-file@1.1.0
19 silly currentTree │ │ ├── map-obj@1.0.1
19 silly currentTree │ │ ├── meow@3.7.0
19 silly currentTree │ │ ├── parse-json@2.2.0
19 silly currentTree │ │ ├── path-exists@2.1.0
19 silly currentTree │ │ ├── path-type@1.1.0
19 silly currentTree │ │ ├── pify@2.3.0
19 silly currentTree │ │ ├── read-pkg-up@1.0.1
19 silly currentTree │ │ ├── read-pkg@1.1.0
19 silly currentTree │ │ ├── readable-stream@2.3.7
19 silly currentTree │ │ ├── redent@1.0.0
19 silly currentTree │ │ ├── string_decoder@1.1.1
19 silly currentTree │ │ ├── strip-bom@2.0.0
19 silly currentTree │ │ ├── strip-indent@1.0.1
19 silly currentTree │ │ ├── through2@2.0.5
19 silly currentTree │ │ └── trim-newlines@1.0.0
19 silly currentTree │ ├── get-stdin@5.0.1
19 silly currentTree │ ├── get-stream@4.1.0
19 silly currentTree │ ├── get-value@2.0.6
19 silly currentTree │ ├─┬ gh-pages@2.2.0
19 silly currentTree │ │ ├── array-union@1.0.2
19 silly currentTree │ │ ├── globby@6.1.0
19 silly currentTree │ │ └── pify@2.3.0
19 silly currentTree │ ├─┬ git-raw-commits@2.0.0
19 silly currentTree │ │ ├── load-json-file@4.0.0
19 silly currentTree │ │ ├── meow@4.0.1
19 silly currentTree │ │ ├── path-type@3.0.0
19 silly currentTree │ │ ├── pify@3.0.0
19 silly currentTree │ │ ├── read-pkg-up@3.0.0
19 silly currentTree │ │ ├── read-pkg@3.0.0
19 silly currentTree │ │ ├── readable-stream@2.3.7
19 silly currentTree │ │ ├── string_decoder@1.1.1
19 silly currentTree │ │ └── through2@2.0.5
19 silly currentTree │ ├─┬ git-remote-origin-url@2.0.0
19 silly currentTree │ │ └── pify@2.3.0
19 silly currentTree │ ├─┬ git-semver-tags@3.0.1
19 silly currentTree │ │ └── semver@6.3.0
19 silly currentTree │ ├── gitconfiglocal@1.0.0
19 silly currentTree │ ├── glob-parent@5.1.1
19 silly currentTree │ ├── glob@7.1.6
19 silly currentTree │ ├── global-dirs@2.0.1
19 silly currentTree │ ├── global-modules@1.0.0
19 silly currentTree │ ├── global-prefix@1.0.2
19 silly currentTree │ ├── globals@11.12.0
19 silly currentTree │ ├── globby@11.0.1
19 silly currentTree │ ├── got@9.6.0
19 silly currentTree │ ├── graceful-fs@4.2.4
19 silly currentTree │ ├── handlebars@4.7.6
19 silly currentTree │ ├─┬ has-ansi@2.0.0
19 silly currentTree │ │ └── ansi-regex@2.1.1
19 silly currentTree │ ├── has-flag@4.0.0
19 silly currentTree │ ├── has-symbols@1.0.1
19 silly currentTree │ ├── has-value@1.0.0
19 silly currentTree │ ├─┬ has-values@1.0.0
19 silly currentTree │ │ ├─┬ is-number@3.0.0
19 silly currentTree │ │ │ └── kind-of@3.2.2
19 silly currentTree │ │ └── kind-of@4.0.0
19 silly currentTree │ ├── has-yarn@2.1.0
19 silly currentTree │ ├── has@1.0.3
19 silly currentTree │ ├── hasha@3.0.0
19 silly currentTree │ ├── highlight.js@10.1.1
19 silly currentTree │ ├── homedir-polyfill@1.0.3
19 silly currentTree │ ├── hosted-git-info@2.8.8
19 silly currentTree │ ├── html-escaper@2.0.2
19 silly currentTree │ ├── http-cache-semantics@4.1.0
19 silly currentTree │ ├── http-proxy-agent@4.0.1
19 silly currentTree │ ├─┬ https-proxy-agent@4.0.0
19 silly currentTree │ │ └── agent-base@5.1.1
19 silly currentTree │ ├─┬ humanize-url@1.0.1
19 silly currentTree │ │ ├── normalize-url@1.9.1
19 silly currentTree │ │ └── prepend-http@1.0.4
19 silly currentTree │ ├── iconv-lite@0.4.24
19 silly currentTree │ ├── ignore-by-default@2.0.0
19 silly currentTree │ ├── ignore-walk@3.0.3
19 silly currentTree │ ├── ignore@5.1.8
19 silly currentTree │ ├─┬ import-fresh@3.2.1
19 silly currentTree │ │ └── resolve-from@4.0.0
19 silly currentTree │ ├── import-lazy@2.1.0
19 silly currentTree │ ├── import-local@3.0.2
19 silly currentTree │ ├── imurmurhash@0.1.4
19 silly currentTree │ ├── indent-string@4.0.0
19 silly currentTree │ ├── inflight@1.0.6
19 silly currentTree │ ├── inherits@2.0.4
19 silly currentTree │ ├── ini@1.3.5
19 silly currentTree │ ├─┬ inquirer@6.5.0
19 silly currentTree │ │ ├── ansi-regex@3.0.0
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├── chalk@2.4.2
19 silly currentTree │ │ ├── cli-cursor@2.1.0
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── figures@2.0.0
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ ├── is-fullwidth-code-point@2.0.0
19 silly currentTree │ │ ├── mimic-fn@1.2.0
19 silly currentTree │ │ ├── mute-stream@0.0.7
19 silly currentTree │ │ ├── onetime@2.0.1
19 silly currentTree │ │ ├── restore-cursor@2.0.0
19 silly currentTree │ │ ├─┬ string-width@2.1.1
19 silly currentTree │ │ │ └── strip-ansi@4.0.0
19 silly currentTree │ │ ├─┬ strip-ansi@5.2.0
19 silly currentTree │ │ │ └── ansi-regex@4.1.0
19 silly currentTree │ │ └── supports-color@5.5.0
19 silly currentTree │ ├── interpret@1.4.0
19 silly currentTree │ ├── irregular-plurals@3.2.0
19 silly currentTree │ ├─┬ is-accessor-descriptor@0.1.6
19 silly currentTree │ │ └── kind-of@3.2.2
19 silly currentTree │ ├── is-arrayish@0.2.1
19 silly currentTree │ ├── is-binary-path@2.1.0
19 silly currentTree │ ├── is-buffer@1.1.6
19 silly currentTree │ ├── is-callable@1.2.0
19 silly currentTree │ ├── is-ci@2.0.0
19 silly currentTree │ ├─┬ is-data-descriptor@0.1.4
19 silly currentTree │ │ └── kind-of@3.2.2
19 silly currentTree │ ├── is-date-object@1.0.2
19 silly currentTree │ ├─┬ is-descriptor@0.1.6
19 silly currentTree │ │ └── kind-of@5.1.0
19 silly currentTree │ ├── is-error@2.2.2
19 silly currentTree │ ├── is-extendable@0.1.1
19 silly currentTree │ ├── is-extglob@2.1.1
19 silly currentTree │ ├── is-finite@1.1.0
19 silly currentTree │ ├── is-fullwidth-code-point@3.0.0
19 silly currentTree │ ├── is-glob@4.0.1
19 silly currentTree │ ├── is-installed-globally@0.3.2
19 silly currentTree │ ├── is-interactive@1.0.0
19 silly currentTree │ ├── is-npm@4.0.0
19 silly currentTree │ ├── is-number@7.0.0
19 silly currentTree │ ├── is-obj@2.0.0
19 silly currentTree │ ├── is-path-cwd@2.2.0
19 silly currentTree │ ├── is-path-inside@3.0.2
19 silly currentTree │ ├── is-plain-obj@1.1.0
19 silly currentTree │ ├── is-plain-object@3.0.1
19 silly currentTree │ ├── is-promise@4.0.0
19 silly currentTree │ ├── is-redirect@1.0.0
19 silly currentTree │ ├── is-regex@1.1.0
19 silly currentTree │ ├── is-retry-allowed@1.2.0
19 silly currentTree │ ├── is-stream@1.1.0
19 silly currentTree │ ├── is-symbol@1.0.3
19 silly currentTree │ ├── is-text-path@1.0.1
19 silly currentTree │ ├── is-typedarray@1.0.0
19 silly currentTree │ ├── is-utf8@0.2.1
19 silly currentTree │ ├── is-windows@1.0.2
19 silly currentTree │ ├── is-yarn-global@0.3.0
19 silly currentTree │ ├── isarray@1.0.0
19 silly currentTree │ ├── isexe@2.0.0
19 silly currentTree │ ├── isobject@3.0.1
19 silly currentTree │ ├── istanbul-lib-coverage@2.0.5
19 silly currentTree │ ├── istanbul-lib-hook@2.0.7
19 silly currentTree │ ├─┬ istanbul-lib-instrument@3.3.0
19 silly currentTree │ │ └── semver@6.3.0
19 silly currentTree │ ├─┬ istanbul-lib-report@2.0.8
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ ├── make-dir@2.1.0
19 silly currentTree │ │ ├── semver@5.7.1
19 silly currentTree │ │ └── supports-color@6.1.0
19 silly currentTree │ ├─┬ istanbul-lib-source-maps@3.0.6
19 silly currentTree │ │ ├── make-dir@2.1.0
19 silly currentTree │ │ ├── rimraf@2.7.1
19 silly currentTree │ │ └── semver@5.7.1
19 silly currentTree │ ├── istanbul-reports@2.2.7
19 silly currentTree │ ├── js-string-escape@1.0.1
19 silly currentTree │ ├── js-tokens@4.0.0
19 silly currentTree │ ├── js-yaml@3.14.0
19 silly currentTree │ ├── jsesc@2.5.2
19 silly currentTree │ ├── json-buffer@3.0.0
19 silly currentTree │ ├── json-parse-better-errors@1.0.2
19 silly currentTree │ ├── json-stringify-safe@5.0.1
19 silly currentTree │ ├── jsonfile@4.0.0
19 silly currentTree │ ├── jsonparse@1.3.1
19 silly currentTree │ ├── JSONStream@1.3.5
19 silly currentTree │ ├── keyv@3.1.0
19 silly currentTree │ ├── kind-of@6.0.3
19 silly currentTree │ ├── klaw@1.3.1
19 silly currentTree │ ├── latest-version@5.1.0
19 silly currentTree │ ├── lazy-req@1.1.0
19 silly currentTree │ ├── lines-and-columns@1.1.6
19 silly currentTree │ ├── load-json-file@5.3.0
19 silly currentTree │ ├─┬ locate-path@2.0.0
19 silly currentTree │ │ └── path-exists@3.0.0
19 silly currentTree │ ├── lodash._reinterpolate@3.0.0
19 silly currentTree │ ├── lodash.flattendeep@4.4.0
19 silly currentTree │ ├── lodash.ismatch@4.4.0
19 silly currentTree │ ├── lodash.map@4.6.0
19 silly currentTree │ ├── lodash.template@4.5.0
19 silly currentTree │ ├── lodash.templatesettings@4.2.0
19 silly currentTree │ ├── lodash.toarray@4.4.0
19 silly currentTree │ ├── lodash@4.17.19
19 silly currentTree │ ├─┬ log-symbols@3.0.0
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├── chalk@2.4.2
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ └── supports-color@5.5.0
19 silly currentTree │ ├─┬ log-update@1.0.2
19 silly currentTree │ │ ├── ansi-escapes@1.4.0
19 silly currentTree │ │ ├── cli-cursor@1.0.2
19 silly currentTree │ │ ├── onetime@1.1.0
19 silly currentTree │ │ └── restore-cursor@1.0.1
19 silly currentTree │ ├── longest@2.0.1
19 silly currentTree │ ├── loud-rejection@1.6.0
19 silly currentTree │ ├── lowercase-keys@1.0.1
19 silly currentTree │ ├── lru-cache@4.1.5
19 silly currentTree │ ├── lunr@2.3.8
19 silly currentTree │ ├─┬ make-dir@3.1.0
19 silly currentTree │ │ └── semver@6.3.0
19 silly currentTree │ ├── map-age-cleaner@0.1.3
19 silly currentTree │ ├── map-cache@0.2.2
19 silly currentTree │ ├── map-obj@2.0.0
19 silly currentTree │ ├── map-visit@1.0.0
19 silly currentTree │ ├── marked@1.0.0
19 silly currentTree │ ├─┬ matcher@3.0.0
19 silly currentTree │ │ └── escape-string-regexp@4.0.0
19 silly currentTree │ ├── md5-hex@3.0.1
19 silly currentTree │ ├─┬ mem@6.1.0
19 silly currentTree │ │ └── mimic-fn@3.0.0
19 silly currentTree │ ├── memorystream@0.3.1
19 silly currentTree │ ├─┬ meow@5.0.0
19 silly currentTree │ │ ├── load-json-file@4.0.0
19 silly currentTree │ │ ├── path-type@3.0.0
19 silly currentTree │ │ ├── pify@3.0.0
19 silly currentTree │ │ ├── read-pkg-up@3.0.0
19 silly currentTree │ │ └── read-pkg@3.0.0
19 silly currentTree │ ├── merge-source-map@1.1.0
19 silly currentTree │ ├── merge@1.2.1
19 silly currentTree │ ├── merge2@1.4.1
19 silly currentTree │ ├── micromatch@4.0.2
19 silly currentTree │ ├── mimic-fn@2.1.0
19 silly currentTree │ ├── mimic-response@1.0.1
19 silly currentTree │ ├── minimatch@3.0.4
19 silly currentTree │ ├─┬ minimist-options@3.0.2
19 silly currentTree │ │ └── arrify@1.0.1
19 silly currentTree │ ├── minimist@1.2.5
19 silly currentTree │ ├─┬ mixin-deep@1.3.2
19 silly currentTree │ │ ├── is-extendable@1.0.1
19 silly currentTree │ │ └── is-plain-object@2.0.4
19 silly currentTree │ ├── mkdirp@0.5.5
19 silly currentTree │ ├── modify-values@1.0.1
19 silly currentTree │ ├─┬ mount-point@3.0.0
19 silly currentTree │ │ ├── @sindresorhus/df@1.0.1
19 silly currentTree │ │ └── pify@2.3.0
19 silly currentTree │ ├── ms@2.1.2
19 silly currentTree │ ├── mute-stream@0.0.8
19 silly currentTree │ ├── nanomatch@1.2.13
19 silly currentTree │ ├── neo-async@2.6.1
19 silly currentTree │ ├── nested-error-stacks@2.1.0
19 silly currentTree │ ├── nice-try@1.0.5
19 silly currentTree │ ├── node-emoji@1.8.1
19 silly currentTree │ ├── node-fetch@2.6.0
19 silly currentTree │ ├── node-status-codes@1.0.0
19 silly currentTree │ ├─┬ normalize-package-data@2.5.0
19 silly currentTree │ │ └── semver@5.7.1
19 silly currentTree │ ├── normalize-path@3.0.0
19 silly currentTree │ ├── normalize-url@4.5.0
19 silly currentTree │ ├─┬ npm-run-all@4.1.5
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├── chalk@2.4.2
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ ├── load-json-file@4.0.0
19 silly currentTree │ │ ├── path-type@3.0.0
19 silly currentTree │ │ ├── pify@3.0.0
19 silly currentTree │ │ ├── read-pkg@3.0.0
19 silly currentTree │ │ └── supports-color@5.5.0
19 silly currentTree │ ├── npm-run-path@2.0.2
19 silly currentTree │ ├── null-check@1.0.0
19 silly currentTree │ ├── number-is-nan@1.0.1
19 silly currentTree │ ├─┬ nyc@14.1.1
19 silly currentTree │ │ ├── ansi-regex@4.1.0
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├── camelcase@5.3.1
19 silly currentTree │ │ ├── cliui@5.0.0
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── decamelize@1.2.0
19 silly currentTree │ │ ├── emoji-regex@7.0.3
19 silly currentTree │ │ ├── find-up@3.0.0
19 silly currentTree │ │ ├── is-fullwidth-code-point@2.0.0
19 silly currentTree │ │ ├── locate-path@3.0.0
19 silly currentTree │ │ ├── make-dir@2.1.0
19 silly currentTree │ │ ├── p-limit@2.3.0
19 silly currentTree │ │ ├── p-locate@3.0.0
19 silly currentTree │ │ ├── path-exists@3.0.0
19 silly currentTree │ │ ├── resolve-from@4.0.0
19 silly currentTree │ │ ├── rimraf@2.7.1
19 silly currentTree │ │ ├── semver@5.7.1
19 silly currentTree │ │ ├── string-width@3.1.0
19 silly currentTree │ │ ├── strip-ansi@5.2.0
19 silly currentTree │ │ ├── wrap-ansi@5.1.0
19 silly currentTree │ │ ├── yargs-parser@13.1.2
19 silly currentTree │ │ └── yargs@13.3.2
19 silly currentTree │ ├── object-assign@4.1.1
19 silly currentTree │ ├─┬ object-copy@0.1.0
19 silly currentTree │ │ ├── define-property@0.2.5
19 silly currentTree │ │ └── kind-of@3.2.2
19 silly currentTree │ ├── object-inspect@1.8.0
19 silly currentTree │ ├── object-keys@1.1.1
19 silly currentTree │ ├── object-visit@1.0.1
19 silly currentTree │ ├── object.assign@4.1.0
19 silly currentTree │ ├── object.pick@1.3.0
19 silly currentTree │ ├── once@1.4.0
19 silly currentTree │ ├── onetime@5.1.0
19 silly currentTree │ ├─┬ opn-cli@3.1.0
19 silly currentTree │ │ ├── camelcase-keys@2.1.0
19 silly currentTree │ │ ├── camelcase@2.1.1
19 silly currentTree │ │ ├── decamelize@1.2.0
19 silly currentTree │ │ ├── find-up@1.1.2
19 silly currentTree │ │ ├── indent-string@2.1.0
19 silly currentTree │ │ ├── load-json-file@1.1.0
19 silly currentTree │ │ ├── map-obj@1.0.1
19 silly currentTree │ │ ├── meow@3.7.0
19 silly currentTree │ │ ├── parse-json@2.2.0
19 silly currentTree │ │ ├── path-exists@2.1.0
19 silly currentTree │ │ ├── path-type@1.1.0
19 silly currentTree │ │ ├── pify@2.3.0
19 silly currentTree │ │ ├── read-pkg-up@1.0.1
19 silly currentTree │ │ ├── read-pkg@1.1.0
19 silly currentTree │ │ ├── redent@1.0.0
19 silly currentTree │ │ ├── strip-bom@2.0.0
19 silly currentTree │ │ ├─┬ strip-indent@1.0.1
19 silly currentTree │ │ │ └── get-stdin@4.0.1
19 silly currentTree │ │ └── trim-newlines@1.0.0
19 silly currentTree │ ├── opn@4.0.2
19 silly currentTree │ ├─┬ ora@4.0.4
19 silly currentTree │ │ └── chalk@3.0.0
19 silly currentTree │ ├── os-family@1.1.0
19 silly currentTree │ ├── os-homedir@1.0.2
19 silly currentTree │ ├── os-tmpdir@1.0.2
19 silly currentTree │ ├── osenv@0.1.5
19 silly currentTree │ ├── p-cancelable@1.1.0
19 silly currentTree │ ├── p-defer@1.0.0
19 silly currentTree │ ├── p-finally@1.0.0
19 silly currentTree │ ├─┬ p-limit@1.3.0
19 silly currentTree │ │ └── p-try@1.0.0
19 silly currentTree │ ├── p-locate@2.0.0
19 silly currentTree │ ├── p-map@4.0.0
19 silly currentTree │ ├── p-try@2.2.0
19 silly currentTree │ ├── package-hash@3.0.0
19 silly currentTree │ ├─┬ package-json@6.5.0
19 silly currentTree │ │ └── semver@6.3.0
19 silly currentTree │ ├── parent-module@1.0.1
19 silly currentTree │ ├── parse-github-repo-url@1.4.1
19 silly currentTree │ ├── parse-json@4.0.0
19 silly currentTree │ ├── parse-ms@2.1.0
19 silly currentTree │ ├── parse-passwd@1.0.0
19 silly currentTree │ ├── pascalcase@0.1.1
19 silly currentTree │ ├── path-exists@4.0.0
19 silly currentTree │ ├── path-is-absolute@1.0.1
19 silly currentTree │ ├── path-key@2.0.1
19 silly currentTree │ ├── path-parse@1.0.6
19 silly currentTree │ ├── path-type@4.0.0
19 silly currentTree │ ├── picomatch@2.2.2
19 silly currentTree │ ├── pidtree@0.3.1
19 silly currentTree │ ├── pify@4.0.1
19 silly currentTree │ ├── pinkie-promise@2.0.1
19 silly currentTree │ ├── pinkie@2.0.4
19 silly currentTree │ ├─┬ pkg-conf@3.1.0
19 silly currentTree │ │ ├── find-up@3.0.0
19 silly currentTree │ │ ├── locate-path@3.0.0
19 silly currentTree │ │ ├── p-limit@2.3.0
19 silly currentTree │ │ ├── p-locate@3.0.0
19 silly currentTree │ │ └── path-exists@3.0.0
19 silly currentTree │ ├─┬ pkg-dir@4.2.0
19 silly currentTree │ │ ├── find-up@4.1.0
19 silly currentTree │ │ ├── locate-path@5.0.0
19 silly currentTree │ │ ├── p-limit@2.3.0
19 silly currentTree │ │ └── p-locate@4.1.0
19 silly currentTree │ ├── plur@4.0.0
19 silly currentTree │ ├── posix-character-classes@0.1.1
19 silly currentTree │ ├── prepend-http@2.0.0
19 silly currentTree │ ├── prettier@1.19.1
19 silly currentTree │ ├── pretty-ms@7.0.0
19 silly currentTree │ ├── process-nextick-args@2.0.1
19 silly currentTree │ ├── progress@2.0.3
19 silly currentTree │ ├── promisify-event@1.0.0
19 silly currentTree │ ├── pseudomap@1.0.2
19 silly currentTree │ ├─┬ publish-please@5.5.1
19 silly currentTree │ │ ├── ansi-regex@3.0.0
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├─┬ braces@2.3.2
19 silly currentTree │ │ │ └── extend-shallow@2.0.1
19 silly currentTree │ │ ├── chalk@2.4.1
19 silly currentTree │ │ ├── ci-info@1.6.0
19 silly currentTree │ │ ├── cli-cursor@2.1.0
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── figures@2.0.0
19 silly currentTree │ │ ├─┬ fill-range@4.0.0
19 silly currentTree │ │ │ └── extend-shallow@2.0.1
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ ├── inquirer@6.2.0
19 silly currentTree │ │ ├── is-ci@1.2.1
19 silly currentTree │ │ ├── is-fullwidth-code-point@2.0.0
19 silly currentTree │ │ ├─┬ is-number@3.0.0
19 silly currentTree │ │ │ └── kind-of@3.2.2
19 silly currentTree │ │ ├── lodash@4.17.12
19 silly currentTree │ │ ├── micromatch@3.1.10
19 silly currentTree │ │ ├── mimic-fn@1.2.0
19 silly currentTree │ │ ├── mute-stream@0.0.7
19 silly currentTree │ │ ├── onetime@2.0.1
19 silly currentTree │ │ ├── restore-cursor@2.0.0
19 silly currentTree │ │ ├── semver@5.6.0
19 silly currentTree │ │ ├── string-width@2.1.1
19 silly currentTree │ │ ├── strip-ansi@4.0.0
19 silly currentTree │ │ ├── supports-color@5.5.0
19 silly currentTree │ │ └── to-regex-range@2.1.1
19 silly currentTree │ ├── pump@3.0.0
19 silly currentTree │ ├── pupa@2.0.1
19 silly currentTree │ ├── q@1.5.1
19 silly currentTree │ ├── query-string@4.3.4
19 silly currentTree │ ├── quick-lru@1.1.0
19 silly currentTree │ ├── rc@1.2.8
19 silly currentTree │ ├─┬ read-all-stream@3.1.0
19 silly currentTree │ │ ├── readable-stream@2.3.7
19 silly currentTree │ │ └── string_decoder@1.1.1
19 silly currentTree │ ├─┬ read-pkg-up@4.0.0
19 silly currentTree │ │ ├── find-up@3.0.0
19 silly currentTree │ │ ├── load-json-file@4.0.0
19 silly currentTree │ │ ├── locate-path@3.0.0
19 silly currentTree │ │ ├── p-limit@2.3.0
19 silly currentTree │ │ ├── p-locate@3.0.0
19 silly currentTree │ │ ├── path-exists@3.0.0
19 silly currentTree │ │ ├── path-type@3.0.0
19 silly currentTree │ │ ├── pify@3.0.0
19 silly currentTree │ │ └── read-pkg@3.0.0
19 silly currentTree │ ├─┬ read-pkg@5.2.0
19 silly currentTree │ │ ├── parse-json@5.0.0
19 silly currentTree │ │ └── type-fest@0.6.0
19 silly currentTree │ ├── readable-stream@3.6.0
19 silly currentTree │ ├── readdirp@3.4.0
19 silly currentTree │ ├── rechoir@0.6.2
19 silly currentTree │ ├─┬ redent@2.0.0
19 silly currentTree │ │ └── indent-string@3.2.0
19 silly currentTree │ ├── regenerator-runtime@0.13.5
19 silly currentTree │ ├── regex-not@1.0.2
19 silly currentTree │ ├── registry-auth-token@4.1.1
19 silly currentTree │ ├── registry-url@5.1.0
19 silly currentTree │ ├── release-zalgo@1.0.0
19 silly currentTree │ ├── repeat-element@1.1.3
19 silly currentTree │ ├── repeat-string@1.6.1
19 silly currentTree │ ├── repeating@2.0.1
19 silly currentTree │ ├── require-directory@2.1.1
19 silly currentTree │ ├── require-main-filename@2.0.0
19 silly currentTree │ ├── resolve-cwd@3.0.0
19 silly currentTree │ ├── resolve-dir@1.0.1
19 silly currentTree │ ├── resolve-from@5.0.0
19 silly currentTree │ ├─┬ resolve-global@1.0.0
19 silly currentTree │ │ └── global-dirs@0.1.1
19 silly currentTree │ ├── resolve-url@0.2.1
19 silly currentTree │ ├── resolve@1.17.0
19 silly currentTree │ ├── responselike@1.0.2
19 silly currentTree │ ├── restore-cursor@3.1.0
19 silly currentTree │ ├── ret@0.1.15
19 silly currentTree │ ├── reusify@1.0.4
19 silly currentTree │ ├── right-pad@1.0.1
19 silly currentTree │ ├── rimraf@3.0.2
19 silly currentTree │ ├── run-applescript@3.2.0
19 silly currentTree │ ├── run-async@2.4.1
19 silly currentTree │ ├── run-parallel@1.1.9
19 silly currentTree │ ├── rxjs@6.6.0
19 silly currentTree │ ├── safe-buffer@5.1.2
19 silly currentTree │ ├── safe-regex@1.1.0
19 silly currentTree │ ├── safer-buffer@2.1.2
19 silly currentTree │ ├─┬ semver-diff@3.1.1
19 silly currentTree │ │ └── semver@6.3.0
19 silly currentTree │ ├── semver@7.3.2
19 silly currentTree │ ├── serialize-error@2.1.0
19 silly currentTree │ ├── set-blocking@2.0.0
19 silly currentTree │ ├─┬ set-value@2.0.1
19 silly currentTree │ │ ├── extend-shallow@2.0.1
19 silly currentTree │ │ └── is-plain-object@2.0.4
19 silly currentTree │ ├── shebang-command@1.2.0
19 silly currentTree │ ├── shebang-regex@1.0.0
19 silly currentTree │ ├── shell-quote@1.7.2
19 silly currentTree │ ├── shelljs@0.8.4
19 silly currentTree │ ├── signal-exit@3.0.3
19 silly currentTree │ ├── slash@3.0.0
19 silly currentTree │ ├── slice-ansi@3.0.0
19 silly currentTree │ ├── slide@1.1.6
19 silly currentTree │ ├─┬ snapdragon-node@2.1.1
19 silly currentTree │ │ ├── define-property@1.0.0
19 silly currentTree │ │ ├── is-accessor-descriptor@1.0.0
19 silly currentTree │ │ ├── is-data-descriptor@1.0.0
19 silly currentTree │ │ └── is-descriptor@1.0.2
19 silly currentTree │ ├─┬ snapdragon-util@3.0.1
19 silly currentTree │ │ └── kind-of@3.2.2
19 silly currentTree │ ├─┬ snapdragon@0.8.2
19 silly currentTree │ │ ├── debug@2.6.9
19 silly currentTree │ │ ├── define-property@0.2.5
19 silly currentTree │ │ ├── extend-shallow@2.0.1
19 silly currentTree │ │ ├── ms@2.0.0
19 silly currentTree │ │ └── source-map@0.5.7
19 silly currentTree │ ├── sort-keys@1.1.2
19 silly currentTree │ ├── source-map-resolve@0.5.3
19 silly currentTree │ ├── source-map-support@0.5.19
19 silly currentTree │ ├── source-map-url@0.4.0
19 silly currentTree │ ├── source-map@0.6.1
19 silly currentTree │ ├─┬ spawn-wrap@1.4.3
19 silly currentTree │ │ └── rimraf@2.7.1
19 silly currentTree │ ├── spdx-correct@3.1.1
19 silly currentTree │ ├── spdx-exceptions@2.3.0
19 silly currentTree │ ├── spdx-expression-parse@3.0.1
19 silly currentTree │ ├── spdx-license-ids@3.0.5
19 silly currentTree │ ├── split-string@3.1.0
19 silly currentTree │ ├── split@1.0.1
19 silly currentTree │ ├─┬ split2@2.2.0
19 silly currentTree │ │ ├── readable-stream@2.3.7
19 silly currentTree │ │ ├── string_decoder@1.1.1
19 silly currentTree │ │ └── through2@2.0.5
19 silly currentTree │ ├── sprintf-js@1.0.3
19 silly currentTree │ ├─┬ stack-utils@2.0.2
19 silly currentTree │ │ └── escape-string-regexp@2.0.0
19 silly currentTree │ ├─┬ standard-version@8.0.0
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├── camelcase@5.3.1
19 silly currentTree │ │ ├── chalk@2.4.2
19 silly currentTree │ │ ├── cliui@6.0.0
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── conventional-changelog-config-spec@2.1.0
19 silly currentTree │ │ ├── conventional-changelog@3.1.18
19 silly currentTree │ │ ├── decamelize@1.2.0
19 silly currentTree │ │ ├── figures@3.1.0
19 silly currentTree │ │ ├── find-up@4.1.0
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ ├── locate-path@5.0.0
19 silly currentTree │ │ ├── p-limit@2.2.2
19 silly currentTree │ │ ├── p-locate@4.1.0
19 silly currentTree │ │ ├── p-try@2.2.0
19 silly currentTree │ │ ├── path-exists@4.0.0
19 silly currentTree │ │ ├─┬ read-pkg-up@7.0.1
19 silly currentTree │ │ │ └── type-fest@0.8.1
19 silly currentTree │ │ ├── semver@7.1.1
19 silly currentTree │ │ ├── strip-indent@3.0.0
19 silly currentTree │ │ ├── supports-color@5.5.0
19 silly currentTree │ │ ├─┬ wrap-ansi@6.2.0
19 silly currentTree │ │ │ ├── ansi-styles@4.2.1
19 silly currentTree │ │ │ ├── color-convert@2.0.1
19 silly currentTree │ │ │ └── color-name@1.1.4
19 silly currentTree │ │ ├── yargs-parser@18.1.3
19 silly currentTree │ │ └── yargs@15.3.1
19 silly currentTree │ ├─┬ static-extend@0.1.2
19 silly currentTree │ │ └── define-property@0.2.5
19 silly currentTree │ ├── stream-events@1.0.5
19 silly currentTree │ ├── strict-uri-encode@1.1.0
19 silly currentTree │ ├─┬ string_decoder@1.3.0
19 silly currentTree │ │ └── safe-buffer@5.2.1
19 silly currentTree │ ├── string-width@4.2.0
19 silly currentTree │ ├── string.prototype.padend@3.1.0
19 silly currentTree │ ├── string.prototype.trimend@1.0.1
19 silly currentTree │ ├── string.prototype.trimstart@1.0.1
19 silly currentTree │ ├── stringify-package@1.0.1
19 silly currentTree │ ├── strip-ansi@6.0.0
19 silly currentTree │ ├── strip-bom@3.0.0
19 silly currentTree │ ├── strip-eof@1.0.0
19 silly currentTree │ ├── strip-indent@2.0.0
19 silly currentTree │ ├── strip-json-comments@2.0.1
19 silly currentTree │ ├── strip-outer@1.0.1
19 silly currentTree │ ├── strip-url-auth@1.0.1
19 silly currentTree │ ├── stubs@3.0.0
19 silly currentTree │ ├─┬ supertap@1.0.0
19 silly currentTree │ │ ├── ansi-regex@3.0.0
19 silly currentTree │ │ ├── arrify@1.0.1
19 silly currentTree │ │ ├── indent-string@3.2.0
19 silly currentTree │ │ └── strip-ansi@4.0.0
19 silly currentTree │ ├── supports-color@7.1.0
19 silly currentTree │ ├── teeny-request@6.0.1
19 silly currentTree │ ├── temp-dir@2.0.0
19 silly currentTree │ ├─┬ temp-write@2.1.0
19 silly currentTree │ │ ├── pify@2.3.0
19 silly currentTree │ │ └── uuid@2.0.3
19 silly currentTree │ ├── term-size@2.2.0
19 silly currentTree │ ├── test-exclude@5.2.3
19 silly currentTree │ ├── text-extensions@1.9.0
19 silly currentTree │ ├── through@2.3.8
19 silly currentTree │ ├── through2@3.0.2
19 silly currentTree │ ├── time-zone@1.0.0
19 silly currentTree │ ├── timed-out@3.1.3
19 silly currentTree │ ├── tmp@0.0.33
19 silly currentTree │ ├── to-fast-properties@2.0.0
19 silly currentTree │ ├─┬ to-object-path@0.3.0
19 silly currentTree │ │ └── kind-of@3.2.2
19 silly currentTree │ ├── to-readable-stream@1.0.0
19 silly currentTree │ ├── to-regex-range@5.0.1
19 silly currentTree │ ├── to-regex@3.0.2
19 silly currentTree │ ├─┬ trash-cli@1.4.0
19 silly currentTree │ │ ├── ansi-align@1.1.0
19 silly currentTree │ │ ├── ansi-regex@2.1.1
19 silly currentTree │ │ ├── ansi-styles@2.2.1
19 silly currentTree │ │ ├── boxen@0.6.0
19 silly currentTree │ │ ├── camelcase-keys@2.1.0
19 silly currentTree │ │ ├── camelcase@2.1.1
19 silly currentTree │ │ ├── chalk@1.1.3
19 silly currentTree │ │ ├── cli-boxes@1.0.0
19 silly currentTree │ │ ├── configstore@2.1.0
19 silly currentTree │ │ ├── decamelize@1.2.0
19 silly currentTree │ │ ├── dot-prop@3.0.0
19 silly currentTree │ │ ├── find-up@1.1.2
19 silly currentTree │ │ ├── get-stdin@4.0.1
19 silly currentTree │ │ ├── got@5.7.1
19 silly currentTree │ │ ├── indent-string@2.1.0
19 silly currentTree │ │ ├── is-fullwidth-code-point@1.0.0
19 silly currentTree │ │ ├── is-npm@1.0.0
19 silly currentTree │ │ ├── is-obj@1.0.1
19 silly currentTree │ │ ├── latest-version@2.0.0
19 silly currentTree │ │ ├── load-json-file@1.1.0
19 silly currentTree │ │ ├── map-obj@1.0.1
19 silly currentTree │ │ ├── meow@3.7.0
19 silly currentTree │ │ ├── package-json@2.4.0
19 silly currentTree │ │ ├── parse-json@2.2.0
19 silly currentTree │ │ ├── path-exists@2.1.0
19 silly currentTree │ │ ├── path-type@1.1.0
19 silly currentTree │ │ ├── pify@2.3.0
19 silly currentTree │ │ ├── prepend-http@1.0.4
19 silly currentTree │ │ ├── read-pkg-up@1.0.1
19 silly currentTree │ │ ├── read-pkg@1.1.0
19 silly currentTree │ │ ├── readable-stream@2.3.7
19 silly currentTree │ │ ├── redent@1.0.0
19 silly currentTree │ │ ├── registry-auth-token@3.4.0
19 silly currentTree │ │ ├── registry-url@3.1.0
19 silly currentTree │ │ ├── semver-diff@2.1.0
19 silly currentTree │ │ ├── semver@5.7.1
19 silly currentTree │ │ ├── string_decoder@1.1.1
19 silly currentTree │ │ ├── string-width@1.0.2
19 silly currentTree │ │ ├── strip-ansi@3.0.1
19 silly currentTree │ │ ├── strip-bom@2.0.0
19 silly currentTree │ │ ├── strip-indent@1.0.1
19 silly currentTree │ │ ├── supports-color@2.0.0
19 silly currentTree │ │ ├── trim-newlines@1.0.0
19 silly currentTree │ │ ├── update-notifier@1.0.3
19 silly currentTree │ │ ├── url-parse-lax@1.0.0
19 silly currentTree │ │ ├── uuid@2.0.3
19 silly currentTree │ │ ├── widest-line@1.0.0
19 silly currentTree │ │ ├── write-file-atomic@1.3.4
19 silly currentTree │ │ └── xdg-basedir@2.0.0
19 silly currentTree │ ├─┬ trash@4.3.0
19 silly currentTree │ │ ├── array-union@1.0.2
19 silly currentTree │ │ ├── dir-glob@2.2.2
19 silly currentTree │ │ ├── fs-extra@0.30.0
19 silly currentTree │ │ ├── globby@7.1.1
19 silly currentTree │ │ ├── ignore@3.3.10
19 silly currentTree │ │ ├── jsonfile@2.4.0
19 silly currentTree │ │ ├── p-map@1.2.0
19 silly currentTree │ │ ├── p-try@1.0.0
19 silly currentTree │ │ ├── path-type@3.0.0
19 silly currentTree │ │ ├── pify@3.0.0
19 silly currentTree │ │ ├── rimraf@2.7.1
19 silly currentTree │ │ └── slash@1.0.0
19 silly currentTree │ ├── trim-newlines@2.0.0
19 silly currentTree │ ├── trim-off-newlines@1.0.1
19 silly currentTree │ ├── trim-repeated@1.0.0
19 silly currentTree │ ├── tslib@1.13.0
19 silly currentTree │ ├── tslint-config-prettier@1.18.0
19 silly currentTree │ ├── tslint-immutable@4.9.1
19 silly currentTree │ ├─┬ tslint@5.20.1
19 silly currentTree │ │ ├── ansi-styles@3.2.1
19 silly currentTree │ │ ├── chalk@2.4.2
19 silly currentTree │ │ ├── color-convert@1.9.3
19 silly currentTree │ │ ├── color-name@1.1.3
19 silly currentTree │ │ ├── has-flag@3.0.0
19 silly currentTree │ │ ├── js-tokens@4.0.0
19 silly currentTree │ │ ├── semver@5.7.1
19 silly currentTree │ │ └── supports-color@5.5.0
19 silly currentTree │ ├── tsutils@2.29.0
19 silly currentTree │ ├── type-fest@0.3.1
19 silly currentTree │ ├── typedarray-to-buffer@3.1.5
19 silly currentTree │ ├── typedarray@0.0.6
19 silly currentTree │ ├── typedoc-default-themes@0.10.2
19 silly currentTree │ ├── typedoc@0.17.8
19 silly currentTree │ ├── typescript@3.9.6
19 silly currentTree │ ├── uglify-js@3.10.0
19 silly currentTree │ ├── union-value@1.0.1
19 silly currentTree │ ├── unique-string@2.0.0
19 silly currentTree │ ├── universalify@0.1.2
19 silly currentTree │ ├─┬ unset-value@1.0.0
19 silly currentTree │ │ ├─┬ has-value@0.3.1
19 silly currentTree │ │ │ └── isobject@2.1.0
19 silly currentTree │ │ └── has-values@0.1.4
19 silly currentTree │ ├── unzip-response@1.0.2
19 silly currentTree │ ├─┬ update-notifier@4.1.0
19 silly currentTree │ │ └── chalk@3.0.0
19 silly currentTree │ ├── urix@0.1.0
19 silly currentTree │ ├── url-parse-lax@3.0.0
19 silly currentTree │ ├── urlgrey@0.4.4
19 silly currentTree │ ├── use@3.1.1
19 silly currentTree │ ├── user-home@2.0.0
19 silly currentTree │ ├── util-deprecate@1.0.2
19 silly currentTree │ ├── uuid@3.4.0
19 silly currentTree │ ├── validate-npm-package-license@3.0.4
19 silly currentTree │ ├── wcwidth@1.0.1
19 silly currentTree │ ├── well-known-symbols@2.0.0
19 silly currentTree │ ├── which-module@2.0.0
19 silly currentTree │ ├── which@1.3.1
19 silly currentTree │ ├── widest-line@3.1.0
19 silly currentTree │ ├── word-wrap@1.2.3
19 silly currentTree │ ├── wordwrap@1.0.0
19 silly currentTree │ ├── wrap-ansi@6.2.0
19 silly currentTree │ ├── wrappy@1.0.2
19 silly currentTree │ ├── write-file-atomic@3.0.3
19 silly currentTree │ ├── xdg-basedir@4.0.0
19 silly currentTree │ ├─┬ xdg-trashdir@2.1.1
19 silly currentTree │ │ ├── pify@2.3.0
19 silly currentTree │ │ └── xdg-basedir@2.0.0
19 silly currentTree │ ├── xregexp@4.3.0
19 silly currentTree │ ├── xtend@4.0.1
19 silly currentTree │ ├── y18n@4.0.0
19 silly currentTree │ ├── yallist@2.1.2
19 silly currentTree │ ├── yaml@1.10.0
19 silly currentTree │ ├── yargs-parser@10.1.0
19 silly currentTree │ └─┬ yargs@15.4.0
19 silly currentTree │   ├── camelcase@5.3.1
19 silly currentTree │   ├── find-up@4.1.0
19 silly currentTree │   ├── locate-path@5.0.0
19 silly currentTree │   ├── p-limit@2.3.0
19 silly currentTree │   ├── p-locate@4.1.0
19 silly currentTree │   └─┬ yargs-parser@18.1.3
19 silly currentTree │     └── decamelize@1.2.0
19 silly currentTree ├── @sindresorhus/is@0.14.0
19 silly currentTree ├── @szmarczak/http-timer@1.1.2
19 silly currentTree ├── @types/color-name@1.1.1
19 silly currentTree ├── @types/eslint-visitor-keys@1.0.0
19 silly currentTree ├── @types/json-schema@7.0.5
19 silly currentTree ├── @types/json5@0.0.29
19 silly currentTree ├── @types/minimatch@3.0.3
19 silly currentTree ├── @types/node@14.0.19
19 silly currentTree ├─┬ @typescript-eslint/experimental-utils@2.34.0
19 silly currentTree │ ├── eslint-utils@2.0.0
19 silly currentTree │ └── eslint-visitor-keys@1.2.0
19 silly currentTree ├─┬ @typescript-eslint/parser@2.34.0
19 silly currentTree │ └── eslint-visitor-keys@1.2.0
19 silly currentTree ├─┬ @typescript-eslint/typescript-estree@2.34.0
19 silly currentTree │ ├── eslint-visitor-keys@1.2.0
19 silly currentTree │ ├── glob@7.1.6
19 silly currentTree │ ├── semver@7.3.2
19 silly currentTree │ └── tsutils@3.17.1
19 silly currentTree ├── @webassemblyjs/ast@1.9.0
19 silly currentTree ├── @webassemblyjs/floating-point-hex-parser@1.9.0
19 silly currentTree ├── @webassemblyjs/helper-api-error@1.9.0
19 silly currentTree ├── @webassemblyjs/helper-buffer@1.9.0
19 silly currentTree ├── @webassemblyjs/helper-code-frame@1.9.0
19 silly currentTree ├── @webassemblyjs/helper-fsm@1.9.0
19 silly currentTree ├── @webassemblyjs/helper-module-context@1.9.0
19 silly currentTree ├── @webassemblyjs/helper-wasm-bytecode@1.9.0
19 silly currentTree ├── @webassemblyjs/helper-wasm-section@1.9.0
19 silly currentTree ├── @webassemblyjs/ieee754@1.9.0
19 silly currentTree ├── @webassemblyjs/leb128@1.9.0
19 silly currentTree ├── @webassemblyjs/utf8@1.9.0
19 silly currentTree ├── @webassemblyjs/wasm-edit@1.9.0
19 silly currentTree ├── @webassemblyjs/wasm-gen@1.9.0
19 silly currentTree ├── @webassemblyjs/wasm-opt@1.9.0
19 silly currentTree ├── @webassemblyjs/wasm-parser@1.9.0
19 silly currentTree ├── @webassemblyjs/wast-parser@1.9.0
19 silly currentTree ├── @webassemblyjs/wast-printer@1.9.0
19 silly currentTree ├── @xtuc/ieee754@1.2.0
19 silly currentTree ├── @xtuc/long@4.2.2
19 silly currentTree ├── acorn-jsx@5.0.1
19 silly currentTree ├── acorn@7.3.1
19 silly currentTree ├── adbkit-logcat@1.1.0
19 silly currentTree ├── adbkit-monkey@1.0.1
19 silly currentTree ├─┬ adbkit@2.11.1
19 silly currentTree │ ├── debug@2.6.9
19 silly currentTree │ └── ms@2.0.0
19 silly currentTree ├─┬ addons-linter@1.26.0
19 silly currentTree │ ├── @babel/runtime@7.10.2
19 silly currentTree │ ├── acorn-jsx@5.2.0
19 silly currentTree │ ├── ajv@6.12.2
19 silly currentTree │ ├── ansi-styles@4.2.1
19 silly currentTree │ ├── chalk@4.1.0
19 silly currentTree │ ├── color-convert@2.0.1
19 silly currentTree │ ├── color-name@1.1.4
19 silly currentTree │ ├── eslint-visitor-keys@1.2.0
19 silly currentTree │ ├── espree@6.2.1
19 silly currentTree │ ├── fast-deep-equal@3.1.3
19 silly currentTree │ ├── glob@7.1.6
19 silly currentTree │ ├── has-flag@4.0.0
19 silly currentTree │ ├── semver@7.3.2
19 silly currentTree │ ├── source-map-support@0.5.19
19 silly currentTree │ ├── source-map@0.6.1
19 silly currentTree │ └── supports-color@7.1.0
19 silly currentTree ├── adm-zip@0.4.16
19 silly currentTree ├── ajv-errors@1.0.1
19 silly currentTree ├── ajv-merge-patch@4.1.0
19 silly currentTree ├─┬ ajv@6.12.0
19 silly currentTree │ └── fast-deep-equal@3.1.1
19 silly currentTree ├─┬ ansi-align@3.0.0
19 silly currentTree │ ├── ansi-regex@4.1.0
19 silly currentTree │ ├── string-width@3.1.0
19 silly currentTree │ └── strip-ansi@5.2.0
19 silly currentTree ├── ansi-regex@3.0.0
19 silly currentTree ├── ansi-styles@3.2.1
19 silly currentTree ├── any-promise@1.3.0
19 silly currentTree ├── anymatch@2.0.0
19 silly currentTree ├── aproba@1.2.0
19 silly currentTree ├── archiver-utils@1.3.0
19 silly currentTree ├─┬ archiver@2.1.1
19 silly currentTree │ └── async@2.6.3
19 silly currentTree ├── argparse@1.0.10
19 silly currentTree ├── arr-diff@4.0.0
19 silly currentTree ├── arr-flatten@1.1.0
19 silly currentTree ├── arr-union@3.1.0
19 silly currentTree ├── array-differ@3.0.0
19 silly currentTree ├── array-filter@0.0.1
19 silly currentTree ├─┬ array-includes@3.1.1
19 silly currentTree │ ├── es-abstract@1.17.6
19 silly currentTree │ ├── es-to-primitive@1.2.1
19 silly currentTree │ ├── has-symbols@1.0.1
19 silly currentTree │ ├── is-callable@1.2.0
19 silly currentTree │ ├── is-regex@1.1.0
19 silly currentTree │ └── object-keys@1.1.1
19 silly currentTree ├── array-map@0.0.0
19 silly currentTree ├── array-reduce@0.0.0
19 silly currentTree ├── array-union@2.1.0
19 silly currentTree ├── array-unique@0.3.2
19 silly currentTree ├─┬ array.prototype.flat@1.2.3
19 silly currentTree │ ├── es-abstract@1.17.6
19 silly currentTree │ ├── es-to-primitive@1.2.1
19 silly currentTree │ ├── has-symbols@1.0.1
19 silly currentTree │ ├── is-callable@1.2.0
19 silly currentTree │ ├── is-regex@1.1.0
19 silly currentTree │ └── object-keys@1.1.1
19 silly currentTree ├── arrify@2.0.1
19 silly currentTree ├── asn1.js@4.10.1
19 silly currentTree ├── asn1@0.2.4
19 silly currentTree ├── assert-plus@1.0.0
19 silly currentTree ├─┬ assert@1.5.0
19 silly currentTree │ ├── inherits@2.0.1
19 silly currentTree │ └── util@0.10.3
19 silly currentTree ├── assign-symbols@1.0.0
19 silly currentTree ├── astral-regex@1.0.0
19 silly currentTree ├── async-each@1.0.3
19 silly currentTree ├── async@0.2.10
19 silly currentTree ├── asynckit@0.4.0
19 silly currentTree ├── atob@2.1.2
19 silly currentTree ├── atomic-sleep@1.0.0
19 silly currentTree ├── aws-sign2@0.7.0
19 silly currentTree ├── aws4@1.10.0
19 silly currentTree ├── balanced-match@1.0.0
19 silly currentTree ├─┬ base@0.11.2
19 silly currentTree │ ├── define-property@1.0.0
19 silly currentTree │ ├── is-accessor-descriptor@1.0.0
19 silly currentTree │ ├── is-data-descriptor@1.0.0
19 silly currentTree │ └── is-descriptor@1.0.2
19 silly currentTree ├── base64-js@1.3.1
19 silly currentTree ├── bcrypt-pbkdf@1.0.2
19 silly currentTree ├── big.js@5.2.2
19 silly currentTree ├── binary-extensions@1.13.1
19 silly currentTree ├── bl@1.2.2
19 silly currentTree ├── bluebird@2.9.34
19 silly currentTree ├── bn.js@4.11.8
19 silly currentTree ├── boolbase@1.0.0
19 silly currentTree ├─┬ boxen@4.2.0
19 silly currentTree │ ├── ansi-regex@5.0.0
19 silly currentTree │ ├── ansi-styles@4.2.1
19 silly currentTree │ ├── chalk@3.0.0
19 silly currentTree │ ├── color-convert@2.0.1
19 silly currentTree │ ├── color-name@1.1.4
19 silly currentTree │ ├── emoji-regex@8.0.0
19 silly currentTree │ ├── has-flag@4.0.0
19 silly currentTree │ ├── is-fullwidth-code-point@3.0.0
19 silly currentTree │ ├── string-width@4.2.0
19 silly currentTree │ ├── strip-ansi@6.0.0
19 silly currentTree │ └── supports-color@7.1.0
19 silly currentTree ├── brace-expansion@1.1.11
19 silly currentTree ├─┬ braces@2.3.2
19 silly currentTree │ └── extend-shallow@2.0.1
19 silly currentTree ├── brorand@1.1.0
19 silly currentTree ├── browserify-aes@1.2.0
19 silly currentTree ├── browserify-cipher@1.0.1
19 silly currentTree ├── browserify-des@1.0.2
19 silly currentTree ├── browserify-rsa@4.0.1
19 silly currentTree ├── browserify-sign@4.0.4
19 silly currentTree ├── browserify-zlib@0.2.0
19 silly currentTree ├── buffer-alloc-unsafe@1.1.0
19 silly currentTree ├── buffer-alloc@1.2.0
19 silly currentTree ├── buffer-crc32@0.2.13
19 silly currentTree ├── buffer-equal-constant-time@1.0.1
19 silly currentTree ├── buffer-fill@1.0.0
19 silly currentTree ├── buffer-from@1.1.1
19 silly currentTree ├── buffer-xor@1.0.3
19 silly currentTree ├── buffer@5.6.0
19 silly currentTree ├── builtin-modules@1.1.1
19 silly currentTree ├── builtin-status-codes@3.0.0
19 silly currentTree ├── bunyan@1.8.12
19 silly currentTree ├─┬ cacache@12.0.4
19 silly currentTree │ ├── bluebird@3.7.2
19 silly currentTree │ └── glob@7.1.6
19 silly currentTree ├── cache-base@1.0.1
19 silly currentTree ├─┬ cacheable-request@6.1.0
19 silly currentTree │ └── lowercase-keys@2.0.0
19 silly currentTree ├── camelcase@5.3.1
19 silly currentTree ├── caseless@0.12.0
19 silly currentTree ├── chalk@2.4.1
19 silly currentTree ├── chardet@0.7.0
19 silly currentTree ├─┬ cheerio@1.0.0-rc.3
19 silly currentTree │ └── dom-serializer@0.1.1
19 silly currentTree ├─┬ chokidar@2.1.8
19 silly currentTree │ └── normalize-path@3.0.0
19 silly currentTree ├── chownr@1.1.4
19 silly currentTree ├─┬ chrome-launcher@0.13.3
19 silly currentTree │ └── rimraf@3.0.2
19 silly currentTree ├── chrome-trace-event@1.0.2
19 silly currentTree ├── ci-info@2.0.0
19 silly currentTree ├── cipher-base@1.0.4
19 silly currentTree ├─┬ class-utils@0.3.6
19 silly currentTree │ └── define-property@0.2.5
19 silly currentTree ├── cli-boxes@2.2.0
19 silly currentTree ├── cli-cursor@2.1.0
19 silly currentTree ├── cli-width@2.2.0
19 silly currentTree ├─┬ cliui@6.0.0
19 silly currentTree │ ├── ansi-regex@5.0.0
19 silly currentTree │ ├── emoji-regex@8.0.0
19 silly currentTree │ ├── is-fullwidth-code-point@3.0.0
19 silly currentTree │ ├── string-width@4.2.0
19 silly currentTree │ └── strip-ansi@6.0.0
19 silly currentTree ├── clone-response@1.0.2
19 silly currentTree ├── clone@1.0.4
19 silly currentTree ├── collection-visit@1.0.0
19 silly currentTree ├── color-convert@1.9.3
19 silly currentTree ├── color-name@1.1.3
19 silly currentTree ├── colors@0.5.1
19 silly currentTree ├─┬ columnify@1.5.4
19 silly currentTree │ ├── ansi-regex@2.1.1
19 silly currentTree │ └── strip-ansi@3.0.1
19 silly currentTree ├── combined-stream@1.0.8
19 silly currentTree ├── commander@2.20.3
19 silly currentTree ├── common-tags@1.8.0
19 silly currentTree ├── commondir@1.0.1
19 silly currentTree ├── component-emitter@1.2.1
19 silly currentTree ├── compress-commons@1.2.2
19 silly currentTree ├── concat-map@0.0.1
19 silly currentTree ├── concat-stream@1.6.2
19 silly currentTree ├── configstore@5.0.1
19 silly currentTree ├── console-browserify@1.2.0
19 silly currentTree ├── constants-browserify@1.0.0
19 silly currentTree ├── contains-path@0.1.0
19 silly currentTree ├── copy-concurrently@1.0.5
19 silly currentTree ├── copy-descriptor@0.1.1
19 silly currentTree ├── core-js-pure@3.6.5
19 silly currentTree ├── core-js@2.6.11
19 silly currentTree ├── core-util-is@1.0.2
19 silly currentTree ├── crc@3.8.0
19 silly currentTree ├── crc32-stream@2.0.0
19 silly currentTree ├── create-ecdh@4.0.3
19 silly currentTree ├── create-hash@1.2.0
19 silly currentTree ├── create-hmac@1.1.7
19 silly currentTree ├── cross-spawn@6.0.5
19 silly currentTree ├── crypto-browserify@3.12.0
19 silly currentTree ├── crypto-random-string@2.0.0
19 silly currentTree ├─┬ css-select@1.2.0
19 silly currentTree │ └── domutils@1.5.1
19 silly currentTree ├── css-what@2.1.3
19 silly currentTree ├── cyclist@1.0.1
19 silly currentTree ├── dashdash@1.14.1
19 silly currentTree ├── debounce@1.2.0
19 silly currentTree ├── debug@4.1.1
19 silly currentTree ├── decamelize@3.2.0
19 silly currentTree ├── decode-uri-component@0.2.0
19 silly currentTree ├── decompress-response@3.3.0
19 silly currentTree ├─┬ deep-equal@1.1.1
19 silly currentTree │ └── object-keys@1.1.1
19 silly currentTree ├── deep-extend@0.6.0
19 silly currentTree ├── deep-is@0.1.3
19 silly currentTree ├── deepcopy@2.0.0
19 silly currentTree ├── deepmerge@4.2.2
19 silly currentTree ├── defaults@1.0.3
19 silly currentTree ├── defer-to-connect@1.1.3
19 silly currentTree ├── define-properties@1.1.3
19 silly currentTree ├─┬ define-property@2.0.2
19 silly currentTree │ ├── is-accessor-descriptor@1.0.0
19 silly currentTree │ ├── is-data-descriptor@1.0.0
19 silly currentTree │ └── is-descriptor@1.0.2
19 silly currentTree ├── delayed-stream@1.0.0
19 silly currentTree ├── des.js@1.0.1
19 silly currentTree ├── detect-file@1.0.0
19 silly currentTree ├── diff@4.0.1
19 silly currentTree ├── diffie-hellman@5.0.3
19 silly currentTree ├─┬ dispensary@0.51.2
19 silly currentTree │ ├── async@3.2.0
19 silly currentTree │ └── pino@6.0.0
19 silly currentTree ├── doctrine@1.5.0
19 silly currentTree ├─┬ dom-serializer@0.1.0
19 silly currentTree │ └── domelementtype@1.1.3
19 silly currentTree ├── domain-browser@1.2.0
19 silly currentTree ├── domelementtype@1.3.1
19 silly currentTree ├── domhandler@2.4.2
19 silly currentTree ├── domutils@1.7.0
19 silly currentTree ├── dot-prop@5.2.0
19 silly currentTree ├── dtrace-provider@0.8.8
19 silly currentTree ├── duplexer3@0.1.4
19 silly currentTree ├── duplexify@3.7.1
19 silly currentTree ├── ecc-jsbn@0.1.2
19 silly currentTree ├── ecdsa-sig-formatter@1.0.11
19 silly currentTree ├── elliptic@6.5.3
19 silly currentTree ├── emoji-regex@7.0.3
19 silly currentTree ├── emojis-list@2.1.0
19 silly currentTree ├── end-of-stream@1.4.4
19 silly currentTree ├── enhanced-resolve@4.1.0
19 silly currentTree ├── entities@1.1.2
19 silly currentTree ├── errno@0.1.7
19 silly currentTree ├── error-ex@1.3.2
19 silly currentTree ├── es-abstract@1.13.0
19 silly currentTree ├── es-to-primitive@1.2.0
19 silly currentTree ├── es6-error@4.1.1
19 silly currentTree ├── es6-promise@2.3.0
19 silly currentTree ├── es6-promisify@6.1.1
19 silly currentTree ├── escape-string-regexp@1.0.5
19 silly currentTree ├─┬ eslint-import-resolver-node@0.3.4
19 silly currentTree │ ├── debug@2.6.9
19 silly currentTree │ └── ms@2.0.0
19 silly currentTree ├─┬ eslint-module-utils@2.6.0
19 silly currentTree │ ├── debug@2.6.9
19 silly currentTree │ └── ms@2.0.0
19 silly currentTree ├─┬ eslint-plugin-import@2.22.0
19 silly currentTree │ ├── debug@2.6.9
19 silly currentTree │ ├── ms@2.0.0
19 silly currentTree │ └── resolve@1.17.0
19 silly currentTree ├── eslint-plugin-json@1.4.0
19 silly currentTree ├── eslint-plugin-mozilla@0.14.0
19 silly currentTree ├── eslint-plugin-no-unsanitized@3.1.2
19 silly currentTree ├── eslint-scope@5.1.0
19 silly currentTree ├── eslint-utils@1.4.2
19 silly currentTree ├── eslint-visitor-keys@1.0.0
19 silly currentTree ├─┬ eslint@5.16.0
19 silly currentTree │ ├── ajv@6.10.0
19 silly currentTree │ ├── doctrine@3.0.0
19 silly currentTree │ ├── eslint-scope@4.0.3
19 silly currentTree │ ├── file-entry-cache@5.0.1
19 silly currentTree │ ├── flat-cache@2.0.1
19 silly currentTree │ ├── js-yaml@3.13.1
19 silly currentTree │ ├── mkdirp@0.5.5
19 silly currentTree │ └── write@1.0.3
19 silly currentTree ├─┬ espree@5.0.1
19 silly currentTree │ └── acorn@6.4.1
19 silly currentTree ├── esprima@4.0.1
19 silly currentTree ├── esquery@1.0.1
19 silly currentTree ├── esrecurse@4.2.1
19 silly currentTree ├── estraverse@4.2.0
19 silly currentTree ├── esutils@2.0.2
19 silly currentTree ├── event-to-promise@0.8.0
19 silly currentTree ├── events@3.1.0
19 silly currentTree ├── evp_bytestokey@1.0.3
19 silly currentTree ├─┬ execa@4.0.3
19 silly currentTree │ ├── cross-spawn@7.0.3
19 silly currentTree │ ├── mimic-fn@2.1.0
19 silly currentTree │ ├── onetime@5.1.0
19 silly currentTree │ ├── path-key@3.1.1
19 silly currentTree │ ├── shebang-command@2.0.0
19 silly currentTree │ ├── shebang-regex@3.0.0
19 silly currentTree │ └── which@2.0.2
19 silly currentTree ├─┬ expand-brackets@2.1.4
19 silly currentTree │ ├── debug@2.6.9
19 silly currentTree │ ├── define-property@0.2.5
19 silly currentTree │ ├── extend-shallow@2.0.1
19 silly currentTree │ └── ms@2.0.0
19 silly currentTree ├── expand-tilde@2.0.2
19 silly currentTree ├─┬ extend-shallow@3.0.2
19 silly currentTree │ └── is-extendable@1.0.1
19 silly currentTree ├── extend@3.0.2
19 silly currentTree ├── external-editor@3.0.3
19 silly currentTree ├─┬ extglob@2.0.4
19 silly currentTree │ ├── define-property@1.0.0
19 silly currentTree │ ├── extend-shallow@2.0.1
19 silly currentTree │ ├── is-accessor-descriptor@1.0.0
19 silly currentTree │ ├── is-data-descriptor@1.0.0
19 silly currentTree │ └── is-descriptor@1.0.2
19 silly currentTree ├── extsprintf@1.3.0
19 silly currentTree ├── fast-deep-equal@2.0.1
19 silly currentTree ├── fast-json-patch@2.2.1
19 silly currentTree ├── fast-json-stable-stringify@2.0.0
19 silly currentTree ├── fast-levenshtein@2.0.6
19 silly currentTree ├── fast-redact@2.0.0
19 silly currentTree ├── fast-safe-stringify@2.0.7
19 silly currentTree ├── fd-slicer@1.1.0
19 silly currentTree ├── figgy-pudding@3.5.2
19 silly currentTree ├── figures@2.0.0
19 silly currentTree ├─┬ fill-range@4.0.0
19 silly currentTree │ └── extend-shallow@2.0.1
19 silly currentTree ├─┬ find-cache-dir@2.1.0
19 silly currentTree │ ├── find-up@3.0.0
19 silly currentTree │ ├── locate-path@3.0.0
19 silly currentTree │ ├── make-dir@2.1.0
19 silly currentTree │ ├── p-limit@2.3.0
19 silly currentTree │ ├── p-locate@3.0.0
19 silly currentTree │ ├── p-try@2.2.0
19 silly currentTree │ ├── pify@4.0.1
19 silly currentTree │ └── pkg-dir@3.0.0
19 silly currentTree ├── find-up@2.1.0
19 silly currentTree ├── findup-sync@3.0.0
19 silly currentTree ├─┬ firefox-profile@1.3.1
19 silly currentTree │ └── async@2.5.0
19 silly currentTree ├── first-chunk-stream@3.0.0
19 silly currentTree ├── flatstr@1.0.12
19 silly currentTree ├── flatted@2.0.0
19 silly currentTree ├── fluent-syntax@0.13.0
19 silly currentTree ├── flush-write-stream@1.1.1
19 silly currentTree ├── for-in@1.0.2
19 silly currentTree ├── forever-agent@0.6.1
19 silly currentTree ├── form-data@2.3.3
19 silly currentTree ├── fragment-cache@0.2.1
19 silly currentTree ├── from2@2.3.0
19 silly currentTree ├── fs-constants@1.0.0
19 silly currentTree ├── fs-extra@4.0.3
19 silly currentTree ├── fs-write-stream-atomic@1.0.10
19 silly currentTree ├── fs.realpath@1.0.0
19 silly currentTree ├── function-bind@1.1.1
19 silly currentTree ├── functional-red-black-tree@1.0.1
19 silly currentTree ├─┬ fx-runner@1.0.12
19 silly currentTree │ ├── commander@2.9.0
19 silly currentTree │ ├── isexe@1.1.2
19 silly currentTree │ └── which@1.2.4
19 silly currentTree ├── get-caller-file@2.0.5
19 silly currentTree ├── get-stream@5.1.0
19 silly currentTree ├── get-value@2.0.6
19 silly currentTree ├── getpass@0.1.7
19 silly currentTree ├── git-rev-sync@2.0.0
19 silly currentTree ├─┬ glob-parent@3.1.0
19 silly currentTree │ └── is-glob@3.1.0
19 silly currentTree ├── glob@7.1.3
19 silly currentTree ├── global-dirs@2.0.1
19 silly currentTree ├─┬ global-modules@2.0.0
19 silly currentTree │ └── global-prefix@3.0.0
19 silly currentTree ├── global-prefix@1.0.2
19 silly currentTree ├── globals@11.9.0
19 silly currentTree ├─┬ got@9.6.0
19 silly currentTree │ └── get-stream@4.1.0
19 silly currentTree ├── graceful-fs@4.1.15
19 silly currentTree ├── graceful-readlink@1.0.1
19 silly currentTree ├── growly@1.3.0
19 silly currentTree ├── har-schema@2.0.0
19 silly currentTree ├── har-validator@5.1.3
19 silly currentTree ├── has-flag@3.0.0
19 silly currentTree ├── has-symbols@1.0.0
19 silly currentTree ├── has-value@1.0.0
19 silly currentTree ├─┬ has-values@1.0.0
19 silly currentTree │ └── kind-of@4.0.0
19 silly currentTree ├── has-yarn@2.1.0
19 silly currentTree ├── has@1.0.3
19 silly currentTree ├── hash-base@3.0.4
19 silly currentTree ├── hash.js@1.1.7
19 silly currentTree ├── hmac-drbg@1.0.1
19 silly currentTree ├── homedir-polyfill@1.0.3
19 silly currentTree ├── hosted-git-info@2.7.1
19 silly currentTree ├── htmlparser2@3.9.2
19 silly currentTree ├── http-cache-semantics@4.1.0
19 silly currentTree ├── http-signature@1.2.0
19 silly currentTree ├── https-browserify@1.0.0
19 silly currentTree ├── human-signals@1.1.1
19 silly currentTree ├── iconv-lite@0.4.24
19 silly currentTree ├── ieee754@1.1.13
19 silly currentTree ├── iferr@0.1.5
19 silly currentTree ├── ignore@4.0.6
19 silly currentTree ├─┬ import-fresh@3.0.0
19 silly currentTree │ └── resolve-from@4.0.0
19 silly currentTree ├── import-lazy@2.1.0
19 silly currentTree ├─┬ import-local@2.0.0
19 silly currentTree │ ├── find-up@3.0.0
19 silly currentTree │ ├── locate-path@3.0.0
19 silly currentTree │ ├── p-limit@2.3.0
19 silly currentTree │ ├── p-locate@3.0.0
19 silly currentTree │ ├── p-try@2.2.0
19 silly currentTree │ └── pkg-dir@3.0.0
19 silly currentTree ├── imurmurhash@0.1.4
19 silly currentTree ├── infer-owner@1.0.4
19 silly currentTree ├── inflight@1.0.6
19 silly currentTree ├── inherits@2.0.3
19 silly currentTree ├── ini-parser@0.0.2
19 silly currentTree ├── ini@1.3.5
19 silly currentTree ├─┬ inquirer@6.3.1
19 silly currentTree │ ├── ansi-escapes@3.2.0
19 silly currentTree │ ├── ansi-regex@4.1.0
19 silly currentTree │ ├── chalk@2.4.2
19 silly currentTree │ └── strip-ansi@5.2.0
19 silly currentTree ├── interpret@1.4.0
19 silly currentTree ├── invert-kv@3.0.1
19 silly currentTree ├── is-absolute@0.1.7
19 silly currentTree ├─┬ is-accessor-descriptor@0.1.6
19 silly currentTree │ └── kind-of@3.2.2
19 silly currentTree ├── is-arguments@1.0.4
19 silly currentTree ├── is-arrayish@0.2.1
19 silly currentTree ├── is-binary-path@1.0.1
19 silly currentTree ├── is-buffer@1.1.6
19 silly currentTree ├── is-builtin-module@1.0.0
19 silly currentTree ├── is-callable@1.1.4
19 silly currentTree ├── is-ci@2.0.0
19 silly currentTree ├─┬ is-data-descriptor@0.1.4
19 silly currentTree │ └── kind-of@3.2.2
19 silly currentTree ├── is-date-object@1.0.1
19 silly currentTree ├─┬ is-descriptor@0.1.6
19 silly currentTree │ └── kind-of@5.1.0
19 silly currentTree ├── is-docker@2.0.0
19 silly currentTree ├── is-extendable@0.1.1
19 silly currentTree ├── is-extglob@2.1.1
19 silly currentTree ├── is-fullwidth-code-point@2.0.0
19 silly currentTree ├── is-glob@4.0.1
19 silly currentTree ├── is-installed-globally@0.3.2
19 silly currentTree ├── is-mergeable-object@1.1.1
19 silly currentTree ├── is-npm@4.0.0
19 silly currentTree ├─┬ is-number@3.0.0
19 silly currentTree │ └── kind-of@3.2.2
19 silly currentTree ├── is-obj@2.0.0
19 silly currentTree ├── is-path-inside@3.0.2
19 silly currentTree ├── is-plain-object@2.0.4
19 silly currentTree ├── is-promise@2.1.0
19 silly currentTree ├── is-regex@1.0.4
19 silly currentTree ├── is-relative@0.1.3
19 silly currentTree ├── is-stream@2.0.0
19 silly currentTree ├── is-string@1.0.5
19 silly currentTree ├── is-symbol@1.0.2
19 silly currentTree ├── is-typedarray@1.0.0
19 silly currentTree ├── is-utf8@0.2.1
19 silly currentTree ├── is-windows@1.0.2
19 silly currentTree ├── is-wsl@2.2.0
19 silly currentTree ├── is-yarn-global@0.3.0
19 silly currentTree ├── isarray@1.0.0
19 silly currentTree ├── isexe@2.0.0
19 silly currentTree ├── isobject@3.0.1
19 silly currentTree ├── isstream@0.1.2
19 silly currentTree ├── jed@1.1.1
19 silly currentTree ├── jetpack-id@1.0.0
19 silly currentTree ├── js-select@0.6.0
19 silly currentTree ├── js-tokens@4.0.0
19 silly currentTree ├── js-yaml@3.13.1
19 silly currentTree ├── jsbn@0.1.1
19 silly currentTree ├── json-buffer@3.0.0
19 silly currentTree ├── json-merge-patch@0.2.3
19 silly currentTree ├── json-parse-better-errors@1.0.2
19 silly currentTree ├── json-schema-traverse@0.4.1
19 silly currentTree ├── json-schema@0.2.3
19 silly currentTree ├── json-stable-stringify-without-jsonify@1.0.1
19 silly currentTree ├── json-stringify-safe@5.0.1
19 silly currentTree ├─┬ json5@1.0.1
19 silly currentTree │ └── minimist@1.2.5
19 silly currentTree ├── jsonc-parser@2.1.0
19 silly currentTree ├── jsonfile@4.0.0
19 silly currentTree ├── jsonify@0.0.0
19 silly currentTree ├── JSONSelect@0.2.1
19 silly currentTree ├── jsonwebtoken@8.5.1
19 silly currentTree ├── jsprim@1.4.1
19 silly currentTree ├── jszip@2.6.1
19 silly currentTree ├── jwa@1.4.1
19 silly currentTree ├── jws@3.2.2
19 silly currentTree ├── keyv@3.1.0
19 silly currentTree ├── kind-of@6.0.3
19 silly currentTree ├── latest-version@5.1.0
19 silly currentTree ├── lazystream@1.0.0
19 silly currentTree ├── lcid@3.1.1
19 silly currentTree ├── levn@0.3.0
19 silly currentTree ├─┬ lighthouse-logger@1.2.0
19 silly currentTree │ ├── debug@2.6.9
19 silly currentTree │ └── ms@2.0.0
19 silly currentTree ├── lines-and-columns@1.1.6
19 silly currentTree ├── load-json-file@2.0.0
19 silly currentTree ├── loader-runner@2.4.0
19 silly currentTree ├── loader-utils@1.2.3
19 silly currentTree ├── locate-path@2.0.0
19 silly currentTree ├── lodash.includes@4.3.0
19 silly currentTree ├── lodash.isboolean@3.0.3
19 silly currentTree ├── lodash.isinteger@4.0.4
19 silly currentTree ├── lodash.isnumber@3.0.3
19 silly currentTree ├── lodash.isplainobject@4.0.6
19 silly currentTree ├── lodash.isstring@4.0.1
19 silly currentTree ├── lodash.once@4.1.1
19 silly currentTree ├── lodash.sortby@4.7.0
19 silly currentTree ├── lodash@4.17.19
19 silly currentTree ├── lowercase-keys@1.0.1
19 silly currentTree ├── lru-cache@5.1.1
19 silly currentTree ├─┬ make-dir@3.1.0
19 silly currentTree │ └── semver@6.3.0
19 silly currentTree ├── map-age-cleaner@0.1.3
19 silly currentTree ├── map-cache@0.2.2
19 silly currentTree ├── map-visit@1.0.0
19 silly currentTree ├── marky@1.2.1
19 silly currentTree ├── md5.js@1.3.5
19 silly currentTree ├── mdn-browser-compat-data@1.0.25
19 silly currentTree ├─┬ mem@5.1.1
19 silly currentTree │ └── mimic-fn@2.1.0
19 silly currentTree ├── memory-fs@0.4.1
19 silly currentTree ├── memorystream@0.3.1
19 silly currentTree ├── merge-stream@2.0.0
19 silly currentTree ├── micromatch@3.1.10
19 silly currentTree ├── miller-rabin@4.0.1
19 silly currentTree ├── mime-db@1.44.0
19 silly currentTree ├── mime-types@2.1.27
19 silly currentTree ├── mimic-fn@1.2.0
19 silly currentTree ├── mimic-response@1.0.1
19 silly currentTree ├── minimalistic-assert@1.0.1
19 silly currentTree ├── minimalistic-crypto-utils@1.0.1
19 silly currentTree ├── minimatch@3.0.4
19 silly currentTree ├── minimist@1.2.5
19 silly currentTree ├── mississippi@3.0.0
19 silly currentTree ├─┬ mixin-deep@1.3.2
19 silly currentTree │ └── is-extendable@1.0.1
19 silly currentTree ├── mkdirp@0.5.5
19 silly currentTree ├── moment@2.27.0
19 silly currentTree ├── move-concurrently@1.0.1
19 silly currentTree ├── ms@2.1.1
19 silly currentTree ├── multimatch@4.0.0
19 silly currentTree ├── mute-stream@0.0.7
19 silly currentTree ├─┬ mv@2.1.1
19 silly currentTree │ ├── glob@6.0.4
19 silly currentTree │ └── rimraf@2.4.5
19 silly currentTree ├── mz@2.7.0
19 silly currentTree ├── nan@2.14.1
19 silly currentTree ├── nanomatch@1.2.13
19 silly currentTree ├── natural-compare-lite@1.4.0
19 silly currentTree ├── natural-compare@1.4.0
19 silly currentTree ├── ncp@2.0.0
19 silly currentTree ├── neo-async@2.6.1
19 silly currentTree ├── next-tick@1.1.0
19 silly currentTree ├── nice-try@1.0.5
19 silly currentTree ├── node-forge@0.7.6
19 silly currentTree ├─┬ node-libs-browser@2.2.1
19 silly currentTree │ ├── buffer@4.9.2
19 silly currentTree │ └── punycode@1.4.1
19 silly currentTree ├─┬ node-notifier@6.0.0
19 silly currentTree │ └── semver@6.3.0
19 silly currentTree ├── normalize-package-data@2.4.0
19 silly currentTree ├── normalize-path@2.1.1
19 silly currentTree ├── normalize-url@4.5.0
19 silly currentTree ├─┬ npm-run-all@4.1.5
19 silly currentTree │ ├── load-json-file@4.0.0
19 silly currentTree │ ├── parse-json@4.0.0
19 silly currentTree │ ├── path-type@3.0.0
19 silly currentTree │ ├── pify@3.0.0
19 silly currentTree │ └── read-pkg@3.0.0
19 silly currentTree ├─┬ npm-run-path@4.0.1
19 silly currentTree │ └── path-key@3.1.1
19 silly currentTree ├── nth-check@1.0.2
19 silly currentTree ├── oauth-sign@0.9.0
19 silly currentTree ├── object-assign@4.1.1
19 silly currentTree ├─┬ object-copy@0.1.0
19 silly currentTree │ ├── define-property@0.2.5
19 silly currentTree │ └── kind-of@3.2.2
19 silly currentTree ├── object-inspect@1.8.0
19 silly currentTree ├─┬ object-is@1.1.2
19 silly currentTree │ ├── es-abstract@1.17.6
19 silly currentTree │ ├── es-to-primitive@1.2.1
19 silly currentTree │ ├── has-symbols@1.0.1
19 silly currentTree │ ├── is-callable@1.2.0
19 silly currentTree │ ├── is-regex@1.1.0
19 silly currentTree │ └── object-keys@1.1.1
19 silly currentTree ├── object-keys@1.0.12
19 silly currentTree ├── object-visit@1.0.1
19 silly currentTree ├── object.assign@4.1.0
19 silly currentTree ├── object.pick@1.3.0
19 silly currentTree ├─┬ object.values@1.1.1
19 silly currentTree │ ├── es-abstract@1.17.6
19 silly currentTree │ ├── es-to-primitive@1.2.1
19 silly currentTree │ ├── has-symbols@1.0.1
19 silly currentTree │ ├── is-callable@1.2.0
19 silly currentTree │ ├── is-regex@1.1.0
19 silly currentTree │ └── object-keys@1.1.1
19 silly currentTree ├── once@1.4.0
19 silly currentTree ├── onetime@2.0.1
19 silly currentTree ├── open@7.0.0
19 silly currentTree ├── optionator@0.8.2
19 silly currentTree ├── os-browserify@0.3.0
19 silly currentTree ├── os-locale@5.0.0
19 silly currentTree ├── os-shim@0.1.3
19 silly currentTree ├── os-tmpdir@1.0.2
19 silly currentTree ├── p-cancelable@1.1.0
19 silly currentTree ├── p-defer@1.0.0
19 silly currentTree ├── p-is-promise@2.1.0
19 silly currentTree ├── p-limit@1.3.0
19 silly currentTree ├── p-locate@2.0.0
19 silly currentTree ├── p-try@1.0.0
19 silly currentTree ├─┬ package-json@6.5.0
19 silly currentTree │ └── semver@6.3.0
19 silly currentTree ├── pako@1.0.11
19 silly currentTree ├── parallel-transform@1.2.0
19 silly currentTree ├─┬ parent-module@1.0.1
19 silly currentTree │ └── callsites@3.1.0
19 silly currentTree ├── parse-asn1@5.1.5
19 silly currentTree ├── parse-json@2.2.0
19 silly currentTree ├── parse-passwd@1.0.0
19 silly currentTree ├── parse5@3.0.3
19 silly currentTree ├── pascalcase@0.1.1
19 silly currentTree ├── path-browserify@0.0.1
19 silly currentTree ├── path-dirname@1.0.2
19 silly currentTree ├── path-exists@3.0.0
19 silly currentTree ├── path-is-absolute@1.0.1
19 silly currentTree ├── path-is-inside@1.0.2
19 silly currentTree ├── path-key@2.0.1
19 silly currentTree ├── path-parse@1.0.6
19 silly currentTree ├── path-type@2.0.0
19 silly currentTree ├── pbkdf2@3.0.17
19 silly currentTree ├── pend@1.2.0
19 silly currentTree ├── performance-now@2.1.0
19 silly currentTree ├── pidtree@0.3.0
19 silly currentTree ├── pify@2.3.0
19 silly currentTree ├── pino-std-serializers@2.4.2
19 silly currentTree ├── pino@6.3.2
19 silly currentTree ├── pkg-dir@2.0.0
19 silly currentTree ├── posix-character-classes@0.1.1
19 silly currentTree ├─┬ postcss@7.0.32
19 silly currentTree │ ├─┬ chalk@2.4.2
19 silly currentTree │ │ └── supports-color@5.5.0
19 silly currentTree │ ├── source-map@0.6.1
19 silly currentTree │ └── supports-color@6.1.0
19 silly currentTree ├── prelude-ls@1.1.2
19 silly currentTree ├── prepend-http@2.0.0
19 silly currentTree ├── probe-image-size@5.0.0
19 silly currentTree ├── process-nextick-args@2.0.0
19 silly currentTree ├── process@0.11.10
19 silly currentTree ├── progress@2.0.3
19 silly currentTree ├── promise-inflight@1.0.1
19 silly currentTree ├── prr@1.0.1
19 silly currentTree ├── psl@1.8.0
19 silly currentTree ├── public-encrypt@4.0.3
19 silly currentTree ├── pump@3.0.0
19 silly currentTree ├─┬ pumpify@1.5.1
19 silly currentTree │ └── pump@2.0.1
19 silly currentTree ├── punycode@2.1.1
19 silly currentTree ├── qs@6.5.2
19 silly currentTree ├── querystring-es3@0.2.1
19 silly currentTree ├── querystring@0.2.0
19 silly currentTree ├── quick-format-unescaped@4.0.1
19 silly currentTree ├── randombytes@2.1.0
19 silly currentTree ├── randomfill@1.0.4
19 silly currentTree ├── rc@1.2.8
19 silly currentTree ├── read-pkg-up@2.0.0
19 silly currentTree ├── read-pkg@2.0.0
19 silly currentTree ├── readable-stream@2.3.6
19 silly currentTree ├── readdirp@2.2.1
19 silly currentTree ├── rechoir@0.6.2
19 silly currentTree ├── regenerator-runtime@0.13.5
19 silly currentTree ├── regex-not@1.0.2
19 silly currentTree ├─┬ regexp.prototype.flags@1.3.0
19 silly currentTree │ ├── es-abstract@1.17.6
19 silly currentTree │ ├── es-to-primitive@1.2.1
19 silly currentTree │ ├── has-symbols@1.0.1
19 silly currentTree │ ├── is-callable@1.2.0
19 silly currentTree │ ├── is-regex@1.1.0
19 silly currentTree │ └── object-keys@1.1.1
19 silly currentTree ├── regexpp@2.0.1
19 silly currentTree ├── registry-auth-token@4.1.1
19 silly currentTree ├── registry-url@5.1.0
19 silly currentTree ├─┬ relaxed-json@1.0.3
19 silly currentTree │ └── chalk@2.4.2
19 silly currentTree ├── remove-trailing-separator@1.1.0
19 silly currentTree ├── repeat-element@1.1.3
19 silly currentTree ├── repeat-string@1.6.1
19 silly currentTree ├── request@2.88.2
19 silly currentTree ├── require-directory@2.1.1
19 silly currentTree ├── require-main-filename@2.0.0
19 silly currentTree ├─┬ resolve-cwd@2.0.0
19 silly currentTree │ └── resolve-from@3.0.0
19 silly currentTree ├─┬ resolve-dir@1.0.1
19 silly currentTree │ └── global-modules@1.0.0
19 silly currentTree ├── resolve-from@4.0.0
19 silly currentTree ├── resolve-url@0.2.1
19 silly currentTree ├── resolve@1.14.2
19 silly currentTree ├── responselike@1.0.2
19 silly currentTree ├── restore-cursor@2.0.0
19 silly currentTree ├── ret@0.1.15
19 silly currentTree ├── rimraf@2.6.3
19 silly currentTree ├── ripemd160@2.0.2
19 silly currentTree ├── run-async@2.3.0
19 silly currentTree ├── run-queue@1.0.3
19 silly currentTree ├── rxjs@6.5.2
19 silly currentTree ├── safe-buffer@5.1.2
19 silly currentTree ├── safe-json-stringify@1.2.0
19 silly currentTree ├── safe-regex@1.1.0
19 silly currentTree ├── safer-buffer@2.1.2
19 silly currentTree ├── sax@1.2.4
19 silly currentTree ├─┬ schema-utils@1.0.0
19 silly currentTree │ └── ajv-keywords@3.4.1
19 silly currentTree ├─┬ semver-diff@3.1.1
19 silly currentTree │ └── semver@6.3.0
19 silly currentTree ├── semver@5.6.0
19 silly currentTree ├── serialize-javascript@2.1.2
19 silly currentTree ├── set-blocking@2.0.0
19 silly currentTree ├─┬ set-value@2.0.1
19 silly currentTree │ └── extend-shallow@2.0.1
19 silly currentTree ├── setimmediate@1.0.5
19 silly currentTree ├── sha.js@2.4.11
19 silly currentTree ├── shebang-command@1.2.0
19 silly currentTree ├── shebang-regex@1.0.0
19 silly currentTree ├── shell-quote@1.6.1
19 silly currentTree ├── shelljs@0.7.7
19 silly currentTree ├── shellwords@0.1.1
19 silly currentTree ├─┬ sign-addon@2.0.5
19 silly currentTree │ ├── core-js@3.6.4
19 silly currentTree │ ├── es6-promisify@6.0.2
19 silly currentTree │ ├── punycode@1.4.1
19 silly currentTree │ ├── request@2.88.0
19 silly currentTree │ └── tough-cookie@2.4.3
19 silly currentTree ├── signal-exit@3.0.2
19 silly currentTree ├── slice-ansi@2.1.0
19 silly currentTree ├─┬ snapdragon-node@2.1.1
19 silly currentTree │ ├── define-property@1.0.0
19 silly currentTree │ ├── is-accessor-descriptor@1.0.0
19 silly currentTree │ ├── is-data-descriptor@1.0.0
19 silly currentTree │ └── is-descriptor@1.0.2
19 silly currentTree ├─┬ snapdragon-util@3.0.1
19 silly currentTree │ └── kind-of@3.2.2
19 silly currentTree ├─┬ snapdragon@0.8.2
19 silly currentTree │ ├── debug@2.6.9
19 silly currentTree │ ├── define-property@0.2.5
19 silly currentTree │ ├── extend-shallow@2.0.1
19 silly currentTree │ └── ms@2.0.0
19 silly currentTree ├── sonic-boom@1.0.1
19 silly currentTree ├── source-list-map@2.0.1
19 silly currentTree ├── source-map-resolve@0.5.2
19 silly currentTree ├─┬ source-map-support@0.5.16
19 silly currentTree │ └── source-map@0.6.1
19 silly currentTree ├── source-map-url@0.4.0
19 silly currentTree ├── source-map@0.5.7
19 silly currentTree ├── spawn-sync@1.0.15
19 silly currentTree ├── spdx-correct@3.1.0
19 silly currentTree ├── spdx-exceptions@2.2.0
19 silly currentTree ├── spdx-expression-parse@3.0.0
19 silly currentTree ├── spdx-license-ids@3.0.3
19 silly currentTree ├── split-string@3.1.0
19 silly currentTree ├── split@0.3.3
19 silly currentTree ├── sprintf-js@1.0.3
19 silly currentTree ├── sshpk@1.16.1
19 silly currentTree ├── ssri@6.0.1
19 silly currentTree ├─┬ static-extend@0.1.2
19 silly currentTree │ └── define-property@0.2.5
19 silly currentTree ├── stream-browserify@2.0.2
19 silly currentTree ├── stream-each@1.2.3
19 silly currentTree ├── stream-http@2.8.3
19 silly currentTree ├─┬ stream-parser@0.3.1
19 silly currentTree │ ├── debug@2.6.9
19 silly currentTree │ └── ms@2.0.0
19 silly currentTree ├── stream-shift@1.0.1
19 silly currentTree ├── stream-to-array@2.3.0
19 silly currentTree ├─┬ stream-to-promise@2.2.0
19 silly currentTree │ ├── end-of-stream@1.1.0
19 silly currentTree │ └── once@1.3.3
19 silly currentTree ├── string_decoder@1.1.1
19 silly currentTree ├── string-width@2.1.1
19 silly currentTree ├── string.prototype.padend@3.0.0
19 silly currentTree ├─┬ string.prototype.trimend@1.0.1
19 silly currentTree │ ├── es-abstract@1.17.6
19 silly currentTree │ ├── es-to-primitive@1.2.1
19 silly currentTree │ ├── has-symbols@1.0.1
19 silly currentTree │ ├── is-callable@1.2.0
19 silly currentTree │ ├── is-regex@1.1.0
19 silly currentTree │ └── object-keys@1.1.1
19 silly currentTree ├─┬ string.prototype.trimstart@1.0.1
19 silly currentTree │ ├── es-abstract@1.17.6
19 silly currentTree │ ├── es-to-primitive@1.2.1
19 silly currentTree │ ├── has-symbols@1.0.1
19 silly currentTree │ ├── is-callable@1.2.0
19 silly currentTree │ ├── is-regex@1.1.0
19 silly currentTree │ └── object-keys@1.1.1
19 silly currentTree ├── strip-ansi@4.0.0
19 silly currentTree ├── strip-bom-buf@2.0.0
19 silly currentTree ├── strip-bom-stream@4.0.0
19 silly currentTree ├── strip-bom@3.0.0
19 silly currentTree ├── strip-final-newline@2.0.0
19 silly currentTree ├── strip-json-comments@2.0.1
19 silly currentTree ├── supports-color@5.5.0
19 silly currentTree ├─┬ table@5.4.0
19 silly currentTree │ ├── ajv@6.10.0
19 silly currentTree │ ├── ansi-regex@4.1.0
19 silly currentTree │ ├── string-width@3.1.0
19 silly currentTree │ └── strip-ansi@5.2.0
19 silly currentTree ├── tapable@1.1.3
19 silly currentTree ├── tar-stream@1.6.2
19 silly currentTree ├── term-size@2.2.0
19 silly currentTree ├─┬ terser-webpack-plugin@1.4.3
19 silly currentTree │ ├── is-wsl@1.1.0
19 silly currentTree │ └── source-map@0.6.1
19 silly currentTree ├─┬ terser@4.6.12
19 silly currentTree │ └── source-map@0.6.1
19 silly currentTree ├── text-table@0.2.0
19 silly currentTree ├── thenify-all@1.6.0
19 silly currentTree ├── thenify@3.3.1
19 silly currentTree ├── through@2.3.8
19 silly currentTree ├── through2@2.0.5
19 silly currentTree ├── timers-browserify@2.0.11
19 silly currentTree ├── tmp@0.0.33
19 silly currentTree ├── to-arraybuffer@1.0.1
19 silly currentTree ├── to-buffer@1.1.1
19 silly currentTree ├─┬ to-object-path@0.3.0
19 silly currentTree │ └── kind-of@3.2.2
19 silly currentTree ├── to-readable-stream@1.0.0
19 silly currentTree ├── to-regex-range@2.1.1
19 silly currentTree ├── to-regex@3.0.2
19 silly currentTree ├── tosource@1.0.0
19 silly currentTree ├── tough-cookie@2.5.0
19 silly currentTree ├── tr46@2.0.2
19 silly currentTree ├── traverse@0.4.6
19 silly currentTree ├── ts-loader@5.4.5
19 silly currentTree ├── tsconfig-paths@3.9.0
19 silly currentTree ├── tslib@1.9.3
19 silly currentTree ├─┬ tslint-eslint-rules@5.4.0
19 silly currentTree │ ├── doctrine@0.7.2
19 silly currentTree │ ├── esutils@1.1.6
19 silly currentTree │ ├── isarray@0.0.1
19 silly currentTree │ ├── tslib@1.9.0
19 silly currentTree │ └── tsutils@3.6.0
19 silly currentTree ├── tslint@5.20.1
19 silly currentTree ├── tsutils@2.29.0
19 silly currentTree ├── tty-browserify@0.0.0
19 silly currentTree ├── tunnel-agent@0.6.0
19 silly currentTree ├── tweetnacl@0.14.5
19 silly currentTree ├── type-check@0.3.2
19 silly currentTree ├── type-detect@4.0.8
19 silly currentTree ├── type-fest@0.8.1
19 silly currentTree ├── typedarray-to-buffer@3.1.5
19 silly currentTree ├── typedarray@0.0.6
19 silly currentTree ├── typescript@3.9.6
19 silly currentTree ├── union-value@1.0.1
19 silly currentTree ├── unique-filename@1.1.1
19 silly currentTree ├── unique-slug@2.0.2
19 silly currentTree ├── unique-string@2.0.0
19 silly currentTree ├── universalify@0.1.2
19 silly currentTree ├─┬ unset-value@1.0.0
19 silly currentTree │ ├─┬ has-value@0.3.1
19 silly currentTree │ │ └── isobject@2.1.0
19 silly currentTree │ └── has-values@0.1.4
19 silly currentTree ├── upath@1.2.0
19 silly currentTree ├─┬ update-notifier@4.0.0
19 silly currentTree │ ├── ansi-styles@4.2.1
19 silly currentTree │ ├── chalk@3.0.0
19 silly currentTree │ ├── color-convert@2.0.1
19 silly currentTree │ ├── color-name@1.1.4
19 silly currentTree │ ├── has-flag@4.0.0
19 silly currentTree │ └── supports-color@7.1.0
19 silly currentTree ├── uri-js@4.2.2
19 silly currentTree ├── urix@0.1.0
19 silly currentTree ├── url-parse-lax@3.0.0
19 silly currentTree ├─┬ url@0.11.0
19 silly currentTree │ └── punycode@1.3.2
19 silly currentTree ├── use@3.1.1
19 silly currentTree ├── util-deprecate@1.0.2
19 silly currentTree ├── util@0.11.1
19 silly currentTree ├── uuid@3.4.0
19 silly currentTree ├── v8-compile-cache@2.1.1
19 silly currentTree ├── validate-npm-package-license@3.0.4
19 silly currentTree ├── verror@1.10.0
19 silly currentTree ├── vm-browserify@1.1.2
19 silly currentTree ├─┬ vscode-json-languageservice@3.3.0
19 silly currentTree │ └── vscode-languageserver-types@3.15.0-next.2
19 silly currentTree ├── vscode-nls@4.1.1
19 silly currentTree ├── vscode-uri@2.0.1
19 silly currentTree ├── watchpack@1.6.1
19 silly currentTree ├── wcwidth@1.0.1
19 silly currentTree ├─┬ web-ext@4.3.0
19 silly currentTree │ ├── import-fresh@3.2.1
19 silly currentTree │ ├── mkdirp@1.0.4
19 silly currentTree │ ├── parse-json@5.0.0
19 silly currentTree │ ├── source-map-support@0.5.19
19 silly currentTree │ ├── source-map@0.6.1
19 silly currentTree │ ├── strip-bom@4.0.0
19 silly currentTree │ ├── strip-json-comments@3.0.1
19 silly currentTree │ └── tmp@0.1.0
19 silly currentTree ├── webidl-conversions@5.0.0
19 silly currentTree ├─┬ webpack-cli@3.3.12
19 silly currentTree │ ├── ansi-regex@4.1.0
19 silly currentTree │ ├─┬ chalk@2.4.2
19 silly currentTree │ │ └── supports-color@5.5.0
19 silly currentTree │ ├── cliui@5.0.0
19 silly currentTree │ ├── decamelize@1.2.0
19 silly currentTree │ ├── emojis-list@3.0.0
19 silly currentTree │ ├── enhanced-resolve@4.2.0
19 silly currentTree │ ├── find-up@3.0.0
19 silly currentTree │ ├── loader-utils@1.4.0
19 silly currentTree │ ├── locate-path@3.0.0
19 silly currentTree │ ├── memory-fs@0.5.0
19 silly currentTree │ ├── p-limit@2.3.0
19 silly currentTree │ ├── p-locate@3.0.0
19 silly currentTree │ ├── p-try@2.2.0
19 silly currentTree │ ├── string-width@3.1.0
19 silly currentTree │ ├── strip-ansi@5.2.0
19 silly currentTree │ ├── supports-color@6.1.0
19 silly currentTree │ ├── wrap-ansi@5.1.0
19 silly currentTree │ ├── yargs-parser@13.1.2
19 silly currentTree │ └── yargs@13.3.2
19 silly currentTree ├─┬ webpack-sources@1.4.3
19 silly currentTree │ └── source-map@0.6.1
19 silly currentTree ├─┬ webpack@4.43.0
19 silly currentTree │ ├── acorn@6.4.1
19 silly currentTree │ ├── ajv-keywords@3.4.1
19 silly currentTree │ └── eslint-scope@4.0.3
19 silly currentTree ├── whatwg-url@8.1.0
19 silly currentTree ├── when@3.7.7
19 silly currentTree ├── which-module@2.0.0
19 silly currentTree ├── which@1.3.1
19 silly currentTree ├─┬ widest-line@3.1.0
19 silly currentTree │ ├── ansi-regex@5.0.0
19 silly currentTree │ ├── emoji-regex@8.0.0
19 silly currentTree │ ├── is-fullwidth-code-point@3.0.0
19 silly currentTree │ ├── string-width@4.2.0
19 silly currentTree │ └── strip-ansi@6.0.0
19 silly currentTree ├── winreg@0.0.12
19 silly currentTree ├── wordwrap@1.0.0
19 silly currentTree ├── worker-farm@1.7.0
19 silly currentTree ├─┬ wrap-ansi@6.2.0
19 silly currentTree │ ├── ansi-regex@5.0.0
19 silly currentTree │ ├── ansi-styles@4.2.1
19 silly currentTree │ ├── color-convert@2.0.1
19 silly currentTree │ ├── color-name@1.1.4
19 silly currentTree │ ├── emoji-regex@8.0.0
19 silly currentTree │ ├── is-fullwidth-code-point@3.0.0
19 silly currentTree │ ├── string-width@4.2.0
19 silly currentTree │ └── strip-ansi@6.0.0
19 silly currentTree ├── wrappy@1.0.2
19 silly currentTree ├── write-file-atomic@3.0.3
19 silly currentTree ├── ws@7.2.3
19 silly currentTree ├── xdg-basedir@4.0.0
19 silly currentTree ├── xml2js@0.4.23
19 silly currentTree ├── xmlbuilder@11.0.1
19 silly currentTree ├── xregexp@4.3.0
19 silly currentTree ├── xtend@4.0.2
19 silly currentTree ├── y18n@4.0.0
19 silly currentTree ├── yallist@3.1.1
19 silly currentTree ├─┬ yargs-parser@18.1.3
19 silly currentTree │ └── decamelize@1.2.0
19 silly currentTree ├─┬ yargs@15.3.1
19 silly currentTree │ ├── ansi-regex@5.0.0
19 silly currentTree │ ├── decamelize@1.2.0
19 silly currentTree │ ├── emoji-regex@8.0.0
19 silly currentTree │ ├── find-up@4.1.0
19 silly currentTree │ ├── is-fullwidth-code-point@3.0.0
19 silly currentTree │ ├── locate-path@5.0.0
19 silly currentTree │ ├── p-limit@2.3.0
19 silly currentTree │ ├── p-locate@4.1.0
19 silly currentTree │ ├── p-try@2.2.0
19 silly currentTree │ ├── path-exists@4.0.0
19 silly currentTree │ ├── string-width@4.2.0
19 silly currentTree │ └── strip-ansi@6.0.0
19 silly currentTree ├── yauzl@2.10.0
19 silly currentTree ├─┬ zip-dir@1.0.2
19 silly currentTree │ └── async@1.5.2
19 silly currentTree └── zip-stream@1.2.0
20 silly idealTree OpenWPM@
20 silly idealTree ├── @babel/code-frame@7.0.0
20 silly idealTree ├── @babel/highlight@7.0.0
20 silly idealTree ├── @babel/polyfill@7.7.0
20 silly idealTree ├── @babel/runtime-corejs3@7.10.4
20 silly idealTree ├── @babel/runtime@7.7.7
20 silly idealTree ├── @cliqz-oss/firefox-client@0.3.1
20 silly idealTree ├── @cliqz-oss/node-firefox-connect@1.2.1
20 silly idealTree ├─┬ @openwpm/webext-instrumentation@
20 silly idealTree │ ├── extend@3.0.2
20 silly idealTree │ └── mdn-browser-compat-data@1.0.20
20 silly idealTree ├── @sindresorhus/is@0.14.0
20 silly idealTree ├── @szmarczak/http-timer@1.1.2
20 silly idealTree ├── @types/color-name@1.1.1
20 silly idealTree ├── @types/eslint-visitor-keys@1.0.0
20 silly idealTree ├── @types/json-schema@7.0.5
20 silly idealTree ├── @types/json5@0.0.29
20 silly idealTree ├── @types/minimatch@3.0.3
20 silly idealTree ├── @types/node@14.0.19
20 silly idealTree ├─┬ @typescript-eslint/experimental-utils@2.34.0
20 silly idealTree │ ├── eslint-utils@2.0.0
20 silly idealTree │ └── eslint-visitor-keys@1.2.0
20 silly idealTree ├─┬ @typescript-eslint/parser@2.34.0
20 silly idealTree │ └── eslint-visitor-keys@1.2.0
20 silly idealTree ├─┬ @typescript-eslint/typescript-estree@2.34.0
20 silly idealTree │ ├── eslint-visitor-keys@1.2.0
20 silly idealTree │ ├── glob@7.1.6
20 silly idealTree │ ├── semver@7.3.2
20 silly idealTree │ └── tsutils@3.17.1
20 silly idealTree ├── @webassemblyjs/ast@1.9.0
20 silly idealTree ├── @webassemblyjs/floating-point-hex-parser@1.9.0
20 silly idealTree ├── @webassemblyjs/helper-api-error@1.9.0
20 silly idealTree ├── @webassemblyjs/helper-buffer@1.9.0
20 silly idealTree ├── @webassemblyjs/helper-code-frame@1.9.0
20 silly idealTree ├── @webassemblyjs/helper-fsm@1.9.0
20 silly idealTree ├── @webassemblyjs/helper-module-context@1.9.0
20 silly idealTree ├── @webassemblyjs/helper-wasm-bytecode@1.9.0
20 silly idealTree ├── @webassemblyjs/helper-wasm-section@1.9.0
20 silly idealTree ├── @webassemblyjs/ieee754@1.9.0
20 silly idealTree ├── @webassemblyjs/leb128@1.9.0
20 silly idealTree ├── @webassemblyjs/utf8@1.9.0
20 silly idealTree ├── @webassemblyjs/wasm-edit@1.9.0
20 silly idealTree ├── @webassemblyjs/wasm-gen@1.9.0
20 silly idealTree ├── @webassemblyjs/wasm-opt@1.9.0
20 silly idealTree ├── @webassemblyjs/wasm-parser@1.9.0
20 silly idealTree ├── @webassemblyjs/wast-parser@1.9.0
20 silly idealTree ├── @webassemblyjs/wast-printer@1.9.0
20 silly idealTree ├── @xtuc/ieee754@1.2.0
20 silly idealTree ├── @xtuc/long@4.2.2
20 silly idealTree ├── acorn-jsx@5.0.1
20 silly idealTree ├── acorn@7.3.1
20 silly idealTree ├── adbkit-logcat@1.1.0
20 silly idealTree ├── adbkit-monkey@1.0.1
20 silly idealTree ├─┬ adbkit@2.11.1
20 silly idealTree │ ├── debug@2.6.9
20 silly idealTree │ └── ms@2.0.0
20 silly idealTree ├─┬ addons-linter@1.26.0
20 silly idealTree │ ├── @babel/runtime@7.10.2
20 silly idealTree │ ├── acorn-jsx@5.2.0
20 silly idealTree │ ├── ajv@6.12.2
20 silly idealTree │ ├── ansi-styles@4.2.1
20 silly idealTree │ ├── chalk@4.1.0
20 silly idealTree │ ├── color-convert@2.0.1
20 silly idealTree │ ├── color-name@1.1.4
20 silly idealTree │ ├── eslint-visitor-keys@1.2.0
20 silly idealTree │ ├── espree@6.2.1
20 silly idealTree │ ├── fast-deep-equal@3.1.3
20 silly idealTree │ ├── glob@7.1.6
20 silly idealTree │ ├── has-flag@4.0.0
20 silly idealTree │ ├── semver@7.3.2
20 silly idealTree │ ├── source-map-support@0.5.19
20 silly idealTree │ ├── source-map@0.6.1
20 silly idealTree │ └── supports-color@7.1.0
20 silly idealTree ├── adm-zip@0.4.16
20 silly idealTree ├── ajv-errors@1.0.1
20 silly idealTree ├── ajv-merge-patch@4.1.0
20 silly idealTree ├─┬ ajv@6.12.0
20 silly idealTree │ └── fast-deep-equal@3.1.1
20 silly idealTree ├─┬ ansi-align@3.0.0
20 silly idealTree │ ├── ansi-regex@4.1.0
20 silly idealTree │ ├── string-width@3.1.0
20 silly idealTree │ └── strip-ansi@5.2.0
20 silly idealTree ├── ansi-regex@3.0.0
20 silly idealTree ├── ansi-styles@3.2.1
20 silly idealTree ├── any-promise@1.3.0
20 silly idealTree ├── anymatch@2.0.0
20 silly idealTree ├── aproba@1.2.0
20 silly idealTree ├── archiver-utils@1.3.0
20 silly idealTree ├─┬ archiver@2.1.1
20 silly idealTree │ └── async@2.6.3
20 silly idealTree ├── argparse@1.0.10
20 silly idealTree ├── arr-diff@4.0.0
20 silly idealTree ├── arr-flatten@1.1.0
20 silly idealTree ├── arr-union@3.1.0
20 silly idealTree ├── array-differ@3.0.0
20 silly idealTree ├── array-filter@0.0.1
20 silly idealTree ├─┬ array-includes@3.1.1
20 silly idealTree │ ├── es-abstract@1.17.6
20 silly idealTree │ ├── es-to-primitive@1.2.1
20 silly idealTree │ ├── has-symbols@1.0.1
20 silly idealTree │ ├── is-callable@1.2.0
20 silly idealTree │ ├── is-regex@1.1.0
20 silly idealTree │ └── object-keys@1.1.1
20 silly idealTree ├── array-map@0.0.0
20 silly idealTree ├── array-reduce@0.0.0
20 silly idealTree ├── array-union@2.1.0
20 silly idealTree ├── array-unique@0.3.2
20 silly idealTree ├─┬ array.prototype.flat@1.2.3
20 silly idealTree │ ├── es-abstract@1.17.6
20 silly idealTree │ ├── es-to-primitive@1.2.1
20 silly idealTree │ ├── has-symbols@1.0.1
20 silly idealTree │ ├── is-callable@1.2.0
20 silly idealTree │ ├── is-regex@1.1.0
20 silly idealTree │ └── object-keys@1.1.1
20 silly idealTree ├── arrify@2.0.1
20 silly idealTree ├── asn1.js@4.10.1
20 silly idealTree ├── asn1@0.2.4
20 silly idealTree ├── assert-plus@1.0.0
20 silly idealTree ├─┬ assert@1.5.0
20 silly idealTree │ ├── inherits@2.0.1
20 silly idealTree │ └── util@0.10.3
20 silly idealTree ├── assign-symbols@1.0.0
20 silly idealTree ├── astral-regex@1.0.0
20 silly idealTree ├── async-each@1.0.3
20 silly idealTree ├── async@0.2.10
20 silly idealTree ├── asynckit@0.4.0
20 silly idealTree ├── atob@2.1.2
20 silly idealTree ├── atomic-sleep@1.0.0
20 silly idealTree ├── aws-sign2@0.7.0
20 silly idealTree ├── aws4@1.10.0
20 silly idealTree ├── balanced-match@1.0.0
20 silly idealTree ├─┬ base@0.11.2
20 silly idealTree │ ├── define-property@1.0.0
20 silly idealTree │ ├── is-accessor-descriptor@1.0.0
20 silly idealTree │ ├── is-data-descriptor@1.0.0
20 silly idealTree │ └── is-descriptor@1.0.2
20 silly idealTree ├── base64-js@1.3.1
20 silly idealTree ├── bcrypt-pbkdf@1.0.2
20 silly idealTree ├── big.js@5.2.2
20 silly idealTree ├── binary-extensions@1.13.1
20 silly idealTree ├── bl@1.2.2
20 silly idealTree ├── bluebird@2.9.34
20 silly idealTree ├── bn.js@4.11.8
20 silly idealTree ├── boolbase@1.0.0
20 silly idealTree ├─┬ boxen@4.2.0
20 silly idealTree │ ├── ansi-regex@5.0.0
20 silly idealTree │ ├── ansi-styles@4.2.1
20 silly idealTree │ ├── chalk@3.0.0
20 silly idealTree │ ├── color-convert@2.0.1
20 silly idealTree │ ├── color-name@1.1.4
20 silly idealTree │ ├── emoji-regex@8.0.0
20 silly idealTree │ ├── has-flag@4.0.0
20 silly idealTree │ ├── is-fullwidth-code-point@3.0.0
20 silly idealTree │ ├── string-width@4.2.0
20 silly idealTree │ ├── strip-ansi@6.0.0
20 silly idealTree │ └── supports-color@7.1.0
20 silly idealTree ├── brace-expansion@1.1.11
20 silly idealTree ├─┬ braces@2.3.2
20 silly idealTree │ └── extend-shallow@2.0.1
20 silly idealTree ├── brorand@1.1.0
20 silly idealTree ├── browserify-aes@1.2.0
20 silly idealTree ├── browserify-cipher@1.0.1
20 silly idealTree ├── browserify-des@1.0.2
20 silly idealTree ├── browserify-rsa@4.0.1
20 silly idealTree ├── browserify-sign@4.0.4
20 silly idealTree ├── browserify-zlib@0.2.0
20 silly idealTree ├── buffer-alloc-unsafe@1.1.0
20 silly idealTree ├── buffer-alloc@1.2.0
20 silly idealTree ├── buffer-crc32@0.2.13
20 silly idealTree ├── buffer-equal-constant-time@1.0.1
20 silly idealTree ├── buffer-fill@1.0.0
20 silly idealTree ├── buffer-from@1.1.1
20 silly idealTree ├── buffer-xor@1.0.3
20 silly idealTree ├── buffer@5.6.0
20 silly idealTree ├── builtin-modules@1.1.1
20 silly idealTree ├── builtin-status-codes@3.0.0
20 silly idealTree ├── bunyan@1.8.12
20 silly idealTree ├─┬ cacache@12.0.4
20 silly idealTree │ ├── bluebird@3.7.2
20 silly idealTree │ └── glob@7.1.6
20 silly idealTree ├── cache-base@1.0.1
20 silly idealTree ├─┬ cacheable-request@6.1.0
20 silly idealTree │ └── lowercase-keys@2.0.0
20 silly idealTree ├── camelcase@5.3.1
20 silly idealTree ├── caseless@0.12.0
20 silly idealTree ├── chalk@2.4.1
20 silly idealTree ├── chardet@0.7.0
20 silly idealTree ├─┬ cheerio@1.0.0-rc.3
20 silly idealTree │ └── dom-serializer@0.1.1
20 silly idealTree ├─┬ chokidar@2.1.8
20 silly idealTree │ ├─┬ fsevents@1.2.12
20 silly idealTree │ │ ├── abbrev@1.1.1
20 silly idealTree │ │ ├── ansi-regex@2.1.1
20 silly idealTree │ │ ├── aproba@1.2.0
20 silly idealTree │ │ ├── are-we-there-yet@1.1.5
20 silly idealTree │ │ ├── balanced-match@1.0.0
20 silly idealTree │ │ ├── brace-expansion@1.1.11
20 silly idealTree │ │ ├── chownr@1.1.4
20 silly idealTree │ │ ├── code-point-at@1.1.0
20 silly idealTree │ │ ├── concat-map@0.0.1
20 silly idealTree │ │ ├── console-control-strings@1.1.0
20 silly idealTree │ │ ├── core-util-is@1.0.2
20 silly idealTree │ │ ├── debug@3.2.6
20 silly idealTree │ │ ├── deep-extend@0.6.0
20 silly idealTree │ │ ├── delegates@1.0.0
20 silly idealTree │ │ ├── detect-libc@1.0.3
20 silly idealTree │ │ ├── fs-minipass@1.2.7
20 silly idealTree │ │ ├── fs.realpath@1.0.0
20 silly idealTree │ │ ├── gauge@2.7.4
20 silly idealTree │ │ ├── glob@7.1.6
20 silly idealTree │ │ ├── has-unicode@2.0.1
20 silly idealTree │ │ ├── iconv-lite@0.4.24
20 silly idealTree │ │ ├── ignore-walk@3.0.3
20 silly idealTree │ │ ├── inflight@1.0.6
20 silly idealTree │ │ ├── inherits@2.0.4
20 silly idealTree │ │ ├── ini@1.3.5
20 silly idealTree │ │ ├── is-fullwidth-code-point@1.0.0
20 silly idealTree │ │ ├── isarray@1.0.0
20 silly idealTree │ │ ├── minimatch@3.0.4
20 silly idealTree │ │ ├── minimist@1.2.5
20 silly idealTree │ │ ├── minipass@2.9.0
20 silly idealTree │ │ ├── minizlib@1.3.3
20 silly idealTree │ │ ├── mkdirp@0.5.3
20 silly idealTree │ │ ├── ms@2.1.2
20 silly idealTree │ │ ├── needle@2.3.3
20 silly idealTree │ │ ├── node-pre-gyp@0.14.0
20 silly idealTree │ │ ├── nopt@4.0.3
20 silly idealTree │ │ ├── npm-bundled@1.1.1
20 silly idealTree │ │ ├── npm-normalize-package-bin@1.0.1
20 silly idealTree │ │ ├── npm-packlist@1.4.8
20 silly idealTree │ │ ├── npmlog@4.1.2
20 silly idealTree │ │ ├── number-is-nan@1.0.1
20 silly idealTree │ │ ├── object-assign@4.1.1
20 silly idealTree │ │ ├── once@1.4.0
20 silly idealTree │ │ ├── os-homedir@1.0.2
20 silly idealTree │ │ ├── os-tmpdir@1.0.2
20 silly idealTree │ │ ├── osenv@0.1.5
20 silly idealTree │ │ ├── path-is-absolute@1.0.1
20 silly idealTree │ │ ├── process-nextick-args@2.0.1
20 silly idealTree │ │ ├── rc@1.2.8
20 silly idealTree │ │ ├── readable-stream@2.3.7
20 silly idealTree │ │ ├── rimraf@2.7.1
20 silly idealTree │ │ ├── safe-buffer@5.1.2
20 silly idealTree │ │ ├── safer-buffer@2.1.2
20 silly idealTree │ │ ├── sax@1.2.4
20 silly idealTree │ │ ├── semver@5.7.1
20 silly idealTree │ │ ├── set-blocking@2.0.0
20 silly idealTree │ │ ├── signal-exit@3.0.2
20 silly idealTree │ │ ├── string_decoder@1.1.1
20 silly idealTree │ │ ├── string-width@1.0.2
20 silly idealTree │ │ ├── strip-ansi@3.0.1
20 silly idealTree │ │ ├── strip-json-comments@2.0.1
20 silly idealTree │ │ ├── tar@4.4.13
20 silly idealTree │ │ ├── util-deprecate@1.0.2
20 silly idealTree │ │ ├── wide-align@1.1.3
20 silly idealTree │ │ ├── wrappy@1.0.2
20 silly idealTree │ │ └── yallist@3.1.1
20 silly idealTree │ └── normalize-path@3.0.0
20 silly idealTree ├── chownr@1.1.4
20 silly idealTree ├─┬ chrome-launcher@0.13.3
20 silly idealTree │ └── rimraf@3.0.2
20 silly idealTree ├── chrome-trace-event@1.0.2
20 silly idealTree ├── ci-info@2.0.0
20 silly idealTree ├── cipher-base@1.0.4
20 silly idealTree ├─┬ class-utils@0.3.6
20 silly idealTree │ └── define-property@0.2.5
20 silly idealTree ├── cli-boxes@2.2.0
20 silly idealTree ├── cli-cursor@2.1.0
20 silly idealTree ├── cli-width@2.2.0
20 silly idealTree ├─┬ cliui@6.0.0
20 silly idealTree │ ├── ansi-regex@5.0.0
20 silly idealTree │ ├── emoji-regex@8.0.0
20 silly idealTree │ ├── is-fullwidth-code-point@3.0.0
20 silly idealTree │ ├── string-width@4.2.0
20 silly idealTree │ └── strip-ansi@6.0.0
20 silly idealTree ├── clone-response@1.0.2
20 silly idealTree ├── clone@1.0.4
20 silly idealTree ├── collection-visit@1.0.0
20 silly idealTree ├── color-convert@1.9.3
20 silly idealTree ├── color-name@1.1.3
20 silly idealTree ├── colors@0.5.1
20 silly idealTree ├─┬ columnify@1.5.4
20 silly idealTree │ ├── ansi-regex@2.1.1
20 silly idealTree │ └── strip-ansi@3.0.1
20 silly idealTree ├── combined-stream@1.0.8
20 silly idealTree ├── commander@2.20.3
20 silly idealTree ├── common-tags@1.8.0
20 silly idealTree ├── commondir@1.0.1
20 silly idealTree ├── component-emitter@1.2.1
20 silly idealTree ├── compress-commons@1.2.2
20 silly idealTree ├── concat-map@0.0.1
20 silly idealTree ├── concat-stream@1.6.2
20 silly idealTree ├── configstore@5.0.1
20 silly idealTree ├── console-browserify@1.2.0
20 silly idealTree ├── constants-browserify@1.0.0
20 silly idealTree ├── contains-path@0.1.0
20 silly idealTree ├── copy-concurrently@1.0.5
20 silly idealTree ├── copy-descriptor@0.1.1
20 silly idealTree ├── core-js-pure@3.6.5
20 silly idealTree ├── core-js@2.6.11
20 silly idealTree ├── core-util-is@1.0.2
20 silly idealTree ├── crc@3.8.0
20 silly idealTree ├── crc32-stream@2.0.0
20 silly idealTree ├── create-ecdh@4.0.3
20 silly idealTree ├── create-hash@1.2.0
20 silly idealTree ├── create-hmac@1.1.7
20 silly idealTree ├── cross-spawn@6.0.5
20 silly idealTree ├── crypto-browserify@3.12.0
20 silly idealTree ├── crypto-random-string@2.0.0
20 silly idealTree ├─┬ css-select@1.2.0
20 silly idealTree │ └── domutils@1.5.1
20 silly idealTree ├── css-what@2.1.3
20 silly idealTree ├── cyclist@1.0.1
20 silly idealTree ├── dashdash@1.14.1
20 silly idealTree ├── debounce@1.2.0
20 silly idealTree ├── debug@4.1.1
20 silly idealTree ├── decamelize@3.2.0
20 silly idealTree ├── decode-uri-component@0.2.0
20 silly idealTree ├── decompress-response@3.3.0
20 silly idealTree ├─┬ deep-equal@1.1.1
20 silly idealTree │ └── object-keys@1.1.1
20 silly idealTree ├── deep-extend@0.6.0
20 silly idealTree ├── deep-is@0.1.3
20 silly idealTree ├── deepcopy@2.0.0
20 silly idealTree ├── deepmerge@4.2.2
20 silly idealTree ├── defaults@1.0.3
20 silly idealTree ├── defer-to-connect@1.1.3
20 silly idealTree ├── define-properties@1.1.3
20 silly idealTree ├─┬ define-property@2.0.2
20 silly idealTree │ ├── is-accessor-descriptor@1.0.0
20 silly idealTree │ ├── is-data-descriptor@1.0.0
20 silly idealTree │ └── is-descriptor@1.0.2
20 silly idealTree ├── delayed-stream@1.0.0
20 silly idealTree ├── des.js@1.0.1
20 silly idealTree ├── detect-file@1.0.0
20 silly idealTree ├── diff@4.0.1
20 silly idealTree ├── diffie-hellman@5.0.3
20 silly idealTree ├─┬ dispensary@0.51.2
20 silly idealTree │ ├── async@3.2.0
20 silly idealTree │ └── pino@6.0.0
20 silly idealTree ├── doctrine@1.5.0
20 silly idealTree ├─┬ dom-serializer@0.1.0
20 silly idealTree │ └── domelementtype@1.1.3
20 silly idealTree ├── domain-browser@1.2.0
20 silly idealTree ├── domelementtype@1.3.1
20 silly idealTree ├── domhandler@2.4.2
20 silly idealTree ├── domutils@1.7.0
20 silly idealTree ├── dot-prop@5.2.0
20 silly idealTree ├── dtrace-provider@0.8.8
20 silly idealTree ├── duplexer3@0.1.4
20 silly idealTree ├── duplexify@3.7.1
20 silly idealTree ├── ecc-jsbn@0.1.2
20 silly idealTree ├── ecdsa-sig-formatter@1.0.11
20 silly idealTree ├── elliptic@6.5.3
20 silly idealTree ├── emoji-regex@7.0.3
20 silly idealTree ├── emojis-list@2.1.0
20 silly idealTree ├── end-of-stream@1.4.4
20 silly idealTree ├── enhanced-resolve@4.1.0
20 silly idealTree ├── entities@1.1.2
20 silly idealTree ├── errno@0.1.7
20 silly idealTree ├── error-ex@1.3.2
20 silly idealTree ├── es-abstract@1.13.0
20 silly idealTree ├── es-to-primitive@1.2.0
20 silly idealTree ├── es6-error@4.1.1
20 silly idealTree ├── es6-promise@2.3.0
20 silly idealTree ├── es6-promisify@6.1.1
20 silly idealTree ├── escape-string-regexp@1.0.5
20 silly idealTree ├─┬ eslint-import-resolver-node@0.3.4
20 silly idealTree │ ├── debug@2.6.9
20 silly idealTree │ └── ms@2.0.0
20 silly idealTree ├─┬ eslint-module-utils@2.6.0
20 silly idealTree │ ├── debug@2.6.9
20 silly idealTree │ └── ms@2.0.0
20 silly idealTree ├─┬ eslint-plugin-import@2.22.0
20 silly idealTree │ ├── debug@2.6.9
20 silly idealTree │ ├── ms@2.0.0
20 silly idealTree │ └── resolve@1.17.0
20 silly idealTree ├── eslint-plugin-json@1.4.0
20 silly idealTree ├── eslint-plugin-mozilla@0.14.0
20 silly idealTree ├── eslint-plugin-no-unsanitized@3.1.2
20 silly idealTree ├── eslint-scope@5.1.0
20 silly idealTree ├── eslint-utils@1.4.2
20 silly idealTree ├── eslint-visitor-keys@1.0.0
20 silly idealTree ├─┬ eslint@5.16.0
20 silly idealTree │ ├── ajv@6.10.0
20 silly idealTree │ ├── doctrine@3.0.0
20 silly idealTree │ ├── eslint-scope@4.0.3
20 silly idealTree │ ├── file-entry-cache@5.0.1
20 silly idealTree │ ├── flat-cache@2.0.1
20 silly idealTree │ ├── js-yaml@3.13.1
20 silly idealTree │ ├── mkdirp@0.5.5
20 silly idealTree │ └── write@1.0.3
20 silly idealTree ├─┬ espree@5.0.1
20 silly idealTree │ └── acorn@6.4.1
20 silly idealTree ├── esprima@4.0.1
20 silly idealTree ├── esquery@1.0.1
20 silly idealTree ├── esrecurse@4.2.1
20 silly idealTree ├── estraverse@4.2.0
20 silly idealTree ├── esutils@2.0.2
20 silly idealTree ├── event-to-promise@0.8.0
20 silly idealTree ├── events@3.1.0
20 silly idealTree ├── evp_bytestokey@1.0.3
20 silly idealTree ├─┬ execa@4.0.3
20 silly idealTree │ ├── cross-spawn@7.0.3
20 silly idealTree │ ├── mimic-fn@2.1.0
20 silly idealTree │ ├── onetime@5.1.0
20 silly idealTree │ ├── path-key@3.1.1
20 silly idealTree │ ├── shebang-command@2.0.0
20 silly idealTree │ ├── shebang-regex@3.0.0
20 silly idealTree │ └── which@2.0.2
20 silly idealTree ├─┬ expand-brackets@2.1.4
20 silly idealTree │ ├── debug@2.6.9
20 silly idealTree │ ├── define-property@0.2.5
20 silly idealTree │ ├── extend-shallow@2.0.1
20 silly idealTree │ └── ms@2.0.0
20 silly idealTree ├── expand-tilde@2.0.2
20 silly idealTree ├─┬ extend-shallow@3.0.2
20 silly idealTree │ └── is-extendable@1.0.1
20 silly idealTree ├── extend@3.0.2
20 silly idealTree ├── external-editor@3.0.3
20 silly idealTree ├─┬ extglob@2.0.4
20 silly idealTree │ ├── define-property@1.0.0
20 silly idealTree │ ├── extend-shallow@2.0.1
20 silly idealTree │ ├── is-accessor-descriptor@1.0.0
20 silly idealTree │ ├── is-data-descriptor@1.0.0
20 silly idealTree │ └── is-descriptor@1.0.2
20 silly idealTree ├── extsprintf@1.3.0
20 silly idealTree ├── fast-deep-equal@2.0.1
20 silly idealTree ├── fast-json-patch@2.2.1
20 silly idealTree ├── fast-json-stable-stringify@2.0.0
20 silly idealTree ├── fast-levenshtein@2.0.6
20 silly idealTree ├── fast-redact@2.0.0
20 silly idealTree ├── fast-safe-stringify@2.0.7
20 silly idealTree ├── fd-slicer@1.1.0
20 silly idealTree ├── figgy-pudding@3.5.2
20 silly idealTree ├── figures@2.0.0
20 silly idealTree ├─┬ fill-range@4.0.0
20 silly idealTree │ └── extend-shallow@2.0.1
20 silly idealTree ├─┬ find-cache-dir@2.1.0
20 silly idealTree │ ├── find-up@3.0.0
20 silly idealTree │ ├── locate-path@3.0.0
20 silly idealTree │ ├── make-dir@2.1.0
20 silly idealTree │ ├── p-limit@2.3.0
20 silly idealTree │ ├── p-locate@3.0.0
20 silly idealTree │ ├── p-try@2.2.0
20 silly idealTree │ ├── pify@4.0.1
20 silly idealTree │ └── pkg-dir@3.0.0
20 silly idealTree ├── find-up@2.1.0
20 silly idealTree ├── findup-sync@3.0.0
20 silly idealTree ├─┬ firefox-profile@1.3.1
20 silly idealTree │ └── async@2.5.0
20 silly idealTree ├── first-chunk-stream@3.0.0
20 silly idealTree ├── flatstr@1.0.12
20 silly idealTree ├── flatted@2.0.0
20 silly idealTree ├── fluent-syntax@0.13.0
20 silly idealTree ├── flush-write-stream@1.1.1
20 silly idealTree ├── for-in@1.0.2
20 silly idealTree ├── forever-agent@0.6.1
20 silly idealTree ├── form-data@2.3.3
20 silly idealTree ├── fragment-cache@0.2.1
20 silly idealTree ├── from2@2.3.0
20 silly idealTree ├── fs-constants@1.0.0
20 silly idealTree ├── fs-extra@4.0.3
20 silly idealTree ├── fs-write-stream-atomic@1.0.10
20 silly idealTree ├── fs.realpath@1.0.0
20 silly idealTree ├── fsevents@2.1.3
20 silly idealTree ├── function-bind@1.1.1
20 silly idealTree ├── functional-red-black-tree@1.0.1
20 silly idealTree ├─┬ fx-runner@1.0.12
20 silly idealTree │ ├── commander@2.9.0
20 silly idealTree │ ├── isexe@1.1.2
20 silly idealTree │ └── which@1.2.4
20 silly idealTree ├── get-caller-file@2.0.5
20 silly idealTree ├── get-stream@5.1.0
20 silly idealTree ├── get-value@2.0.6
20 silly idealTree ├── getpass@0.1.7
20 silly idealTree ├── git-rev-sync@2.0.0
20 silly idealTree ├─┬ glob-parent@3.1.0
20 silly idealTree │ └── is-glob@3.1.0
20 silly idealTree ├── glob@7.1.3
20 silly idealTree ├── global-dirs@2.0.1
20 silly idealTree ├─┬ global-modules@2.0.0
20 silly idealTree │ └── global-prefix@3.0.0
20 silly idealTree ├── global-prefix@1.0.2
20 silly idealTree ├── globals@11.9.0
20 silly idealTree ├─┬ got@9.6.0
20 silly idealTree │ └── get-stream@4.1.0
20 silly idealTree ├── graceful-fs@4.1.15
20 silly idealTree ├── graceful-readlink@1.0.1
20 silly idealTree ├── growly@1.3.0
20 silly idealTree ├── har-schema@2.0.0
20 silly idealTree ├── har-validator@5.1.3
20 silly idealTree ├── has-flag@3.0.0
20 silly idealTree ├── has-symbols@1.0.0
20 silly idealTree ├── has-value@1.0.0
20 silly idealTree ├─┬ has-values@1.0.0
20 silly idealTree │ └── kind-of@4.0.0
20 silly idealTree ├── has-yarn@2.1.0
20 silly idealTree ├── has@1.0.3
20 silly idealTree ├── hash-base@3.0.4
20 silly idealTree ├── hash.js@1.1.7
20 silly idealTree ├── hmac-drbg@1.0.1
20 silly idealTree ├── homedir-polyfill@1.0.3
20 silly idealTree ├── hosted-git-info@2.7.1
20 silly idealTree ├── htmlparser2@3.9.2
20 silly idealTree ├── http-cache-semantics@4.1.0
20 silly idealTree ├── http-signature@1.2.0
20 silly idealTree ├── https-browserify@1.0.0
20 silly idealTree ├── human-signals@1.1.1
20 silly idealTree ├── iconv-lite@0.4.24
20 silly idealTree ├── ieee754@1.1.13
20 silly idealTree ├── iferr@0.1.5
20 silly idealTree ├── ignore@4.0.6
20 silly idealTree ├─┬ import-fresh@3.0.0
20 silly idealTree │ └── resolve-from@4.0.0
20 silly idealTree ├── import-lazy@2.1.0
20 silly idealTree ├─┬ import-local@2.0.0
20 silly idealTree │ ├── find-up@3.0.0
20 silly idealTree │ ├── locate-path@3.0.0
20 silly idealTree │ ├── p-limit@2.3.0
20 silly idealTree │ ├── p-locate@3.0.0
20 silly idealTree │ ├── p-try@2.2.0
20 silly idealTree │ └── pkg-dir@3.0.0
20 silly idealTree ├── imurmurhash@0.1.4
20 silly idealTree ├── infer-owner@1.0.4
20 silly idealTree ├── inflight@1.0.6
20 silly idealTree ├── inherits@2.0.3
20 silly idealTree ├── ini-parser@0.0.2
20 silly idealTree ├── ini@1.3.5
20 silly idealTree ├─┬ inquirer@6.3.1
20 silly idealTree │ ├── ansi-escapes@3.2.0
20 silly idealTree │ ├── ansi-regex@4.1.0
20 silly idealTree │ ├── chalk@2.4.2
20 silly idealTree │ └── strip-ansi@5.2.0
20 silly idealTree ├── interpret@1.4.0
20 silly idealTree ├── invert-kv@3.0.1
20 silly idealTree ├── is-absolute@0.1.7
20 silly idealTree ├─┬ is-accessor-descriptor@0.1.6
20 silly idealTree │ └── kind-of@3.2.2
20 silly idealTree ├── is-arguments@1.0.4
20 silly idealTree ├── is-arrayish@0.2.1
20 silly idealTree ├── is-binary-path@1.0.1
20 silly idealTree ├── is-buffer@1.1.6
20 silly idealTree ├── is-builtin-module@1.0.0
20 silly idealTree ├── is-callable@1.1.4
20 silly idealTree ├── is-ci@2.0.0
20 silly idealTree ├─┬ is-data-descriptor@0.1.4
20 silly idealTree │ └── kind-of@3.2.2
20 silly idealTree ├── is-date-object@1.0.1
20 silly idealTree ├─┬ is-descriptor@0.1.6
20 silly idealTree │ └── kind-of@5.1.0
20 silly idealTree ├── is-docker@2.0.0
20 silly idealTree ├── is-extendable@0.1.1
20 silly idealTree ├── is-extglob@2.1.1
20 silly idealTree ├── is-fullwidth-code-point@2.0.0
20 silly idealTree ├── is-glob@4.0.1
20 silly idealTree ├── is-installed-globally@0.3.2
20 silly idealTree ├── is-mergeable-object@1.1.1
20 silly idealTree ├── is-npm@4.0.0
20 silly idealTree ├─┬ is-number@3.0.0
20 silly idealTree │ └── kind-of@3.2.2
20 silly idealTree ├── is-obj@2.0.0
20 silly idealTree ├── is-path-inside@3.0.2
20 silly idealTree ├── is-plain-object@2.0.4
20 silly idealTree ├── is-promise@2.1.0
20 silly idealTree ├── is-regex@1.0.4
20 silly idealTree ├── is-relative@0.1.3
20 silly idealTree ├── is-stream@2.0.0
20 silly idealTree ├── is-string@1.0.5
20 silly idealTree ├── is-symbol@1.0.2
20 silly idealTree ├── is-typedarray@1.0.0
20 silly idealTree ├── is-utf8@0.2.1
20 silly idealTree ├── is-windows@1.0.2
20 silly idealTree ├── is-wsl@2.2.0
20 silly idealTree ├── is-yarn-global@0.3.0
20 silly idealTree ├── isarray@1.0.0
20 silly idealTree ├── isexe@2.0.0
20 silly idealTree ├── isobject@3.0.1
20 silly idealTree ├── isstream@0.1.2
20 silly idealTree ├── jed@1.1.1
20 silly idealTree ├── jetpack-id@1.0.0
20 silly idealTree ├── js-select@0.6.0
20 silly idealTree ├── js-tokens@4.0.0
20 silly idealTree ├── js-yaml@3.13.1
20 silly idealTree ├── jsbn@0.1.1
20 silly idealTree ├── json-buffer@3.0.0
20 silly idealTree ├── json-merge-patch@0.2.3
20 silly idealTree ├── json-parse-better-errors@1.0.2
20 silly idealTree ├── json-schema-traverse@0.4.1
20 silly idealTree ├── json-schema@0.2.3
20 silly idealTree ├── json-stable-stringify-without-jsonify@1.0.1
20 silly idealTree ├── json-stringify-safe@5.0.1
20 silly idealTree ├─┬ json5@1.0.1
20 silly idealTree │ └── minimist@1.2.5
20 silly idealTree ├── jsonc-parser@2.1.0
20 silly idealTree ├── jsonfile@4.0.0
20 silly idealTree ├── jsonify@0.0.0
20 silly idealTree ├── JSONSelect@0.2.1
20 silly idealTree ├── jsonwebtoken@8.5.1
20 silly idealTree ├── jsprim@1.4.1
20 silly idealTree ├── jszip@2.6.1
20 silly idealTree ├── jwa@1.4.1
20 silly idealTree ├── jws@3.2.2
20 silly idealTree ├── keyv@3.1.0
20 silly idealTree ├── kind-of@6.0.3
20 silly idealTree ├── latest-version@5.1.0
20 silly idealTree ├── lazystream@1.0.0
20 silly idealTree ├── lcid@3.1.1
20 silly idealTree ├── levn@0.3.0
20 silly idealTree ├─┬ lighthouse-logger@1.2.0
20 silly idealTree │ ├── debug@2.6.9
20 silly idealTree │ └── ms@2.0.0
20 silly idealTree ├── lines-and-columns@1.1.6
20 silly idealTree ├── load-json-file@2.0.0
20 silly idealTree ├── loader-runner@2.4.0
20 silly idealTree ├── loader-utils@1.2.3
20 silly idealTree ├── locate-path@2.0.0
20 silly idealTree ├── lodash.includes@4.3.0
20 silly idealTree ├── lodash.isboolean@3.0.3
20 silly idealTree ├── lodash.isinteger@4.0.4
20 silly idealTree ├── lodash.isnumber@3.0.3
20 silly idealTree ├── lodash.isplainobject@4.0.6
20 silly idealTree ├── lodash.isstring@4.0.1
20 silly idealTree ├── lodash.once@4.1.1
20 silly idealTree ├── lodash.sortby@4.7.0
20 silly idealTree ├── lodash@4.17.19
20 silly idealTree ├── lowercase-keys@1.0.1
20 silly idealTree ├── lru-cache@5.1.1
20 silly idealTree ├─┬ make-dir@3.1.0
20 silly idealTree │ └── semver@6.3.0
20 silly idealTree ├── map-age-cleaner@0.1.3
20 silly idealTree ├── map-cache@0.2.2
20 silly idealTree ├── map-visit@1.0.0
20 silly idealTree ├── marky@1.2.1
20 silly idealTree ├── md5.js@1.3.5
20 silly idealTree ├── mdn-browser-compat-data@1.0.25
20 silly idealTree ├─┬ mem@5.1.1
20 silly idealTree │ └── mimic-fn@2.1.0
20 silly idealTree ├── memory-fs@0.4.1
20 silly idealTree ├── memorystream@0.3.1
20 silly idealTree ├── merge-stream@2.0.0
20 silly idealTree ├── micromatch@3.1.10
20 silly idealTree ├── miller-rabin@4.0.1
20 silly idealTree ├── mime-db@1.44.0
20 silly idealTree ├── mime-types@2.1.27
20 silly idealTree ├── mimic-fn@1.2.0
20 silly idealTree ├── mimic-response@1.0.1
20 silly idealTree ├── minimalistic-assert@1.0.1
20 silly idealTree ├── minimalistic-crypto-utils@1.0.1
20 silly idealTree ├── minimatch@3.0.4
20 silly idealTree ├── minimist@1.2.5
20 silly idealTree ├── mississippi@3.0.0
20 silly idealTree ├─┬ mixin-deep@1.3.2
20 silly idealTree │ └── is-extendable@1.0.1
20 silly idealTree ├── mkdirp@0.5.5
20 silly idealTree ├── moment@2.27.0
20 silly idealTree ├── move-concurrently@1.0.1
20 silly idealTree ├── ms@2.1.1
20 silly idealTree ├── multimatch@4.0.0
20 silly idealTree ├── mute-stream@0.0.7
20 silly idealTree ├─┬ mv@2.1.1
20 silly idealTree │ ├── glob@6.0.4
20 silly idealTree │ └── rimraf@2.4.5
20 silly idealTree ├── mz@2.7.0
20 silly idealTree ├── nan@2.14.1
20 silly idealTree ├── nanomatch@1.2.13
20 silly idealTree ├── natural-compare-lite@1.4.0
20 silly idealTree ├── natural-compare@1.4.0
20 silly idealTree ├── ncp@2.0.0
20 silly idealTree ├── neo-async@2.6.1
20 silly idealTree ├── next-tick@1.1.0
20 silly idealTree ├── nice-try@1.0.5
20 silly idealTree ├── node-forge@0.7.6
20 silly idealTree ├─┬ node-libs-browser@2.2.1
20 silly idealTree │ ├── buffer@4.9.2
20 silly idealTree │ └── punycode@1.4.1
20 silly idealTree ├─┬ node-notifier@6.0.0
20 silly idealTree │ └── semver@6.3.0
20 silly idealTree ├── normalize-package-data@2.4.0
20 silly idealTree ├── normalize-path@2.1.1
20 silly idealTree ├── normalize-url@4.5.0
20 silly idealTree ├─┬ npm-run-all@4.1.5
20 silly idealTree │ ├── load-json-file@4.0.0
20 silly idealTree │ ├── parse-json@4.0.0
20 silly idealTree │ ├── path-type@3.0.0
20 silly idealTree │ ├── pify@3.0.0
20 silly idealTree │ └── read-pkg@3.0.0
20 silly idealTree ├─┬ npm-run-path@4.0.1
20 silly idealTree │ └── path-key@3.1.1
20 silly idealTree ├── nth-check@1.0.2
20 silly idealTree ├── oauth-sign@0.9.0
20 silly idealTree ├── object-assign@4.1.1
20 silly idealTree ├─┬ object-copy@0.1.0
20 silly idealTree │ ├── define-property@0.2.5
20 silly idealTree │ └── kind-of@3.2.2
20 silly idealTree ├── object-inspect@1.8.0
20 silly idealTree ├─┬ object-is@1.1.2
20 silly idealTree │ ├── es-abstract@1.17.6
20 silly idealTree │ ├── es-to-primitive@1.2.1
20 silly idealTree │ ├── has-symbols@1.0.1
20 silly idealTree │ ├── is-callable@1.2.0
20 silly idealTree │ ├── is-regex@1.1.0
20 silly idealTree │ └── object-keys@1.1.1
20 silly idealTree ├── object-keys@1.0.12
20 silly idealTree ├── object-visit@1.0.1
20 silly idealTree ├── object.assign@4.1.0
20 silly idealTree ├── object.pick@1.3.0
20 silly idealTree ├─┬ object.values@1.1.1
20 silly idealTree │ ├── es-abstract@1.17.6
20 silly idealTree │ ├── es-to-primitive@1.2.1
20 silly idealTree │ ├── has-symbols@1.0.1
20 silly idealTree │ ├── is-callable@1.2.0
20 silly idealTree │ ├── is-regex@1.1.0
20 silly idealTree │ └── object-keys@1.1.1
20 silly idealTree ├── once@1.4.0
20 silly idealTree ├── onetime@2.0.1
20 silly idealTree ├── open@7.0.0
20 silly idealTree ├── optionator@0.8.2
20 silly idealTree ├── os-browserify@0.3.0
20 silly idealTree ├── os-locale@5.0.0
20 silly idealTree ├── os-shim@0.1.3
20 silly idealTree ├── os-tmpdir@1.0.2
20 silly idealTree ├── p-cancelable@1.1.0
20 silly idealTree ├── p-defer@1.0.0
20 silly idealTree ├── p-is-promise@2.1.0
20 silly idealTree ├── p-limit@1.3.0
20 silly idealTree ├── p-locate@2.0.0
20 silly idealTree ├── p-try@1.0.0
20 silly idealTree ├─┬ package-json@6.5.0
20 silly idealTree │ └── semver@6.3.0
20 silly idealTree ├── pako@1.0.11
20 silly idealTree ├── parallel-transform@1.2.0
20 silly idealTree ├─┬ parent-module@1.0.1
20 silly idealTree │ └── callsites@3.1.0
20 silly idealTree ├── parse-asn1@5.1.5
20 silly idealTree ├── parse-json@2.2.0
20 silly idealTree ├── parse-passwd@1.0.0
20 silly idealTree ├── parse5@3.0.3
20 silly idealTree ├── pascalcase@0.1.1
20 silly idealTree ├── path-browserify@0.0.1
20 silly idealTree ├── path-dirname@1.0.2
20 silly idealTree ├── path-exists@3.0.0
20 silly idealTree ├── path-is-absolute@1.0.1
20 silly idealTree ├── path-is-inside@1.0.2
20 silly idealTree ├── path-key@2.0.1
20 silly idealTree ├── path-parse@1.0.6
20 silly idealTree ├── path-type@2.0.0
20 silly idealTree ├── pbkdf2@3.0.17
20 silly idealTree ├── pend@1.2.0
20 silly idealTree ├── performance-now@2.1.0
20 silly idealTree ├── pidtree@0.3.0
20 silly idealTree ├── pify@2.3.0
20 silly idealTree ├── pino-std-serializers@2.4.2
20 silly idealTree ├── pino@6.3.2
20 silly idealTree ├── pkg-dir@2.0.0
20 silly idealTree ├── posix-character-classes@0.1.1
20 silly idealTree ├─┬ postcss@7.0.32
20 silly idealTree │ ├─┬ chalk@2.4.2
20 silly idealTree │ │ └── supports-color@5.5.0
20 silly idealTree │ ├── source-map@0.6.1
20 silly idealTree │ └── supports-color@6.1.0
20 silly idealTree ├── prelude-ls@1.1.2
20 silly idealTree ├── prepend-http@2.0.0
20 silly idealTree ├── probe-image-size@5.0.0
20 silly idealTree ├── process-nextick-args@2.0.0
20 silly idealTree ├── process@0.11.10
20 silly idealTree ├── progress@2.0.3
20 silly idealTree ├── promise-inflight@1.0.1
20 silly idealTree ├── prr@1.0.1
20 silly idealTree ├── psl@1.8.0
20 silly idealTree ├── public-encrypt@4.0.3
20 silly idealTree ├── pump@3.0.0
20 silly idealTree ├─┬ pumpify@1.5.1
20 silly idealTree │ └── pump@2.0.1
20 silly idealTree ├── punycode@2.1.1
20 silly idealTree ├── qs@6.5.2
20 silly idealTree ├── querystring-es3@0.2.1
20 silly idealTree ├── querystring@0.2.0
20 silly idealTree ├── quick-format-unescaped@4.0.1
20 silly idealTree ├── randombytes@2.1.0
20 silly idealTree ├── randomfill@1.0.4
20 silly idealTree ├── rc@1.2.8
20 silly idealTree ├── read-pkg-up@2.0.0
20 silly idealTree ├── read-pkg@2.0.0
20 silly idealTree ├── readable-stream@2.3.6
20 silly idealTree ├── readdirp@2.2.1
20 silly idealTree ├── rechoir@0.6.2
20 silly idealTree ├── regenerator-runtime@0.13.5
20 silly idealTree ├── regex-not@1.0.2
20 silly idealTree ├─┬ regexp.prototype.flags@1.3.0
20 silly idealTree │ ├── es-abstract@1.17.6
20 silly idealTree │ ├── es-to-primitive@1.2.1
20 silly idealTree │ ├── has-symbols@1.0.1
20 silly idealTree │ ├── is-callable@1.2.0
20 silly idealTree │ ├── is-regex@1.1.0
20 silly idealTree │ └── object-keys@1.1.1
20 silly idealTree ├── regexpp@2.0.1
20 silly idealTree ├── registry-auth-token@4.1.1
20 silly idealTree ├── registry-url@5.1.0
20 silly idealTree ├─┬ relaxed-json@1.0.3
20 silly idealTree │ └── chalk@2.4.2
20 silly idealTree ├── remove-trailing-separator@1.1.0
20 silly idealTree ├── repeat-element@1.1.3
20 silly idealTree ├── repeat-string@1.6.1
20 silly idealTree ├── request@2.88.2
20 silly idealTree ├── require-directory@2.1.1
20 silly idealTree ├── require-main-filename@2.0.0
20 silly idealTree ├─┬ resolve-cwd@2.0.0
20 silly idealTree │ └── resolve-from@3.0.0
20 silly idealTree ├─┬ resolve-dir@1.0.1
20 silly idealTree │ └── global-modules@1.0.0
20 silly idealTree ├── resolve-from@4.0.0
20 silly idealTree ├── resolve-url@0.2.1
20 silly idealTree ├── resolve@1.14.2
20 silly idealTree ├── responselike@1.0.2
20 silly idealTree ├── restore-cursor@2.0.0
20 silly idealTree ├── ret@0.1.15
20 silly idealTree ├── rimraf@2.6.3
20 silly idealTree ├── ripemd160@2.0.2
20 silly idealTree ├── run-async@2.3.0
20 silly idealTree ├── run-queue@1.0.3
20 silly idealTree ├── rxjs@6.5.2
20 silly idealTree ├── safe-buffer@5.1.2
20 silly idealTree ├── safe-json-stringify@1.2.0
20 silly idealTree ├── safe-regex@1.1.0
20 silly idealTree ├── safer-buffer@2.1.2
20 silly idealTree ├── sax@1.2.4
20 silly idealTree ├─┬ schema-utils@1.0.0
20 silly idealTree │ └── ajv-keywords@3.4.1
20 silly idealTree ├─┬ semver-diff@3.1.1
20 silly idealTree │ └── semver@6.3.0
20 silly idealTree ├── semver@5.6.0
20 silly idealTree ├── serialize-javascript@2.1.2
20 silly idealTree ├── set-blocking@2.0.0
20 silly idealTree ├─┬ set-value@2.0.1
20 silly idealTree │ └── extend-shallow@2.0.1
20 silly idealTree ├── setimmediate@1.0.5
20 silly idealTree ├── sha.js@2.4.11
20 silly idealTree ├── shebang-command@1.2.0
20 silly idealTree ├── shebang-regex@1.0.0
20 silly idealTree ├── shell-quote@1.6.1
20 silly idealTree ├── shelljs@0.7.7
20 silly idealTree ├── shellwords@0.1.1
20 silly idealTree ├─┬ sign-addon@2.0.5
20 silly idealTree │ ├── core-js@3.6.4
20 silly idealTree │ ├── es6-promisify@6.0.2
20 silly idealTree │ ├── punycode@1.4.1
20 silly idealTree │ ├── request@2.88.0
20 silly idealTree │ └── tough-cookie@2.4.3
20 silly idealTree ├── signal-exit@3.0.2
20 silly idealTree ├── slice-ansi@2.1.0
20 silly idealTree ├─┬ snapdragon-node@2.1.1
20 silly idealTree │ ├── define-property@1.0.0
20 silly idealTree │ ├── is-accessor-descriptor@1.0.0
20 silly idealTree │ ├── is-data-descriptor@1.0.0
20 silly idealTree │ └── is-descriptor@1.0.2
20 silly idealTree ├─┬ snapdragon-util@3.0.1
20 silly idealTree │ └── kind-of@3.2.2
20 silly idealTree ├─┬ snapdragon@0.8.2
20 silly idealTree │ ├── debug@2.6.9
20 silly idealTree │ ├── define-property@0.2.5
20 silly idealTree │ ├── extend-shallow@2.0.1
20 silly idealTree │ └── ms@2.0.0
20 silly idealTree ├── sonic-boom@1.0.1
20 silly idealTree ├── source-list-map@2.0.1
20 silly idealTree ├── source-map-resolve@0.5.2
20 silly idealTree ├─┬ source-map-support@0.5.16
20 silly idealTree │ └── source-map@0.6.1
20 silly idealTree ├── source-map-url@0.4.0
20 silly idealTree ├── source-map@0.5.7
20 silly idealTree ├── spawn-sync@1.0.15
20 silly idealTree ├── spdx-correct@3.1.0
20 silly idealTree ├── spdx-exceptions@2.2.0
20 silly idealTree ├── spdx-expression-parse@3.0.0
20 silly idealTree ├── spdx-license-ids@3.0.3
20 silly idealTree ├── split-string@3.1.0
20 silly idealTree ├── split@0.3.3
20 silly idealTree ├── sprintf-js@1.0.3
20 silly idealTree ├── sshpk@1.16.1
20 silly idealTree ├── ssri@6.0.1
20 silly idealTree ├─┬ static-extend@0.1.2
20 silly idealTree │ └── define-property@0.2.5
20 silly idealTree ├── stream-browserify@2.0.2
20 silly idealTree ├── stream-each@1.2.3
20 silly idealTree ├── stream-http@2.8.3
20 silly idealTree ├─┬ stream-parser@0.3.1
20 silly idealTree │ ├── debug@2.6.9
20 silly idealTree │ └── ms@2.0.0
20 silly idealTree ├── stream-shift@1.0.1
20 silly idealTree ├── stream-to-array@2.3.0
20 silly idealTree ├─┬ stream-to-promise@2.2.0
20 silly idealTree │ ├── end-of-stream@1.1.0
20 silly idealTree │ └── once@1.3.3
20 silly idealTree ├── string_decoder@1.1.1
20 silly idealTree ├── string-width@2.1.1
20 silly idealTree ├── string.prototype.padend@3.0.0
20 silly idealTree ├─┬ string.prototype.trimend@1.0.1
20 silly idealTree │ ├── es-abstract@1.17.6
20 silly idealTree │ ├── es-to-primitive@1.2.1
20 silly idealTree │ ├── has-symbols@1.0.1
20 silly idealTree │ ├── is-callable@1.2.0
20 silly idealTree │ ├── is-regex@1.1.0
20 silly idealTree │ └── object-keys@1.1.1
20 silly idealTree ├─┬ string.prototype.trimstart@1.0.1
20 silly idealTree │ ├── es-abstract@1.17.6
20 silly idealTree │ ├── es-to-primitive@1.2.1
20 silly idealTree │ ├── has-symbols@1.0.1
20 silly idealTree │ ├── is-callable@1.2.0
20 silly idealTree │ ├── is-regex@1.1.0
20 silly idealTree │ └── object-keys@1.1.1
20 silly idealTree ├── strip-ansi@4.0.0
20 silly idealTree ├── strip-bom-buf@2.0.0
20 silly idealTree ├── strip-bom-stream@4.0.0
20 silly idealTree ├── strip-bom@3.0.0
20 silly idealTree ├── strip-final-newline@2.0.0
20 silly idealTree ├── strip-json-comments@2.0.1
20 silly idealTree ├── supports-color@5.5.0
20 silly idealTree ├─┬ table@5.4.0
20 silly idealTree │ ├── ajv@6.10.0
20 silly idealTree │ ├── ansi-regex@4.1.0
20 silly idealTree │ ├── string-width@3.1.0
20 silly idealTree │ └── strip-ansi@5.2.0
20 silly idealTree ├── tapable@1.1.3
20 silly idealTree ├── tar-stream@1.6.2
20 silly idealTree ├── term-size@2.2.0
20 silly idealTree ├─┬ terser-webpack-plugin@1.4.3
20 silly idealTree │ ├── is-wsl@1.1.0
20 silly idealTree │ └── source-map@0.6.1
20 silly idealTree ├─┬ terser@4.6.12
20 silly idealTree │ └── source-map@0.6.1
20 silly idealTree ├── text-table@0.2.0
20 silly idealTree ├── thenify-all@1.6.0
20 silly idealTree ├── thenify@3.3.1
20 silly idealTree ├── through@2.3.8
20 silly idealTree ├── through2@2.0.5
20 silly idealTree ├── timers-browserify@2.0.11
20 silly idealTree ├── tmp@0.0.33
20 silly idealTree ├── to-arraybuffer@1.0.1
20 silly idealTree ├── to-buffer@1.1.1
20 silly idealTree ├─┬ to-object-path@0.3.0
20 silly idealTree │ └── kind-of@3.2.2
20 silly idealTree ├── to-readable-stream@1.0.0
20 silly idealTree ├── to-regex-range@2.1.1
20 silly idealTree ├── to-regex@3.0.2
20 silly idealTree ├── tosource@1.0.0
20 silly idealTree ├── tough-cookie@2.5.0
20 silly idealTree ├── tr46@2.0.2
20 silly idealTree ├── traverse@0.4.6
20 silly idealTree ├── ts-loader@5.4.5
20 silly idealTree ├── tsconfig-paths@3.9.0
20 silly idealTree ├── tslib@1.9.3
20 silly idealTree ├─┬ tslint-eslint-rules@5.4.0
20 silly idealTree │ ├── doctrine@0.7.2
20 silly idealTree │ ├── esutils@1.1.6
20 silly idealTree │ ├── isarray@0.0.1
20 silly idealTree │ ├── tslib@1.9.0
20 silly idealTree │ └── tsutils@3.6.0
20 silly idealTree ├── tslint@5.20.1
20 silly idealTree ├── tsutils@2.29.0
20 silly idealTree ├── tty-browserify@0.0.0
20 silly idealTree ├── tunnel-agent@0.6.0
20 silly idealTree ├── tweetnacl@0.14.5
20 silly idealTree ├── type-check@0.3.2
20 silly idealTree ├── type-detect@4.0.8
20 silly idealTree ├── type-fest@0.8.1
20 silly idealTree ├── typedarray-to-buffer@3.1.5
20 silly idealTree ├── typedarray@0.0.6
20 silly idealTree ├── typescript@3.9.6
20 silly idealTree ├── union-value@1.0.1
20 silly idealTree ├── unique-filename@1.1.1
20 silly idealTree ├── unique-slug@2.0.2
20 silly idealTree ├── unique-string@2.0.0
20 silly idealTree ├── universalify@0.1.2
20 silly idealTree ├─┬ unset-value@1.0.0
20 silly idealTree │ ├─┬ has-value@0.3.1
20 silly idealTree │ │ └── isobject@2.1.0
20 silly idealTree │ └── has-values@0.1.4
20 silly idealTree ├── upath@1.2.0
20 silly idealTree ├─┬ update-notifier@4.0.0
20 silly idealTree │ ├── ansi-styles@4.2.1
20 silly idealTree │ ├── chalk@3.0.0
20 silly idealTree │ ├── color-convert@2.0.1
20 silly idealTree │ ├── color-name@1.1.4
20 silly idealTree │ ├── has-flag@4.0.0
20 silly idealTree │ └── supports-color@7.1.0
20 silly idealTree ├── uri-js@4.2.2
20 silly idealTree ├── urix@0.1.0
20 silly idealTree ├── url-parse-lax@3.0.0
20 silly idealTree ├─┬ url@0.11.0
20 silly idealTree │ └── punycode@1.3.2
20 silly idealTree ├── use@3.1.1
20 silly idealTree ├── util-deprecate@1.0.2
20 silly idealTree ├── util@0.11.1
20 silly idealTree ├── uuid@3.4.0
20 silly idealTree ├── v8-compile-cache@2.1.1
20 silly idealTree ├── validate-npm-package-license@3.0.4
20 silly idealTree ├── verror@1.10.0
20 silly idealTree ├── vm-browserify@1.1.2
20 silly idealTree ├─┬ vscode-json-languageservice@3.3.0
20 silly idealTree │ └── vscode-languageserver-types@3.15.0-next.2
20 silly idealTree ├── vscode-nls@4.1.1
20 silly idealTree ├── vscode-uri@2.0.1
20 silly idealTree ├── watchpack@1.6.1
20 silly idealTree ├── wcwidth@1.0.1
20 silly idealTree ├─┬ web-ext@4.3.0
20 silly idealTree │ ├── import-fresh@3.2.1
20 silly idealTree │ ├── mkdirp@1.0.4
20 silly idealTree │ ├── parse-json@5.0.0
20 silly idealTree │ ├── source-map-support@0.5.19
20 silly idealTree │ ├── source-map@0.6.1
20 silly idealTree │ ├── strip-bom@4.0.0
20 silly idealTree │ ├── strip-json-comments@3.0.1
20 silly idealTree │ └── tmp@0.1.0
20 silly idealTree ├── webidl-conversions@5.0.0
20 silly idealTree ├─┬ webpack-cli@3.3.12
20 silly idealTree │ ├── ansi-regex@4.1.0
20 silly idealTree │ ├─┬ chalk@2.4.2
20 silly idealTree │ │ └── supports-color@5.5.0
20 silly idealTree │ ├── cliui@5.0.0
20 silly idealTree │ ├── decamelize@1.2.0
20 silly idealTree │ ├── emojis-list@3.0.0
20 silly idealTree │ ├── enhanced-resolve@4.2.0
20 silly idealTree │ ├── find-up@3.0.0
20 silly idealTree │ ├── loader-utils@1.4.0
20 silly idealTree │ ├── locate-path@3.0.0
20 silly idealTree │ ├── memory-fs@0.5.0
20 silly idealTree │ ├── p-limit@2.3.0
20 silly idealTree │ ├── p-locate@3.0.0
20 silly idealTree │ ├── p-try@2.2.0
20 silly idealTree │ ├── string-width@3.1.0
20 silly idealTree │ ├── strip-ansi@5.2.0
20 silly idealTree │ ├── supports-color@6.1.0
20 silly idealTree │ ├── wrap-ansi@5.1.0
20 silly idealTree │ ├── yargs-parser@13.1.2
20 silly idealTree │ └── yargs@13.3.2
20 silly idealTree ├─┬ webpack-sources@1.4.3
20 silly idealTree │ └── source-map@0.6.1
20 silly idealTree ├─┬ webpack@4.43.0
20 silly idealTree │ ├── acorn@6.4.1
20 silly idealTree │ ├── ajv-keywords@3.4.1
20 silly idealTree │ └── eslint-scope@4.0.3
20 silly idealTree ├── whatwg-url@8.1.0
20 silly idealTree ├── when@3.7.7
20 silly idealTree ├── which-module@2.0.0
20 silly idealTree ├── which@1.3.1
20 silly idealTree ├─┬ widest-line@3.1.0
20 silly idealTree │ ├── ansi-regex@5.0.0
20 silly idealTree │ ├── emoji-regex@8.0.0
20 silly idealTree │ ├── is-fullwidth-code-point@3.0.0
20 silly idealTree │ ├── string-width@4.2.0
20 silly idealTree │ └── strip-ansi@6.0.0
20 silly idealTree ├── winreg@0.0.12
20 silly idealTree ├── wordwrap@1.0.0
20 silly idealTree ├── worker-farm@1.7.0
20 silly idealTree ├─┬ wrap-ansi@6.2.0
20 silly idealTree │ ├── ansi-regex@5.0.0
20 silly idealTree │ ├── ansi-styles@4.2.1
20 silly idealTree │ ├── color-convert@2.0.1
20 silly idealTree │ ├── color-name@1.1.4
20 silly idealTree │ ├── emoji-regex@8.0.0
20 silly idealTree │ ├── is-fullwidth-code-point@3.0.0
20 silly idealTree │ ├── string-width@4.2.0
20 silly idealTree │ └── strip-ansi@6.0.0
20 silly idealTree ├── wrappy@1.0.2
20 silly idealTree ├── write-file-atomic@3.0.3
20 silly idealTree ├── ws@7.2.3
20 silly idealTree ├── xdg-basedir@4.0.0
20 silly idealTree ├── xml2js@0.4.23
20 silly idealTree ├── xmlbuilder@11.0.1
20 silly idealTree ├── xregexp@4.3.0
20 silly idealTree ├── xtend@4.0.2
20 silly idealTree ├── y18n@4.0.0
20 silly idealTree ├── yallist@3.1.1
20 silly idealTree ├─┬ yargs-parser@18.1.3
20 silly idealTree │ └── decamelize@1.2.0
20 silly idealTree ├─┬ yargs@15.3.1
20 silly idealTree │ ├── ansi-regex@5.0.0
20 silly idealTree │ ├── decamelize@1.2.0
20 silly idealTree │ ├── emoji-regex@8.0.0
20 silly idealTree │ ├── find-up@4.1.0
20 silly idealTree │ ├── is-fullwidth-code-point@3.0.0
20 silly idealTree │ ├── locate-path@5.0.0
20 silly idealTree │ ├── p-limit@2.3.0
20 silly idealTree │ ├── p-locate@4.1.0
20 silly idealTree │ ├── p-try@2.2.0
20 silly idealTree │ ├── path-exists@4.0.0
20 silly idealTree │ ├── string-width@4.2.0
20 silly idealTree │ └── strip-ansi@6.0.0
20 silly idealTree ├── yauzl@2.10.0
20 silly idealTree ├─┬ zip-dir@1.0.2
20 silly idealTree │ └── async@1.5.2
20 silly idealTree └── zip-stream@1.2.0
21 silly install generateActionsToTake
22 timing stage:generateActionsToTake Completed in 804ms
23 silly diffTrees action count 70
24 silly diffTrees add abbrev@1.1.1
25 silly diffTrees add ansi-regex@2.1.1
26 silly diffTrees add aproba@1.2.0
27 silly diffTrees add balanced-match@1.0.0
28 silly diffTrees add chownr@1.1.4
29 silly diffTrees add code-point-at@1.1.0
30 silly diffTrees add concat-map@0.0.1
31 silly diffTrees add brace-expansion@1.1.11
32 silly diffTrees add console-control-strings@1.1.0
33 silly diffTrees add core-util-is@1.0.2
34 silly diffTrees add deep-extend@0.6.0
35 silly diffTrees add delegates@1.0.0
36 silly diffTrees add detect-libc@1.0.3
37 silly diffTrees add fs.realpath@1.0.0
38 silly diffTrees add has-unicode@2.0.1
39 silly diffTrees add inherits@2.0.4
40 silly diffTrees add ini@1.3.5
41 silly diffTrees add isarray@1.0.0
42 silly diffTrees add minimatch@3.0.4
43 silly diffTrees add ignore-walk@3.0.3
44 silly diffTrees add minimist@1.2.5
45 silly diffTrees add mkdirp@0.5.3
46 silly diffTrees add ms@2.1.2
47 silly diffTrees add debug@3.2.6
48 silly diffTrees add npm-normalize-package-bin@1.0.1
49 silly diffTrees add npm-bundled@1.1.1
50 silly diffTrees add npm-packlist@1.4.8
51 silly diffTrees add number-is-nan@1.0.1
52 silly diffTrees add is-fullwidth-code-point@1.0.0
53 silly diffTrees add object-assign@4.1.1
54 silly diffTrees add os-homedir@1.0.2
55 silly diffTrees add os-tmpdir@1.0.2
56 silly diffTrees add osenv@0.1.5
57 silly diffTrees add nopt@4.0.3
58 silly diffTrees add path-is-absolute@1.0.1
59 silly diffTrees add process-nextick-args@2.0.1
60 silly diffTrees add safe-buffer@5.1.2
61 silly diffTrees add safer-buffer@2.1.2
62 silly diffTrees add iconv-lite@0.4.24
63 silly diffTrees add sax@1.2.4
64 silly diffTrees add needle@2.3.3
65 silly diffTrees add semver@5.7.1
66 silly diffTrees add set-blocking@2.0.0
67 silly diffTrees add signal-exit@3.0.2
68 silly diffTrees add string_decoder@1.1.1
69 silly diffTrees add strip-ansi@3.0.1
70 silly diffTrees add string-width@1.0.2
71 silly diffTrees add strip-json-comments@2.0.1
72 silly diffTrees add rc@1.2.8
73 silly diffTrees add util-deprecate@1.0.2
74 silly diffTrees add readable-stream@2.3.7
75 silly diffTrees add are-we-there-yet@1.1.5
76 silly diffTrees add wide-align@1.1.3
77 silly diffTrees add gauge@2.7.4
78 silly diffTrees add npmlog@4.1.2
79 silly diffTrees add wrappy@1.0.2
80 silly diffTrees add once@1.4.0
81 silly diffTrees add inflight@1.0.6
82 silly diffTrees add glob@7.1.6
83 silly diffTrees add rimraf@2.7.1
84 silly diffTrees add yallist@3.1.1
85 silly diffTrees add minipass@2.9.0
86 silly diffTrees add fs-minipass@1.2.7
87 silly diffTrees add minizlib@1.3.3
88 silly diffTrees add tar@4.4.13
89 silly diffTrees add node-pre-gyp@0.14.0
90 silly diffTrees add fsevents@1.2.12
91 silly diffTrees add fsevents@2.1.3
92 silly diffTrees add extend@3.0.2
93 silly diffTrees add mdn-browser-compat-data@1.0.20
94 silly decomposeActions action count 428
95 silly decomposeActions fetch fsevents@1.2.12
96 silly decomposeActions extract fsevents@1.2.12
97 silly decomposeActions preinstall abbrev@1.1.1
98 silly decomposeActions build abbrev@1.1.1
99 silly decomposeActions install abbrev@1.1.1
100 silly decomposeActions postinstall abbrev@1.1.1
101 silly decomposeActions finalize abbrev@1.1.1
102 silly decomposeActions refresh-package-json abbrev@1.1.1
103 silly decomposeActions preinstall ansi-regex@2.1.1
104 silly decomposeActions build ansi-regex@2.1.1
105 silly decomposeActions install ansi-regex@2.1.1
106 silly decomposeActions postinstall ansi-regex@2.1.1
107 silly decomposeActions finalize ansi-regex@2.1.1
108 silly decomposeActions refresh-package-json ansi-regex@2.1.1
109 silly decomposeActions preinstall aproba@1.2.0
110 silly decomposeActions build aproba@1.2.0
111 silly decomposeActions install aproba@1.2.0
112 silly decomposeActions postinstall aproba@1.2.0
113 silly decomposeActions finalize aproba@1.2.0
114 silly decomposeActions refresh-package-json aproba@1.2.0
115 silly decomposeActions preinstall balanced-match@1.0.0
116 silly decomposeActions build balanced-match@1.0.0
117 silly decomposeActions install balanced-match@1.0.0
118 silly decomposeActions postinstall balanced-match@1.0.0
119 silly decomposeActions finalize balanced-match@1.0.0
120 silly decomposeActions refresh-package-json balanced-match@1.0.0
121 silly decomposeActions preinstall chownr@1.1.4
122 silly decomposeActions build chownr@1.1.4
123 silly decomposeActions install chownr@1.1.4
124 silly decomposeActions postinstall chownr@1.1.4
125 silly decomposeActions finalize chownr@1.1.4
126 silly decomposeActions refresh-package-json chownr@1.1.4
127 silly decomposeActions preinstall code-point-at@1.1.0
128 silly decomposeActions build code-point-at@1.1.0
129 silly decomposeActions install code-point-at@1.1.0
130 silly decomposeActions postinstall code-point-at@1.1.0
131 silly decomposeActions finalize code-point-at@1.1.0
132 silly decomposeActions refresh-package-json code-point-at@1.1.0
133 silly decomposeActions preinstall concat-map@0.0.1
134 silly decomposeActions build concat-map@0.0.1
135 silly decomposeActions install concat-map@0.0.1
136 silly decomposeActions postinstall concat-map@0.0.1
137 silly decomposeActions finalize concat-map@0.0.1
138 silly decomposeActions refresh-package-json concat-map@0.0.1
139 silly decomposeActions preinstall brace-expansion@1.1.11
140 silly decomposeActions build brace-expansion@1.1.11
141 silly decomposeActions install brace-expansion@1.1.11
142 silly decomposeActions postinstall brace-expansion@1.1.11
143 silly decomposeActions finalize brace-expansion@1.1.11
144 silly decomposeActions refresh-package-json brace-expansion@1.1.11
145 silly decomposeActions preinstall console-control-strings@1.1.0
146 silly decomposeActions build console-control-strings@1.1.0
147 silly decomposeActions install console-control-strings@1.1.0
148 silly decomposeActions postinstall console-control-strings@1.1.0
149 silly decomposeActions finalize console-control-strings@1.1.0
150 silly decomposeActions refresh-package-json console-control-strings@1.1.0
151 silly decomposeActions preinstall core-util-is@1.0.2
152 silly decomposeActions build core-util-is@1.0.2
153 silly decomposeActions install core-util-is@1.0.2
154 silly decomposeActions postinstall core-util-is@1.0.2
155 silly decomposeActions finalize core-util-is@1.0.2
156 silly decomposeActions refresh-package-json core-util-is@1.0.2
157 silly decomposeActions preinstall deep-extend@0.6.0
158 silly decomposeActions build deep-extend@0.6.0
159 silly decomposeActions install deep-extend@0.6.0
160 silly decomposeActions postinstall deep-extend@0.6.0
161 silly decomposeActions finalize deep-extend@0.6.0
162 silly decomposeActions refresh-package-json deep-extend@0.6.0
163 silly decomposeActions preinstall delegates@1.0.0
164 silly decomposeActions build delegates@1.0.0
165 silly decomposeActions install delegates@1.0.0
166 silly decomposeActions postinstall delegates@1.0.0
167 silly decomposeActions finalize delegates@1.0.0
168 silly decomposeActions refresh-package-json delegates@1.0.0
169 silly decomposeActions preinstall detect-libc@1.0.3
170 silly decomposeActions build detect-libc@1.0.3
171 silly decomposeActions install detect-libc@1.0.3
172 silly decomposeActions postinstall detect-libc@1.0.3
173 silly decomposeActions finalize detect-libc@1.0.3
174 silly decomposeActions refresh-package-json detect-libc@1.0.3
175 silly decomposeActions preinstall fs.realpath@1.0.0
176 silly decomposeActions build fs.realpath@1.0.0
177 silly decomposeActions install fs.realpath@1.0.0
178 silly decomposeActions postinstall fs.realpath@1.0.0
179 silly decomposeActions finalize fs.realpath@1.0.0
180 silly decomposeActions refresh-package-json fs.realpath@1.0.0
181 silly decomposeActions preinstall has-unicode@2.0.1
182 silly decomposeActions build has-unicode@2.0.1
183 silly decomposeActions install has-unicode@2.0.1
184 silly decomposeActions postinstall has-unicode@2.0.1
185 silly decomposeActions finalize has-unicode@2.0.1
186 silly decomposeActions refresh-package-json has-unicode@2.0.1
187 silly decomposeActions preinstall inherits@2.0.4
188 silly decomposeActions build inherits@2.0.4
189 silly decomposeActions install inherits@2.0.4
190 silly decomposeActions postinstall inherits@2.0.4
191 silly decomposeActions finalize inherits@2.0.4
192 silly decomposeActions refresh-package-json inherits@2.0.4
193 silly decomposeActions preinstall ini@1.3.5
194 silly decomposeActions build ini@1.3.5
195 silly decomposeActions install ini@1.3.5
196 silly decomposeActions postinstall ini@1.3.5
197 silly decomposeActions finalize ini@1.3.5
198 silly decomposeActions refresh-package-json ini@1.3.5
199 silly decomposeActions preinstall isarray@1.0.0
200 silly decomposeActions build isarray@1.0.0
201 silly decomposeActions install isarray@1.0.0
202 silly decomposeActions postinstall isarray@1.0.0
203 silly decomposeActions finalize isarray@1.0.0
204 silly decomposeActions refresh-package-json isarray@1.0.0
205 silly decomposeActions preinstall minimatch@3.0.4
206 silly decomposeActions build minimatch@3.0.4
207 silly decomposeActions install minimatch@3.0.4
208 silly decomposeActions postinstall minimatch@3.0.4
209 silly decomposeActions finalize minimatch@3.0.4
210 silly decomposeActions refresh-package-json minimatch@3.0.4
211 silly decomposeActions preinstall ignore-walk@3.0.3
212 silly decomposeActions build ignore-walk@3.0.3
213 silly decomposeActions install ignore-walk@3.0.3
214 silly decomposeActions postinstall ignore-walk@3.0.3
215 silly decomposeActions finalize ignore-walk@3.0.3
216 silly decomposeActions refresh-package-json ignore-walk@3.0.3
217 silly decomposeActions preinstall minimist@1.2.5
218 silly decomposeActions build minimist@1.2.5
219 silly decomposeActions install minimist@1.2.5
220 silly decomposeActions postinstall minimist@1.2.5
221 silly decomposeActions finalize minimist@1.2.5
222 silly decomposeActions refresh-package-json minimist@1.2.5
223 silly decomposeActions preinstall mkdirp@0.5.3
224 silly decomposeActions build mkdirp@0.5.3
225 silly decomposeActions install mkdirp@0.5.3
226 silly decomposeActions postinstall mkdirp@0.5.3
227 silly decomposeActions finalize mkdirp@0.5.3
228 silly decomposeActions refresh-package-json mkdirp@0.5.3
229 silly decomposeActions preinstall ms@2.1.2
230 silly decomposeActions build ms@2.1.2
231 silly decomposeActions install ms@2.1.2
232 silly decomposeActions postinstall ms@2.1.2
233 silly decomposeActions finalize ms@2.1.2
234 silly decomposeActions refresh-package-json ms@2.1.2
235 silly decomposeActions preinstall debug@3.2.6
236 silly decomposeActions build debug@3.2.6
237 silly decomposeActions install debug@3.2.6
238 silly decomposeActions postinstall debug@3.2.6
239 silly decomposeActions finalize debug@3.2.6
240 silly decomposeActions refresh-package-json debug@3.2.6
241 silly decomposeActions preinstall npm-normalize-package-bin@1.0.1
242 silly decomposeActions build npm-normalize-package-bin@1.0.1
243 silly decomposeActions install npm-normalize-package-bin@1.0.1
244 silly decomposeActions postinstall npm-normalize-package-bin@1.0.1
245 silly decomposeActions finalize npm-normalize-package-bin@1.0.1
246 silly decomposeActions refresh-package-json npm-normalize-package-bin@1.0.1
247 silly decomposeActions preinstall npm-bundled@1.1.1
248 silly decomposeActions build npm-bundled@1.1.1
249 silly decomposeActions install npm-bundled@1.1.1
250 silly decomposeActions postinstall npm-bundled@1.1.1
251 silly decomposeActions finalize npm-bundled@1.1.1
252 silly decomposeActions refresh-package-json npm-bundled@1.1.1
253 silly decomposeActions preinstall npm-packlist@1.4.8
254 silly decomposeActions build npm-packlist@1.4.8
255 silly decomposeActions install npm-packlist@1.4.8
256 silly decomposeActions postinstall npm-packlist@1.4.8
257 silly decomposeActions finalize npm-packlist@1.4.8
258 silly decomposeActions refresh-package-json npm-packlist@1.4.8
259 silly decomposeActions preinstall number-is-nan@1.0.1
260 silly decomposeActions build number-is-nan@1.0.1
261 silly decomposeActions install number-is-nan@1.0.1
262 silly decomposeActions postinstall number-is-nan@1.0.1
263 silly decomposeActions finalize number-is-nan@1.0.1
264 silly decomposeActions refresh-package-json number-is-nan@1.0.1
265 silly decomposeActions preinstall is-fullwidth-code-point@1.0.0
266 silly decomposeActions build is-fullwidth-code-point@1.0.0
267 silly decomposeActions install is-fullwidth-code-point@1.0.0
268 silly decomposeActions postinstall is-fullwidth-code-point@1.0.0
269 silly decomposeActions finalize is-fullwidth-code-point@1.0.0
270 silly decomposeActions refresh-package-json is-fullwidth-code-point@1.0.0
271 silly decomposeActions preinstall object-assign@4.1.1
272 silly decomposeActions build object-assign@4.1.1
273 silly decomposeActions install object-assign@4.1.1
274 silly decomposeActions postinstall object-assign@4.1.1
275 silly decomposeActions finalize object-assign@4.1.1
276 silly decomposeActions refresh-package-json object-assign@4.1.1
277 silly decomposeActions preinstall os-homedir@1.0.2
278 silly decomposeActions build os-homedir@1.0.2
279 silly decomposeActions install os-homedir@1.0.2
280 silly decomposeActions postinstall os-homedir@1.0.2
281 silly decomposeActions finalize os-homedir@1.0.2
282 silly decomposeActions refresh-package-json os-homedir@1.0.2
283 silly decomposeActions preinstall os-tmpdir@1.0.2
284 silly decomposeActions build os-tmpdir@1.0.2
285 silly decomposeActions install os-tmpdir@1.0.2
286 silly decomposeActions postinstall os-tmpdir@1.0.2
287 silly decomposeActions finalize os-tmpdir@1.0.2
288 silly decomposeActions refresh-package-json os-tmpdir@1.0.2
289 silly decomposeActions preinstall osenv@0.1.5
290 silly decomposeActions build osenv@0.1.5
291 silly decomposeActions install osenv@0.1.5
292 silly decomposeActions postinstall osenv@0.1.5
293 silly decomposeActions finalize osenv@0.1.5
294 silly decomposeActions refresh-package-json osenv@0.1.5
295 silly decomposeActions preinstall nopt@4.0.3
296 silly decomposeActions build nopt@4.0.3
297 silly decomposeActions install nopt@4.0.3
298 silly decomposeActions postinstall nopt@4.0.3
299 silly decomposeActions finalize nopt@4.0.3
300 silly decomposeActions refresh-package-json nopt@4.0.3
301 silly decomposeActions preinstall path-is-absolute@1.0.1
302 silly decomposeActions build path-is-absolute@1.0.1
303 silly decomposeActions install path-is-absolute@1.0.1
304 silly decomposeActions postinstall path-is-absolute@1.0.1
305 silly decomposeActions finalize path-is-absolute@1.0.1
306 silly decomposeActions refresh-package-json path-is-absolute@1.0.1
307 silly decomposeActions preinstall process-nextick-args@2.0.1
308 silly decomposeActions build process-nextick-args@2.0.1
309 silly decomposeActions install process-nextick-args@2.0.1
310 silly decomposeActions postinstall process-nextick-args@2.0.1
311 silly decomposeActions finalize process-nextick-args@2.0.1
312 silly decomposeActions refresh-package-json process-nextick-args@2.0.1
313 silly decomposeActions preinstall safe-buffer@5.1.2
314 silly decomposeActions build safe-buffer@5.1.2
315 silly decomposeActions install safe-buffer@5.1.2
316 silly decomposeActions postinstall safe-buffer@5.1.2
317 silly decomposeActions finalize safe-buffer@5.1.2
318 silly decomposeActions refresh-package-json safe-buffer@5.1.2
319 silly decomposeActions preinstall safer-buffer@2.1.2
320 silly decomposeActions build safer-buffer@2.1.2
321 silly decomposeActions install safer-buffer@2.1.2
322 silly decomposeActions postinstall safer-buffer@2.1.2
323 silly decomposeActions finalize safer-buffer@2.1.2
324 silly decomposeActions refresh-package-json safer-buffer@2.1.2
325 silly decomposeActions preinstall iconv-lite@0.4.24
326 silly decomposeActions build iconv-lite@0.4.24
327 silly decomposeActions install iconv-lite@0.4.24
328 silly decomposeActions postinstall iconv-lite@0.4.24
329 silly decomposeActions finalize iconv-lite@0.4.24
330 silly decomposeActions refresh-package-json iconv-lite@0.4.24
331 silly decomposeActions preinstall sax@1.2.4
332 silly decomposeActions build sax@1.2.4
333 silly decomposeActions install sax@1.2.4
334 silly decomposeActions postinstall sax@1.2.4
335 silly decomposeActions finalize sax@1.2.4
336 silly decomposeActions refresh-package-json sax@1.2.4
337 silly decomposeActions preinstall needle@2.3.3
338 silly decomposeActions build needle@2.3.3
339 silly decomposeActions install needle@2.3.3
340 silly decomposeActions postinstall needle@2.3.3
341 silly decomposeActions finalize needle@2.3.3
342 silly decomposeActions refresh-package-json needle@2.3.3
343 silly decomposeActions preinstall semver@5.7.1
344 silly decomposeActions build semver@5.7.1
345 silly decomposeActions install semver@5.7.1
346 silly decomposeActions postinstall semver@5.7.1
347 silly decomposeActions finalize semver@5.7.1
348 silly decomposeActions refresh-package-json semver@5.7.1
349 silly decomposeActions preinstall set-blocking@2.0.0
350 silly decomposeActions build set-blocking@2.0.0
351 silly decomposeActions install set-blocking@2.0.0
352 silly decomposeActions postinstall set-blocking@2.0.0
353 silly decomposeActions finalize set-blocking@2.0.0
354 silly decomposeActions refresh-package-json set-blocking@2.0.0
355 silly decomposeActions preinstall signal-exit@3.0.2
356 silly decomposeActions build signal-exit@3.0.2
357 silly decomposeActions install signal-exit@3.0.2
358 silly decomposeActions postinstall signal-exit@3.0.2
359 silly decomposeActions finalize signal-exit@3.0.2
360 silly decomposeActions refresh-package-json signal-exit@3.0.2
361 silly decomposeActions preinstall string_decoder@1.1.1
362 silly decomposeActions build string_decoder@1.1.1
363 silly decomposeActions install string_decoder@1.1.1
364 silly decomposeActions postinstall string_decoder@1.1.1
365 silly decomposeActions finalize string_decoder@1.1.1
366 silly decomposeActions refresh-package-json string_decoder@1.1.1
367 silly decomposeActions preinstall strip-ansi@3.0.1
368 silly decomposeActions build strip-ansi@3.0.1
369 silly decomposeActions install strip-ansi@3.0.1
370 silly decomposeActions postinstall strip-ansi@3.0.1
371 silly decomposeActions finalize strip-ansi@3.0.1
372 silly decomposeActions refresh-package-json strip-ansi@3.0.1
373 silly decomposeActions preinstall string-width@1.0.2
374 silly decomposeActions build string-width@1.0.2
375 silly decomposeActions install string-width@1.0.2
376 silly decomposeActions postinstall string-width@1.0.2
377 silly decomposeActions finalize string-width@1.0.2
378 silly decomposeActions refresh-package-json string-width@1.0.2
379 silly decomposeActions preinstall strip-json-comments@2.0.1
380 silly decomposeActions build strip-json-comments@2.0.1
381 silly decomposeActions install strip-json-comments@2.0.1
382 silly decomposeActions postinstall strip-json-comments@2.0.1
383 silly decomposeActions finalize strip-json-comments@2.0.1
384 silly decomposeActions refresh-package-json strip-json-comments@2.0.1
385 silly decomposeActions preinstall rc@1.2.8
386 silly decomposeActions build rc@1.2.8
387 silly decomposeActions install rc@1.2.8
388 silly decomposeActions postinstall rc@1.2.8
389 silly decomposeActions finalize rc@1.2.8
390 silly decomposeActions refresh-package-json rc@1.2.8
391 silly decomposeActions preinstall util-deprecate@1.0.2
392 silly decomposeActions build util-deprecate@1.0.2
393 silly decomposeActions install util-deprecate@1.0.2
394 silly decomposeActions postinstall util-deprecate@1.0.2
395 silly decomposeActions finalize util-deprecate@1.0.2
396 silly decomposeActions refresh-package-json util-deprecate@1.0.2
397 silly decomposeActions preinstall readable-stream@2.3.7
398 silly decomposeActions build readable-stream@2.3.7
399 silly decomposeActions install readable-stream@2.3.7
400 silly decomposeActions postinstall readable-stream@2.3.7
401 silly decomposeActions finalize readable-stream@2.3.7
402 silly decomposeActions refresh-package-json readable-stream@2.3.7
403 silly decomposeActions preinstall are-we-there-yet@1.1.5
404 silly decomposeActions build are-we-there-yet@1.1.5
405 silly decomposeActions install are-we-there-yet@1.1.5
406 silly decomposeActions postinstall are-we-there-yet@1.1.5
407 silly decomposeActions finalize are-we-there-yet@1.1.5
408 silly decomposeActions refresh-package-json are-we-there-yet@1.1.5
409 silly decomposeActions preinstall wide-align@1.1.3
410 silly decomposeActions build wide-align@1.1.3
411 silly decomposeActions install wide-align@1.1.3
412 silly decomposeActions postinstall wide-align@1.1.3
413 silly decomposeActions finalize wide-align@1.1.3
414 silly decomposeActions refresh-package-json wide-align@1.1.3
415 silly decomposeActions preinstall gauge@2.7.4
416 silly decomposeActions build gauge@2.7.4
417 silly decomposeActions install gauge@2.7.4
418 silly decomposeActions postinstall gauge@2.7.4
419 silly decomposeActions finalize gauge@2.7.4
420 silly decomposeActions refresh-package-json gauge@2.7.4
421 silly decomposeActions preinstall npmlog@4.1.2
422 silly decomposeActions build npmlog@4.1.2
423 silly decomposeActions install npmlog@4.1.2
424 silly decomposeActions postinstall npmlog@4.1.2
425 silly decomposeActions finalize npmlog@4.1.2
426 silly decomposeActions refresh-package-json npmlog@4.1.2
427 silly decomposeActions preinstall wrappy@1.0.2
428 silly decomposeActions build wrappy@1.0.2
429 silly decomposeActions install wrappy@1.0.2
430 silly decomposeActions postinstall wrappy@1.0.2
431 silly decomposeActions finalize wrappy@1.0.2
432 silly decomposeActions refresh-package-json wrappy@1.0.2
433 silly decomposeActions preinstall once@1.4.0
434 silly decomposeActions build once@1.4.0
435 silly decomposeActions install once@1.4.0
436 silly decomposeActions postinstall once@1.4.0
437 silly decomposeActions finalize once@1.4.0
438 silly decomposeActions refresh-package-json once@1.4.0
439 silly decomposeActions preinstall inflight@1.0.6
440 silly decomposeActions build inflight@1.0.6
441 silly decomposeActions install inflight@1.0.6
442 silly decomposeActions postinstall inflight@1.0.6
443 silly decomposeActions finalize inflight@1.0.6
444 silly decomposeActions refresh-package-json inflight@1.0.6
445 silly decomposeActions preinstall glob@7.1.6
446 silly decomposeActions build glob@7.1.6
447 silly decomposeActions install glob@7.1.6
448 silly decomposeActions postinstall glob@7.1.6
449 silly decomposeActions finalize glob@7.1.6
450 silly decomposeActions refresh-package-json glob@7.1.6
451 silly decomposeActions preinstall rimraf@2.7.1
452 silly decomposeActions build rimraf@2.7.1
453 silly decomposeActions install rimraf@2.7.1
454 silly decomposeActions postinstall rimraf@2.7.1
455 silly decomposeActions finalize rimraf@2.7.1
456 silly decomposeActions refresh-package-json rimraf@2.7.1
457 silly decomposeActions preinstall yallist@3.1.1
458 silly decomposeActions build yallist@3.1.1
459 silly decomposeActions install yallist@3.1.1
460 silly decomposeActions postinstall yallist@3.1.1
461 silly decomposeActions finalize yallist@3.1.1
462 silly decomposeActions refresh-package-json yallist@3.1.1
463 silly decomposeActions preinstall minipass@2.9.0
464 silly decomposeActions build minipass@2.9.0
465 silly decomposeActions install minipass@2.9.0
466 silly decomposeActions postinstall minipass@2.9.0
467 silly decomposeActions finalize minipass@2.9.0
468 silly decomposeActions refresh-package-json minipass@2.9.0
469 silly decomposeActions preinstall fs-minipass@1.2.7
470 silly decomposeActions build fs-minipass@1.2.7
471 silly decomposeActions install fs-minipass@1.2.7
472 silly decomposeActions postinstall fs-minipass@1.2.7
473 silly decomposeActions finalize fs-minipass@1.2.7
474 silly decomposeActions refresh-package-json fs-minipass@1.2.7
475 silly decomposeActions preinstall minizlib@1.3.3
476 silly decomposeActions build minizlib@1.3.3
477 silly decomposeActions install minizlib@1.3.3
478 silly decomposeActions postinstall minizlib@1.3.3
479 silly decomposeActions finalize minizlib@1.3.3
480 silly decomposeActions refresh-package-json minizlib@1.3.3
481 silly decomposeActions preinstall tar@4.4.13
482 silly decomposeActions build tar@4.4.13
483 silly decomposeActions install tar@4.4.13
484 silly decomposeActions postinstall tar@4.4.13
485 silly decomposeActions finalize tar@4.4.13
486 silly decomposeActions refresh-package-json tar@4.4.13
487 silly decomposeActions preinstall node-pre-gyp@0.14.0
488 silly decomposeActions build node-pre-gyp@0.14.0
489 silly decomposeActions install node-pre-gyp@0.14.0
490 silly decomposeActions postinstall node-pre-gyp@0.14.0
491 silly decomposeActions finalize node-pre-gyp@0.14.0
492 silly decomposeActions refresh-package-json node-pre-gyp@0.14.0
493 silly decomposeActions preinstall fsevents@1.2.12
494 silly decomposeActions build fsevents@1.2.12
495 silly decomposeActions install fsevents@1.2.12
496 silly decomposeActions postinstall fsevents@1.2.12
497 silly decomposeActions finalize fsevents@1.2.12
498 silly decomposeActions refresh-package-json fsevents@1.2.12
499 silly decomposeActions fetch fsevents@2.1.3
500 silly decomposeActions extract fsevents@2.1.3
501 silly decomposeActions preinstall fsevents@2.1.3
502 silly decomposeActions build fsevents@2.1.3
503 silly decomposeActions install fsevents@2.1.3
504 silly decomposeActions postinstall fsevents@2.1.3
505 silly decomposeActions finalize fsevents@2.1.3
506 silly decomposeActions refresh-package-json fsevents@2.1.3
507 silly decomposeActions fetch extend@3.0.2
508 silly decomposeActions extract extend@3.0.2
509 silly decomposeActions preinstall extend@3.0.2
510 silly decomposeActions build extend@3.0.2
511 silly decomposeActions install extend@3.0.2
512 silly decomposeActions postinstall extend@3.0.2
513 silly decomposeActions finalize extend@3.0.2
514 silly decomposeActions refresh-package-json extend@3.0.2
515 silly decomposeActions fetch mdn-browser-compat-data@1.0.20
516 silly decomposeActions extract mdn-browser-compat-data@1.0.20
517 silly decomposeActions preinstall mdn-browser-compat-data@1.0.20
518 silly decomposeActions build mdn-browser-compat-data@1.0.20
519 silly decomposeActions install mdn-browser-compat-data@1.0.20
520 silly decomposeActions postinstall mdn-browser-compat-data@1.0.20
521 silly decomposeActions finalize mdn-browser-compat-data@1.0.20
522 silly decomposeActions refresh-package-json mdn-browser-compat-data@1.0.20
523 silly install executeActions
524 silly doSerial global-install 428
525 verbose correctMkdir /home/openwpm/.npm/_locks correctMkdir not in flight; initializing
526 verbose lock using /home/openwpm/.npm/_locks/staging-4d6b4dd5a13244e0.lock for /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/.staging
527 silly doParallel extract 4
528 silly extract fsevents@1.2.12
529 silly extract fsevents@2.1.3
530 silly extract extend@3.0.2
531 silly extract mdn-browser-compat-data@1.0.20
532 silly tarball trying fsevents@1.2.12 by hash: sha512-Ggd/Ktt7E7I8pxZRbGIs7vwqAPscSESMrCSkx2FtWeqmheJgCo2R74fTsZFCifr0VTPwqRpPv17+6b8Zp7th0Q==
533 silly tarball trying fsevents@2.1.3 by hash: sha512-Auw9a4AxqWpa9GUfj370BMPzzyncfBABW8Mab7BGWBYDj4Isgq+cDKtx0i6u9jcX9pQDnswsaaOTgTmA5pEjuQ==
534 silly tarball trying extend@3.0.2 by hash: sha512-fjquC59cD7CyW6urNXK0FBufkZcoiGG80wTuPujX590cB5Ttln20E2UB4S/WARVqhXffZl2LNgS+gQdPIIim/g==
535 silly tarball trying mdn-browser-compat-data@1.0.20 by hash: sha512-6sS4ydJ6+/ZnSwKUN8Jtu7NzD2pobXRolYRjHupKtRBDmx+8WZG1yUdOfF9QfCNPkWKrf3GcSFEurIrnJC56Qw==
536 silly extract fsevents@2.1.3 extracted to /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/.staging/fsevents-6ebd9707 (781ms)
537 silly extract extend@3.0.2 extracted to /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/.staging/extend-1d107085 (785ms)
538 http fetch POST 400 https://registry.npmjs.org/-/npm/v1/security/audits/quick 1379ms
539 silly extract fsevents@1.2.12 extracted to /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/.staging/fsevents-4a9a0788 (2006ms)
540 silly extract mdn-browser-compat-data@1.0.20 extracted to /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/.staging/mdn-browser-compat-data-c90e8845 (4317ms)
541 timing action:extract Completed in 4357ms
542 silly doReverseSerial unbuild 428
543 silly doSerial remove 428
544 silly doSerial move 428
545 silly doSerial finalize 428
546 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/abbrev
547 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/ansi-regex
548 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/aproba
549 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/balanced-match
550 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/chownr
551 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/code-point-at
552 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/concat-map
553 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/brace-expansion
554 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/console-control-strings
555 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/core-util-is
556 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/deep-extend
557 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/delegates
558 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/detect-libc
559 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/fs.realpath
560 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/has-unicode
561 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/inherits
562 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/ini
563 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/isarray
564 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/minimatch
565 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/ignore-walk
566 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/minimist
567 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/mkdirp
568 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/ms
569 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/debug
570 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/npm-normalize-package-bin
571 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/npm-bundled
572 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/npm-packlist
573 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/number-is-nan
574 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/is-fullwidth-code-point
575 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/object-assign
576 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/os-homedir
577 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/os-tmpdir
578 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/osenv
579 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/nopt
580 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/path-is-absolute
581 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/process-nextick-args
582 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/safe-buffer
583 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/safer-buffer
584 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/iconv-lite
585 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/sax
586 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/needle
587 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/semver
588 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/set-blocking
589 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/signal-exit
590 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/string_decoder
591 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/strip-ansi
592 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/string-width
593 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/strip-json-comments
594 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/rc
595 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/util-deprecate
596 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/readable-stream
597 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/are-we-there-yet
598 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/wide-align
599 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/gauge
600 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/npmlog
601 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/wrappy
602 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/once
603 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/inflight
604 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/glob
605 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/rimraf
606 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/yallist
607 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/minipass
608 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/fs-minipass
609 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/minizlib
610 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/tar
611 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/node-pre-gyp
612 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents
613 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/fsevents
614 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/node_modules/extend
615 silly finalize /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/node_modules/mdn-browser-compat-data
616 timing action:finalize Completed in 320ms
617 silly doParallel refresh-package-json 70
618 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/abbrev
619 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/ansi-regex
620 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/aproba
621 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/balanced-match
622 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/chownr
623 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/code-point-at
624 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/concat-map
625 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/brace-expansion
626 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/console-control-strings
627 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/core-util-is
628 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/deep-extend
629 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/delegates
630 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/detect-libc
631 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/fs.realpath
632 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/has-unicode
633 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/inherits
634 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/ini
635 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/isarray
636 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/minimatch
637 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/ignore-walk
638 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/minimist
639 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/mkdirp
640 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/ms
641 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/debug
642 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/npm-normalize-package-bin
643 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/npm-bundled
644 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/npm-packlist
645 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/number-is-nan
646 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/is-fullwidth-code-point
647 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/object-assign
648 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/os-homedir
649 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/os-tmpdir
650 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/osenv
651 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/nopt
652 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/path-is-absolute
653 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/process-nextick-args
654 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/safe-buffer
655 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/safer-buffer
656 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/iconv-lite
657 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/sax
658 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/needle
659 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/semver
660 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/set-blocking
661 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/signal-exit
662 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/string_decoder
663 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/strip-ansi
664 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/string-width
665 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/strip-json-comments
666 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/rc
667 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/util-deprecate
668 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/node_modules/mdn-browser-compat-data
669 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/webext-instrumentation/node_modules/extend
670 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/fsevents
671 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents
672 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/node-pre-gyp
673 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/tar
674 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/minizlib
675 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/fs-minipass
676 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/minipass
677 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/yallist
678 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/rimraf
679 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/glob
680 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/inflight
681 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/once
682 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/wrappy
683 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/npmlog
684 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/gauge
685 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/wide-align
686 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/are-we-there-yet
687 silly refresh-package-json /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/chokidar/node_modules/fsevents/node_modules/readable-stream
688 timing action:refresh-package-json Completed in 873ms
689 silly doParallel preinstall 70
690 silly preinstall extend@3.0.2
691 info lifecycle extend@3.0.2~preinstall: extend@3.0.2
692 silly preinstall mdn-browser-compat-data@1.0.20
693 info lifecycle mdn-browser-compat-data@1.0.20~preinstall: mdn-browser-compat-data@1.0.20
694 timing action:preinstall Completed in 6ms
695 silly doSerial build 428
696 silly build extend@3.0.2
697 info linkStuff extend@3.0.2
698 silly linkStuff extend@3.0.2 has /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/openwpm-webext-instrumentation/node_modules as its parent node_modules
699 silly build mdn-browser-compat-data@1.0.20
700 info linkStuff mdn-browser-compat-data@1.0.20
701 silly linkStuff mdn-browser-compat-data@1.0.20 has /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/openwpm-webext-instrumentation/node_modules as its parent node_modules
702 timing action:build Completed in 7ms
703 silly doSerial global-link 428
704 silly doParallel update-linked 0
705 silly doSerial install 428
706 silly install extend@3.0.2
707 info lifecycle extend@3.0.2~install: extend@3.0.2
708 silly install mdn-browser-compat-data@1.0.20
709 info lifecycle mdn-browser-compat-data@1.0.20~install: mdn-browser-compat-data@1.0.20
710 timing action:install Completed in 7ms
711 silly doSerial postinstall 428
712 silly postinstall extend@3.0.2
713 info lifecycle extend@3.0.2~postinstall: extend@3.0.2
714 silly postinstall mdn-browser-compat-data@1.0.20
715 info lifecycle mdn-browser-compat-data@1.0.20~postinstall: mdn-browser-compat-data@1.0.20
716 timing action:postinstall Completed in 40ms
717 verbose unlock done using /home/openwpm/.npm/_locks/staging-4d6b4dd5a13244e0.lock for /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/.staging
718 timing stage:executeActions Completed in 5777ms
719 timing stage:rollbackFailedOptional Completed in 619ms
720 silly install runPostinstallTopLevelLifecycles
721 silly build OpenWPM@
722 info linkStuff OpenWPM@
723 silly linkStuff OpenWPM@ has /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension as its parent node_modules
724 silly install OpenWPM@
725 info lifecycle OpenWPM@~install: OpenWPM@
726 silly postinstall OpenWPM@
727 info lifecycle OpenWPM@~postinstall: OpenWPM@
728 verbose lifecycle OpenWPM@~postinstall: unsafe-perm in lifecycle true
729 verbose lifecycle OpenWPM@~postinstall: PATH: /home/openwpm/anaconda3/envs/openwpm/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox/node_modules/.bin:/home/openwpm/anaconda3/envs/openwpm/bin:/home/openwpm/anaconda3/condabin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
730 verbose lifecycle OpenWPM@~postinstall: CWD: /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox
731 silly lifecycle OpenWPM@~postinstall: Args: [ '-c', 'cd ../webext-instrumentation && npm install' ]
732 silly lifecycle OpenWPM@~postinstall: Returned: code: 1  signal: null
733 info lifecycle OpenWPM@~postinstall: Failed to exec postinstall script
734 verbose stack Error: OpenWPM@ postinstall: `cd ../webext-instrumentation && npm install`
734 verbose stack Exit status 1
734 verbose stack     at EventEmitter.<anonymous> (/home/openwpm/anaconda3/envs/openwpm/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
734 verbose stack     at EventEmitter.emit (events.js:314:20)
734 verbose stack     at ChildProcess.<anonymous> (/home/openwpm/anaconda3/envs/openwpm/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
734 verbose stack     at ChildProcess.emit (events.js:314:20)
734 verbose stack     at maybeClose (internal/child_process.js:1051:16)
734 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
735 verbose pkgid OpenWPM@
736 verbose cwd /home/openwpm/MyOpenWPM/OpenWPM/automation/Extension/firefox
737 verbose Linux 5.4.0-42-generic
738 verbose argv "/home/openwpm/anaconda3/envs/openwpm/bin/node" "/home/openwpm/anaconda3/envs/openwpm/bin/npm" "install"
739 verbose node v14.5.0
740 verbose npm  v6.14.5
741 error code ELIFECYCLE
742 error errno 1
743 error OpenWPM@ postinstall: `cd ../webext-instrumentation && npm install`
743 error Exit status 1
744 error Failed at the OpenWPM@ postinstall script.
744 error This is probably not a problem with npm. There is likely additional logging output above.
745 verbose exit [ 1, true ]
(openwpm) openwpm@openwpm-VirtualBox:~/MyOpenWPM/OpenWPM$ 
```

Reply:
After reading the logs it seems to me like the script is failing to change directories.

> 743 error OpenWPM@ postinstall: `cd ../webext-instrumentation && npm install`
> 743 error Exit status 1
> 744 error Failed at the OpenWPM@ postinstall script.

But I unfortunately don't know that much about how the WebExtension actually gets built.

Reply:
I am unsure whether `cd` or `npm` is the problem here. If I remember corectly, the script failed during `npm`.